<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Longevity report</title>
<style >

	/*general styles*/
    *{
        box-sizing: border-box;
        scroll-behavior: smooth;
    }
	ol{
        margin-top: 2vw;
        margin-bottom: 2vw;
        font-size: 1.1vw;
    }
    ol li{
        margin: 1vw 0;
    }
	a {
        color: #87cec3;
        text-decoration: none;
    }
    a:hover{
        color: #92e0d3;
    }
	body{
        background-image: linear-gradient(to left bottom, rgb(249, 246, 240), #f8f6ec, #f6f6e8, #f2f7e6, #edf8e4, #e5f9e8, #dff9ec, #d9f9f2, #d8f7fb, #def5ff, #e7f1ff, #f0eefd) !important;
        background-repeat: no-repeat;
        background-attachment: fixed;
        color:rgb(66, 92, 88);
        font-family: Arial, Helvetica, sans-serif;

    }
	canvas{
        width: 18vw;
        height: 9vw;
	}

		/*style for intro section*/
		.intro{
        width: 60%;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        line-height: 3;
        font-size: 1vw;
    }
    .intro p{
        color:rgb(84, 85, 110);
        line-height: 2em;
        font-size: 1.3em;
        font-weight: 400;
    }
    .intro a{
        word-wrap: break-word;
    }
    .intro ul{
        padding-left: 0;
        margin-left: auto;
        margin-right: auto;
        font-weight: 500;
        margin-top: 40px;
        list-style-type: none;
        margin-bottom: 120px;
    }
    .li1{
        font-weight: 400;
        font-size: 1.3vw;
    }
	#prs, #prs1{
		color: #b2e457;
	}
	.attention strong{
		color: rgb(71, 155, 148);
		font-size: 1.25vw;
	}
	.attention p{
		color:#3a6e65;
	}
	.attention{
		margin-top: 4vw;
		margin-bottom: 4vw;
		margin-left: auto;
		margin-right: auto;
		width: 65%;
		padding: 0.8vw 1.7vw 0.8vw 1.7vw;
		font-size: 0.8vw;
		border:rgb(122, 189, 183)solid 5px;
		border-radius: 30px;
		background-color: white;
		box-shadow: 0px 5px 5px 0px #4a887c;
	}
	.first{
		margin-top: 3vw;
		color:#269292;
		font-size: 3vw;
	}




	/*table styles*/
    table.ui{
		transition: max-height 0.5s ease-in;
        border-radius: 0.28571429rem;
        font-size: 0.8vw;
        background: white;
		width: 85%;
		margin-right:auto;
		margin-left: auto;
        border-radius: 0.28571429rem;
        padding: 0.78571429em 0.78571429em;
        text-align: left;
        border-collapse: collapse;
        vertical-align: middle;
        border: 1px solid rgba(34,36,38,.15);
        line-height: 1.4285em;
    }
    table.ui th{
        background: #f9fafb;
        color: rgba(0,0,0,.87);
        padding: 0.92857143em 0.78571429em;
        vertical-align: inherit;
        font-style: none;
        font-weight: 700;
        text-transform: none;
        border-bottom: 1px solid rgba(34,36,38,.1);
        border-right: 1px solid rgba(34,36,38,.1);
    }
    table.ui.selectable.celled.teal.table tr:hover{
        background-color: #eef1f5;
    }
    table.ui td{
        border-bottom: 1px solid rgba(34,36,38,.1);
        border-top: none;
        border-left: 1px solid rgba(34,36,38,.1);
        padding: 10px 10px 10px 10px;
    }
    table.ui td:first-child{
        border-left: none;
    }
	.ui.celled.teal.table.prs
    {
        width: 100%;
        margin-top: 3vw;
        color:rgb(66, 92, 88);
        box-shadow: 0 1px 3px 1px rgba(118, 49, 40, 0.3);
        border-top: 0.4em solid #56C3BD;
    }
    .ui.selectable.celled.teal.table.table.variants{
        margin-top: 0px;
        box-shadow: 0 1px 3px 1px rgba(118, 49, 40, 0.3);
        margin-bottom: 80px;
        border-top: 0.4em solid rgb(101, 189, 162);
    }
    .ui.celled.teal.table.table.drugs{
        box-shadow: 0 1px 3px 1px rgba(118, 49, 40, 0.3);
        border-top: 0.4em solid rgb(101, 189, 162);
		margin-top:0;
        }
    .ui.selectable.celled.teal.table.table.cancer{
        box-shadow: 0 1px 3px 1px rgba(118, 49, 40, 0.3);
        margin-bottom: 80px ;
        border-top: 0.4em solid rgb(101, 189, 162);
    }

		/*button style*/
		div.ui.tiny.circular.compact.positive.button {
			border-radius: 10em;
			background-color: #86caf7;
			color: #fff;
			padding: 0.58928571em 1.125em 0.58928571em;
			cursor: pointer;
			display: inline-block;
			min-height: 1em;
			outline: 0;
			border: none;
			vertical-align: baseline;
			margin: 0 0.25em 0 0;
			font-weight: 700;
			line-height: 1em;
			font-style: normal;
			text-align: center;
			}
		div.ui.tiny.circular.compact.negative.button {
			border-radius: 10em;
			background-color: #a6f069;
			color: #fff;
			padding: 0.58928571em 1.125em 0.58928571em;
			cursor: pointer;
			display: inline-block;
			min-height: 1em;
			outline: 0;
			border: none;
			vertical-align: baseline;
			margin: 0 0.25em 0 0;
			font-weight: 700;
			line-height: 1em;
			font-style: normal;
			text-align: center;
		}
	th, td {
      padding: 2px;
    }

	.accordion.accordion0.accordion1.prs-panel{
		margin-top: 0.7vw;
	}


	/* accordions styles*/
	.accordion0{
		position: relative;
		border:rgb(101, 189, 162)  solid 6px;
		border-bottom: none;
		width: 50%;
		margin-left: 25%;
		margin-right: 25%;
		text-align: left;
		padding-left: 2vw;
		height: 4vw;
		font-size: 1.5vw;
		background-color: rgba(255, 255, 255, 0.5);
		border-radius: 10px 10px 10px 10px;
		margin-top: 0.01vw;
		color: rgb(81, 139, 122);
		margin-top: 0.7vw;
		border-bottom: none;
		cursor: pointer;
	}
	.accordion.accordion-heading.active, .accordion:hover{
		background-color: rgba(255, 255, 255, 0.2);
	}
	.accordion.accordion-heading:hover, .accordion.accordion-heading.active:hover{
		color:rgb(127, 201, 182);
	}
	.accordion1{
		margin-top: 3vw;
	}

	.accordion2{
		border:#577de4 solid 5px;
	}
    /* accordion heading styles*/
	.accordion-heading{
		width: 50%;
		margin-left: 25%;
		margin-right: 25%;
		user-select: none;
		height: 5vw;
		padding-top: 1vw;
		padding-bottom: 1vw;
		cursor: pointer;
		border:rgb(101, 189, 162, 0.5)  solid 6px;
		background-color: rgba(255, 255, 255, 0.5);
		font-size: 1.7vw;
		border-radius: 15px;
		margin-top: 2vw;
		transition-property: width, margin-left, margin-right;
		transition-duration: 0.2s, 0.2s, 0.2s;
		color:rgb(47, 116, 106);
		border-bottom: none;}
	.accordion-heading:hover{
		background-color: rgba(255, 255, 255, 0.5);
	}




			

/*used in each div after description and recommendation buttons*/
    
	/*exist in both descr and rec*/
	.board{
        width: 50%;
        margin-top: 40px;
        padding-top: 10px;
        color:rgb(84, 85, 110);
        margin-right:auto;
        margin-left:auto;
		font-size: 1.1vw;
    }
    .board p{
        line-height: 1.7;
        font-weight: 400;
    }
    .board .how_to{
        font-size: 1.3vw;
		color:#29332f
    }


	.panel{
		padding: 1.5vw 2.5vw 1.5vw 2.5vw;
		display: none;
		border:rgb(66, 92, 88) solid 3px;
		border-bottom-left-radius: 15px;
		border-bottom-right-radius: 15px;
		border-top: none;
		background-color: white;
		margin-top: 0px;
	}

	/*just styles to colour panels for results and descriptions*/
	.variants.panel{
		border: rgb(101, 189, 162) 6px solid;
		border-top:none;
	}
	.panel-results{
		border: #a0cf47 6px solid;
		border-top:none;
	}


    /* only appears after results*/
    .content{
        margin-left: auto;
        margin-right: auto;
        overflow-x: auto;
        transition: max-height 0.5s ease-in;
		display: none;
    }


	.hidden{
		max-height: 0;
		overflow: hidden;
		max-height: 0;
		transition: max-height 0.5s ease-in;
	}

	/*styles for accordion to work*/
	.basic-panel, .panel, .content{
		max-height: 0;
		overflow: hidden;
		transition-property: max-height, width;
		transition-duration: 0.2s, 0.5s;
	}
	.basic-panel{
		background-color: rgba(255, 255, 255, 0.2);
		margin-left: auto;
		margin-right: auto;
		width: 85%;
		border-right: rgb(101, 189, 162, 0.5) solid 6px;
		border-left: rgb(101, 189, 162, 0.5)  solid 6px;
		border-bottom-left-radius: 3%;
		border-bottom-right-radius: 3%;
	}

	/*animation styles*/
	.accordion-result, .accordion-result.active{
		transition-property: width, margin-left, margin-right;
		transition-duration: 0.2s, 0.2s, 0.2s;
	}


	/*styles for buttons inside basic-panel*/
	.basic-panel {
		width: 85%;
	}
	.accordion1, .basic-panel .accordion1 +.panel-description{
		width: 50%;
		margin-left: 25%;
		margin-right: 25%;
	}


	/* styles to color names of modules*/
	#variants{
		color: rgb(57, 145, 123);
	}
	
	/*colors for drugs module*/
	.accordion.accordion0.accordion1.accord_drugs, .accordion.accordion0.accordion-result.accord_drugs{
		color: rgb(81, 139, 122);
	}
	.accord_drugs{
		border: rgb(101, 189, 162) 6px solid;
		border-bottom: none;
	}


	/* styles for arrow */
	.accordion0 div{
		border-top: rgba(61, 128, 119, 0.7) solid 0.3vw;
		border-right:rgba(61, 128, 119, 0.7) solid 0.3vw;
		border-radius: 5px;
		height: 1vw;
		width:1vw;
		margin-right: 1vw;
		display: inline-block;
		transform: rotate(45deg);
		transition: transform 0.5s;
	}
	.accordion0.active div{
		transform: rotate(135deg);
	}


/* styles for drug module*/
	.accordion1.accord_drugs, .panel-description.accord_drugs{
		border: rgb(101, 189, 162)  6px solid;

	}
	.accordion1.accord_drugs{
		border-bottom: none;
	}
	.panel-description.accord_drugs{
		border-top: none;
	}
	.accordion.accordion0.accordion1.prs-panel.major-risk.special{
		margin-top: 3vw;
		font-size: 1.7vw;
	}


/*styles for major risks module*/
	.variants.panel.major-risk{
		border: rgb(101, 189, 162)  6px solid;
		border-bottom: none;}
		
	.variants.panel.major-risk{
		border-top:none;
	}
	.accordion0.major-risk{
		color:rgb(81, 139, 122);
 	}
	.content-variants.major-risk{
		border:none;
	}
	.basic-panel.major-risk{
		border-bottom: none;
		border-top: none;
	}

	.last.inside, .last{
		height:4vw;
	}
	.last.outside{
		height: 6vw;
	}


    h2{
        margin-top: 4vw;
        margin-bottom: 20px;
    }
	

	
    h2, h1{
        text-align: center;
		font-size: 2.5vw;
        font-weight: 500;
    }



    /* below is style for navigation button */
    #toTop{
		z-index: 100;
        height: 70px;
        width: 70px;
        background-color: #308499;
        border-radius: 50%;
        position: fixed;
        right: 2.35%;
        bottom: 10%;
        display:none;	
		box-shadow: 3px 3px 15px rgb(30, 77, 88, 0.7);
    }
    #toTop:hover{
        background-color:#56C3BD;
    }
    .arrow{
        position: absolute;
        bottom: 10%;
        z-index: 10;
    }
    .bodypart{
        width: 0;
        height: 0;
        border-left: 16px solid transparent;
        border-right: 16px solid transparent;
        border-bottom: 16px solid white;
        right: 20px;
        top:20%;
    }
    .line{
        right: 31px;
        bottom: 18px;
        background: white;
        width: 8px;
        height: 33px;
    }


	@media only screen and (max-width: 1400px) {
		.accordion.accordion0.accordion1.prs-panel.major-risk.special {
			font-size: 2.5vw;
		}
		.intro{
			width: 75%;
			font-size: 1.4vw;
		}
		.first{
			font-size: 4.5vw;
		}
		.attention p{
			font-size: 1.6vw;
		}
		.attention{
			margin-top: 7vw;
			margin-bottom:7vw;
			width: 90%;
		}
		.attention strong{
			font-size: 1.8vw;
		}
		.first{
			margin-top: 8vw;
		}
		h2{
			line-height: 5vw;
			font-size: 4vw;
		}
		.li1{
			font-size: 2.2vw;
		}
		.accordion0, .accordion-heading, .accordion0.accordion.accordion1{
			width:80%;
			margin-left:10%;
			margin-right:10%;
			height: 8vw;
			font-size: 2.5vw;
		}

		.panel,.accordion0.accordion.accordion1 +.panel-description{
			width:80%;
			margin-left: 10%;
			margin-right: 10%;
			font-size: 1.7vw;
			padding: 2.5vw 3.5vw 2.5vw 3.5vw;
		}

		.accordion0 div{
		height: 2vw;
		width:2vw;
		border-top: rgba(61, 128, 119, 0.7)  solid 0.6vw;
		border-right: rgba(61, 128, 119, 0.7) solid 0.6vw;
		margin-right: 2vw;
		}
	.basic-panel{
		width: 98%;
	}


		ol{
			margin-top:6vw;
			margin-bottom: 6vw;
			font-size: 1.7vw;
		}
		ol li{
			margin-top: 2vw;
		}
		table.ui{
			font-size: 1.3vw;
		}
		table.prs{
			width: 100%;
			font-size: 1.2vw;
		}
		.board .how_to{
		font-size: 2.2vw;
		}

		canvas{
        width: 36vw;
        height: 18vw;
    }
	#variants > button .module-button{
		width: 70%;
		margin-left: 15%;
		margin-right: 15%;
	}
	/*styles for button to top*/
	#toTop{
		right: 4%;
	}
	.acordiv{
		width:85%
	}
	.ui.celled.teal.table.table.drugs td, .ui.selectable.celled.teal.table.table.cancer td{
			padding: 5px;
		}
		.ui.selectable.celled.teal.table.table.cancer.cancer1 td, .ui.selectable.celled.teal.table.table.cancer.cardio th{
			padding: 5px;
		}
		.ui.celled.teal.table.table.drugs, .ui.selectable.celled.teal.table.table.cancer,.ui.selectable.celled.teal.table.table.variants{
			width: 98%;
	}
	/*.ui.selectable.celled.teal.table.table.cancer.cardio{
		width: 90%;
	}*/
	.ui.selectable.celled.teal.table.table.cancer.cardio td, .ui.celled.teal.table.table.drugs td{
		padding: 2px;
		font-size: 1.1vw;
	}

h2{
	margin-top:18vw;
}
}
	@media only screen and (max-width: 600px){
		.accordion.accordion0.accordion1.prs-panel.major-risk.special {
			font-size: 4vw;
		}
		.intro{
			width: 90%;
			font-size: 3.1vw;
		}
		.attention p{
			font-size: 3.1vw;
		}
		.attention strong{
			font-size: 4vw;
		}
		.attention{
			padding: 3vw 4vw 3vw 4vw;
		}
		.basic-panel, .basic-panel.major-risk{
			width:100%;
		}
		.accordion0, .accordion-heading, .accordion0.accordion.accordion1{
			height: 12vw;
			font-size: 4vw;
			width: 95%;
			margin-left: 2.5%;
			margin-right: 2.5%;
		}
		.panel, .accordion0.accordion.accordion1 +.panel-description, .accordion0.accordion.accordion1 + .prs-panel {
			margin-left:2.5%;
			margin-right:2.5%;
			width:95%;
			font-size: 3vw;;
			margin-bottom: 1vw;
			margin-top: 0;
		}
		.accordion0 div{
		height: 3.5vw;
		width:3.5vw;
		border-top: rgba(61, 128, 119, 0.7)  solid 1vw;
		border-right: rgba(61, 128, 119, 0.7) solid 1vw;
		margin-right: 2vw;
		margin-left: 1vw;
		}
		.accordion.accordion-heading{
			height: 16vw;
		}
		.first{
			font-size: 8vw;
		}
		.li1{
			font-size: 4.8vw;
		}
		.attention{
			margin-top: 10vw;
			margin-bottom: 10vw;
			width: 90%;
			border: rgb(71, 194, 184) solid  2px;
			border-radius: 13px;
		}
		.first{
			margin-top: 15vw;
		}
		table  .ui{
			margin-bottom: 1vw;
		}
		h2{
			line-height: 8vw;
			font-size: 7vw;
		}
		h1{
			line-height: 9vw;
			font-size: 6vw;
		}
		h2{
			margin-top: 20vw;
		}
		ol li{
			margin-top: 4vw;
		}
		ol{
			margin-top: 10vw;
			margin-bottom: 10vw;
		}
		.intro ul{
			margin-bottom:15vw;
		}
		.board.prs,.board.drugs,.board.cancer, .board.variants, .board.variants ol{
			font-size: 2.7vw;
			width: 95%;
		}
		.board.variants{
			margin-bottom: 15vw;
		}
		.board .how_to{
			font-size: 3.7vw;
		}
		.ui.celled.teal.table.prs{
			font-size: 2.1vw;
		}
		.ui.celled.teal.table.table.drugs td, .ui.selectable.celled.teal.table.table.cancer td, .ui.celled.teal.table.table.drugs th, .ui.selectable.celled.teal.table.table.cancer th{
			padding: 2px;
			font-size: 1.3vw;
			line-height: 2.5vw;
		}
		.ui.selectable.celled.teal.table.table.variants td, .ui.selectable.celled.teal.table.table.variants th{
			font-size: 1.5vw;
			padding: 3px;
			line-height: 2.3vw;
		}
		.ui.selectable.celled.teal.table.table.variants{
			margin-bottom: 1vw;
		}
		.ui.padded.table{
			font-size: 1.5vw;
			line-height: 2.3vw;.ui.selectable.celled.teal.table.table.cancer.card
		}
		.ui.selectable.celled.teal.table.table.cancer.cancer1 td{
			padding: 2px;
			font-size: 1.4vw;
		}
		.ui.celled.teal.table.table.drugs, .ui.selectable.celled.teal.table.table.cancer{
			width: 100%;
		}
		.ui.celled.teal.table.table.drugs{
			font-size: 1.4;
		}

		.ui.selectable.celled.teal.table.table.cancer.bigger th, .ui.selectable.celled.teal.table.table.cancer.bigger td{
			font-size: 1.5vw;
			line-height: 2.6vw;
		}
		.ui.selectable.celled.teal.table.table.cancer.cardio td{
			font-size: 1.3vw;
		}
		.acordiv{
			width:100%;
		}
		canvas{
        width: 34vw;
        height: 17vw;
    }
	}
</style>
<script>

function expand(obj){
	var tdEl = obj.children[0];
	var el = obj.nextSibling;
	if (el.nodeName == "#text")
		el = el.nextSibling;
	var parent = el.parentElement;
	parent = parent.parentElement.parentElement;
	board = parent.parentElement;
	el.style.display = "";
	el.style.maxHeight = el.scrollHeight + "px";
	if (board.classList.contains("basic-panel")){
		board.style.maxHeight= Number(board.style.maxHeight.slice(0, -2)) + el.scrollHeight + "px";
	}
	parent.style.maxHeight= Number(parent.style.maxHeight.slice(0, -2)) + el.scrollHeight + "px";
	tdEl.innerHTML = "<div class=\"ui tiny circular compact negative button\">-</div>";
	//obj.style.backgroundColor = "#e0e0e0";
}




function collapse(obj){
	var tdEl = obj.children[0];
	var el = obj.nextSibling;
	if (el.nodeName == "#text")
		el = el.nextSibling;
	var parent = el.parentElement;
	parent = parent.parentElement.parentElement;
	board = parent.parentElement;
	if (board.classList.contains("basic-panel")){
		board.style.maxHeight= Number(board.style.maxHeight.slice(0, -2)) - el.scrollHeight + "px";
	}
	parent.style.maxHeight= Number(parent.style.maxHeight.slice(0, -2)) - el.scrollHeigh + "px";
	el.style.maxHeight = '0';
	setTimeout(function() {
		el.style.display = "none";
	}, 150);
	tdEl.innerHTML = "<div class=\"ui tiny circular compact positive button\">+</div>";
	//obj.style.backgroundColor = "#eeeeee";
}

function doOnClick(obj){
	var tdEl = obj.children[0];
	var el = obj.nextSibling;
	if (el.nodeName == "#text")
		el = el.nextSibling;

	if(el.style.display == "none")
		expand(obj);
		 /*el.parentElement.maxHeight += el.clientHeight;*/
	else
		collapse(obj);
}

function collapseExpandAll(obj, id){
	var el = document.getElementById(id);
	if(obj.innerHTML == "<div class=\"ui tiny circular compact positive button\">+</div>"){
		for(var i in el.children)
			if(el.children[i].children && el.children[i].children.length > 3)
				expand(el.children[i]);
		obj.innerHTML = "<div class=\"ui tiny circular compact negative button\">-</div>";
	} else {
		for(var i in el.children)
			if(el.children[i].children && el.children[i].children.length > 3)
				collapse(el.children[i]);
		obj.innerHTML = "<div class=\"ui tiny circular compact positive button\">+</div>";
	}
}


var normalData = {'x': {'max': 2.82, 'min': -2.794, 'data': [-2.794, -2.569, -2.428, -2.323, -2.238, -2.167, -2.106, -2.052, -2.003, -1.958, -1.917, -1.879, -1.844, -1.811, -1.779, -1.75, -1.721, -1.694, -1.669, -1.644, -1.62, -1.597, -1.575, -1.554, -1.533, -1.513, -1.494, -1.475, -1.457, -1.439, -1.421, -1.404, -1.388, -1.372, -1.356, -1.34, -1.325, -1.31, -1.295, -1.281, -1.267, -1.253, -1.239, -1.226, -1.213, -1.2, -1.187, -1.174, -1.162, -1.15, -1.138, -1.126, -1.114, -1.103, -1.091, -1.08, -1.069, -1.058, -1.047, -1.036, -1.025, -1.015, -1.004, -0.994, -0.984, -0.974, -0.964, -0.954, -0.944, -0.934, -0.925, -0.915, -0.906, -0.896, -0.887, -0.878, -0.868, -0.859, -0.85, -0.841, -0.832, -0.824, -0.815, -0.806, -0.797, -0.789, -0.78, -0.772, -0.763, -0.755, -0.747, -0.739, -0.73, -0.722, -0.714, -0.706, -0.698, -0.69, -0.682, -0.674, -0.666, -0.659, -0.651, -0.643, -0.635, -0.628, -0.62, -0.613, -0.605, -0.597, -0.59, -0.583, -0.575, -0.568, -0.56, -0.553, -0.546, -0.539, -0.531, -0.524, -0.517, -0.51, -0.503, -0.496, -0.488, -0.481, -0.474, -0.467, -0.46, -0.453, -0.447, -0.44, -0.433, -0.426, -0.419, -0.412, -0.405, -0.399, -0.392, -0.385, -0.378, -0.372, -0.365, -0.358, -0.352, -0.345, -0.338, -0.332, -0.325, -0.318, -0.312, -0.305, -0.299, -0.292, -0.286, -0.279, -0.273, -0.266, -0.26, -0.253, -0.247, -0.24, -0.234, -0.227, -0.221, -0.214, -0.208, -0.202, -0.195, -0.189, -0.182, -0.176, -0.17, -0.163, -0.157, -0.151, -0.144, -0.138, -0.132, -0.125, -0.119, -0.113, -0.106, -0.1, -0.094, -0.088, -0.081, -0.075, -0.069, -0.062, -0.056, -0.05, -0.044, -0.037, -0.031, -0.025, -0.019, -0.012, -0.006, 0.0, 0.007, 0.013, 0.019, 0.025, 0.032, 0.038, 0.044, 0.05, 0.057, 0.063, 0.069, 0.076, 0.082, 0.088, 0.094, 0.101, 0.107, 0.113, 0.12, 0.126, 0.132, 0.139, 0.145, 0.151, 0.158, 0.164, 0.17, 0.177, 0.183, 0.189, 0.196, 0.202, 0.209, 0.215, 0.221, 0.228, 0.234, 0.241, 0.247, 0.254, 0.26, 0.267, 0.273, 0.28, 0.286, 0.293, 0.299, 0.306, 0.312, 0.319, 0.326, 0.332, 0.339, 0.345, 0.352, 0.359, 0.365, 0.372, 0.379, 0.386, 0.392, 0.399, 0.406, 0.413, 0.42, 0.426, 0.433, 0.44, 0.447, 0.454, 0.461, 0.468, 0.475, 0.482, 0.489, 0.496, 0.503, 0.51, 0.518, 0.525, 0.532, 0.539, 0.546, 0.554, 0.561, 0.568, 0.576, 0.583, 0.591, 0.598, 0.606, 0.613, 0.621, 0.628, 0.636, 0.644, 0.651, 0.659, 0.667, 0.675, 0.683, 0.691, 0.699, 0.707, 0.715, 0.723, 0.731, 0.739, 0.747, 0.756, 0.764, 0.773, 0.781, 0.79, 0.798, 0.807, 0.815, 0.824, 0.833, 0.842, 0.851, 0.86, 0.869, 0.878, 0.888, 0.897, 0.906, 0.916, 0.925, 0.935, 0.945, 0.955, 0.964, 0.975, 0.985, 0.995, 1.005, 1.016, 1.026, 1.037, 1.048, 1.059, 1.07, 1.081, 1.092, 1.104, 1.115, 1.127, 1.139, 1.151, 1.163, 1.175, 1.188, 1.201, 1.214, 1.227, 1.24, 1.254, 1.268, 1.282, 1.297, 1.311, 1.326, 1.341, 1.357, 1.373, 1.389, 1.406, 1.423, 1.44, 1.458, 1.477, 1.495, 1.515, 1.535, 1.556, 1.577, 1.599, 1.622, 1.646, 1.671, 1.696, 1.723, 1.752, 1.782, 1.813, 1.847, 1.882, 1.92, 1.962, 2.007, 2.056, 2.111, 2.173, 2.245, 2.33, 2.437, 2.583, 2.82]}, 'y': {'max': 0.399, 'min': 0.007, 'data': [0.008, 0.015, 0.021, 0.027, 0.033, 0.038, 0.043, 0.049, 0.054, 0.059, 0.063, 0.068, 0.073, 0.077, 0.082, 0.086, 0.091, 0.095, 0.099, 0.103, 0.107, 0.111, 0.115, 0.119, 0.123, 0.127, 0.131, 0.134, 0.138, 0.142, 0.145, 0.149, 0.152, 0.156, 0.159, 0.163, 0.166, 0.169, 0.172, 0.176, 0.179, 0.182, 0.185, 0.188, 0.191, 0.194, 0.197, 0.2, 0.203, 0.206, 0.209, 0.212, 0.214, 0.217, 0.22, 0.223, 0.225, 0.228, 0.231, 0.233, 0.236, 0.238, 0.241, 0.243, 0.246, 0.248, 0.251, 0.253, 0.256, 0.258, 0.26, 0.262, 0.265, 0.267, 0.269, 0.271, 0.274, 0.276, 0.278, 0.28, 0.282, 0.284, 0.286, 0.288, 0.29, 0.292, 0.294, 0.296, 0.298, 0.3, 0.302, 0.304, 0.306, 0.307, 0.309, 0.311, 0.313, 0.314, 0.316, 0.318, 0.32, 0.321, 0.323, 0.324, 0.326, 0.328, 0.329, 0.331, 0.332, 0.334, 0.335, 0.337, 0.338, 0.34, 0.341, 0.342, 0.344, 0.345, 0.346, 0.348, 0.349, 0.35, 0.352, 0.353, 0.354, 0.355, 0.356, 0.358, 0.359, 0.36, 0.361, 0.362, 0.363, 0.364, 0.365, 0.366, 0.367, 0.368, 0.369, 0.37, 0.371, 0.372, 0.373, 0.374, 0.375, 0.376, 0.377, 0.378, 0.378, 0.379, 0.38, 0.381, 0.382, 0.382, 0.383, 0.384, 0.384, 0.385, 0.386, 0.386, 0.387, 0.388, 0.388, 0.389, 0.389, 0.39, 0.39, 0.391, 0.391, 0.392, 0.392, 0.393, 0.393, 0.394, 0.394, 0.394, 0.395, 0.395, 0.395, 0.396, 0.396, 0.396, 0.397, 0.397, 0.397, 0.397, 0.398, 0.398, 0.398, 0.398, 0.398, 0.398, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.399, 0.398, 0.398, 0.398, 0.398, 0.398, 0.398, 0.397, 0.397, 0.397, 0.397, 0.396, 0.396, 0.396, 0.395, 0.395, 0.395, 0.394, 0.394, 0.394, 0.393, 0.393, 0.392, 0.392, 0.391, 0.391, 0.39, 0.39, 0.389, 0.389, 0.388, 0.388, 0.387, 0.386, 0.386, 0.385, 0.384, 0.384, 0.383, 0.382, 0.381, 0.381, 0.38, 0.379, 0.378, 0.378, 0.377, 0.376, 0.375, 0.374, 0.373, 0.372, 0.371, 0.37, 0.369, 0.368, 0.367, 0.366, 0.365, 0.364, 0.363, 0.362, 0.361, 0.36, 0.359, 0.358, 0.356, 0.355, 0.354, 0.353, 0.351, 0.35, 0.349, 0.348, 0.346, 0.345, 0.344, 0.342, 0.341, 0.339, 0.338, 0.337, 0.335, 0.334, 0.332, 0.331, 0.329, 0.327, 0.326, 0.324, 0.323, 0.321, 0.319, 0.318, 0.316, 0.314, 0.313, 0.311, 0.309, 0.307, 0.305, 0.304, 0.302, 0.3, 0.298, 0.296, 0.294, 0.292, 0.29, 0.288, 0.286, 0.284, 0.282, 0.28, 0.278, 0.276, 0.273, 0.271, 0.269, 0.267, 0.265, 0.262, 0.26, 0.258, 0.255, 0.253, 0.251, 0.248, 0.246, 0.243, 0.241, 0.238, 0.236, 0.233, 0.23, 0.228, 0.225, 0.222, 0.22, 0.217, 0.214, 0.211, 0.209, 0.206, 0.203, 0.2, 0.197, 0.194, 0.191, 0.188, 0.185, 0.182, 0.179, 0.175, 0.172, 0.169, 0.166, 0.162, 0.159, 0.155, 0.152, 0.149, 0.145, 0.141, 0.138, 0.134, 0.13, 0.127, 0.123, 0.119, 0.115, 0.111, 0.107, 0.103, 0.099, 0.095, 0.09, 0.086, 0.082, 0.077, 0.073, 0.068, 0.063, 0.058, 0.053, 0.048, 0.043, 0.038, 0.032, 0.026, 0.02, 0.014, 0.007]}, 'p': [0.003, 0.005, 0.008, 0.01, 0.013, 0.015, 0.018, 0.02, 0.023, 0.025, 0.028, 0.03, 0.033, 0.035, 0.038, 0.04, 0.043, 0.045, 0.048, 0.05, 0.053, 0.055, 0.058, 0.06, 0.063, 0.065, 0.068, 0.07, 0.073, 0.075, 0.078, 0.08, 0.083, 0.085, 0.088, 0.09, 0.093, 0.095, 0.098, 0.1, 0.103, 0.105, 0.108, 0.11, 0.113, 0.115, 0.118, 0.12, 0.123, 0.125, 0.128, 0.13, 0.133, 0.135, 0.138, 0.14, 0.143, 0.145, 0.148, 0.15, 0.153, 0.155, 0.158, 0.16, 0.163, 0.165, 0.168, 0.17, 0.173, 0.175, 0.178, 0.18, 0.183, 0.185, 0.188, 0.19, 0.193, 0.195, 0.198, 0.2, 0.203, 0.205, 0.208, 0.21, 0.213, 0.215, 0.218, 0.22, 0.223, 0.225, 0.228, 0.23, 0.233, 0.235, 0.238, 0.24, 0.243, 0.245, 0.248, 0.25, 0.253, 0.255, 0.258, 0.26, 0.263, 0.265, 0.268, 0.27, 0.273, 0.275, 0.278, 0.28, 0.283, 0.285, 0.288, 0.29, 0.293, 0.295, 0.298, 0.3, 0.303, 0.305, 0.308, 0.31, 0.313, 0.315, 0.318, 0.32, 0.323, 0.325, 0.328, 0.33, 0.333, 0.335, 0.338, 0.34, 0.343, 0.345, 0.348, 0.35, 0.353, 0.355, 0.358, 0.36, 0.363, 0.365, 0.368, 0.37, 0.373, 0.375, 0.378, 0.38, 0.383, 0.385, 0.388, 0.39, 0.393, 0.395, 0.398, 0.4, 0.403, 0.405, 0.408, 0.41, 0.413, 0.415, 0.418, 0.42, 0.423, 0.425, 0.428, 0.43, 0.433, 0.435, 0.438, 0.44, 0.443, 0.445, 0.448, 0.45, 0.453, 0.455, 0.458, 0.46, 0.463, 0.465, 0.468, 0.47, 0.473, 0.475, 0.478, 0.48, 0.483, 0.485, 0.488, 0.49, 0.493, 0.495, 0.498, 0.5, 0.503, 0.505, 0.508, 0.51, 0.513, 0.515, 0.518, 0.52, 0.523, 0.525, 0.528, 0.53, 0.533, 0.535, 0.538, 0.54, 0.543, 0.545, 0.548, 0.55, 0.553, 0.555, 0.558, 0.56, 0.563, 0.565, 0.568, 0.57, 0.573, 0.575, 0.578, 0.58, 0.583, 0.585, 0.588, 0.59, 0.593, 0.595, 0.598, 0.6, 0.603, 0.605, 0.608, 0.61, 0.613, 0.615, 0.618, 0.62, 0.623, 0.625, 0.628, 0.63, 0.633, 0.635, 0.638, 0.64, 0.643, 0.645, 0.648, 0.65, 0.653, 0.655, 0.658, 0.66, 0.663, 0.665, 0.668, 0.67, 0.673, 0.675, 0.678, 0.68, 0.683, 0.685, 0.688, 0.69, 0.693, 0.695, 0.698, 0.7, 0.703, 0.705, 0.708, 0.71, 0.713, 0.715, 0.718, 0.72, 0.723, 0.725, 0.728, 0.73, 0.733, 0.735, 0.738, 0.74, 0.743, 0.745, 0.748, 0.75, 0.753, 0.755, 0.758, 0.76, 0.763, 0.765, 0.768, 0.77, 0.773, 0.775, 0.778, 0.78, 0.783, 0.785, 0.788, 0.79, 0.793, 0.795, 0.798, 0.8, 0.803, 0.805, 0.808, 0.81, 0.813, 0.815, 0.818, 0.82, 0.823, 0.825, 0.828, 0.83, 0.833, 0.835, 0.838, 0.84, 0.843, 0.845, 0.848, 0.85, 0.853, 0.855, 0.858, 0.86, 0.863, 0.865, 0.868, 0.87, 0.873, 0.875, 0.878, 0.88, 0.883, 0.885, 0.888, 0.89, 0.893, 0.895, 0.898, 0.9, 0.903, 0.905, 0.908, 0.91, 0.913, 0.915, 0.918, 0.92, 0.923, 0.925, 0.928, 0.93, 0.933, 0.935, 0.938, 0.94, 0.943, 0.945, 0.948, 0.95, 0.953, 0.955, 0.958, 0.96, 0.963, 0.965, 0.968, 0.97, 0.973, 0.975, 0.978, 0.98, 0.983, 0.985, 0.988, 0.99, 0.993, 0.995, 0.998]}

function drawGausian(cnvasId, p, bgColor, gausianColor, percentileColor, filledColor, invers=0){

	if (invers == 1){
		if (p <= 0.25){
			var filledColor = '#ff9b00';
			var strokeColor = '#895510';
		} else if (p > 0.25 && p <= 0.45){
			var filledColor = '#ffdb33';
			var strokeColor = '#846925';
		} else {
			var filledColor = '#b4ea54';
			var strokeColor = '#097050';
		}
	}else{
		if (p <= 0.55){
			var filledColor = '#b4ea54';
			var strokeColor = '#097050';
		} else if (p > 0.55 && p <= 0.75){
			var filledColor = '#ffdb33';
			var strokeColor = '#846925';
		} else {
			var filledColor = '#ff9b00';
			var strokeColor = '#895510';
		}
	}

	var gausianStrokeColor = '#165863';

	var percColor = {0.05: '#146d40', 0.25: '#1faa5d', 0.5: '#aaaa5d', 0.75: '#b45b5d', 0.95: '#680000'};
	if (invers == 1)
		percColor = {0.05: '#680000', 0.25: '#b45b5d', 0.5: '#aaaa5d', 0.75:  '#1faa5d', 0.95: '#146d40'};


	var canvas = document.getElementById(cnvasId);
	//ctx.fillStyle = "blue";
	//ctx.fillRect(0, 0, , canvas.height);
	var width = canvas.width;
	var height = canvas.height;

	var xdata = normalData['x']['data'];
	var ydata = normalData['y']['data'];
	var paddingH = 50;
	var paddingT = 5;
	var bottomShift = 2;
	var xRatio = width / (normalData['x']['max'] - normalData['x']['min'])
	var yRatio = (height - paddingH) / (normalData['y']['max'] - normalData['y']['min'])

	var fillVal = 0;
	var pdata = normalData['p'];
	var pindex = -1;
	for (var i in pdata){
		if (pdata[i] >= p){
			fillVal = xdata[i]*xRatio + width/2;
			pindex = i;
			break;
		}
	}

	var ctx = canvas.getContext("2d");
	ctx.save();

	ctx.fillStyle=bgColor;
	ctx.fillRect(0,0,width,height);


	for (var i in pdata){
		if(pdata[i] == 0.05 || pdata[i] == 0.25 || pdata[i] == 0.5 || pdata[i] == 0.75 || pdata[i] == 0.95){
			ctx.strokeStyle = percColor[pdata[i]];
			ctx.fillStyle = percColor[pdata[i]];
			ctx.lineWidth = 2;
			ctx.beginPath();
			ctx.moveTo(xdata[i]*xRatio + width/2, 32);
			ctx.lineTo(xdata[i]*xRatio + width/2, height);
			ctx.stroke();
			ctx.font = "14px serif";
			ctx.fillText((pdata[i]*100) + "%", xdata[i]*xRatio + width/2 + paddingT - 15, paddingT*2 + 12);
		}
	}

	ctx.fillStyle=gausianColor;
	ctx.strokeStyle=gausianStrokeColor;

    ctx.beginPath();
	ctx.lineWidth = 2.5;
	ctx.moveTo(xdata[0]*xRatio + width/2, ydata[0]*-yRatio + height + bottomShift);
	for (var i in xdata){
		ctx.lineTo(xdata[i]*xRatio + width/2, ydata[i]*-yRatio + height + bottomShift);
	}
	ctx.lineTo(width,height);
	ctx.lineTo(0,height);
	ctx.closePath();
	ctx.fill();
	ctx.stroke();

	ctx.fillStyle = filledColor;
	ctx.strokeStyle = strokeColor;
    ctx.beginPath();
	ctx.lineWidth = 2.5;
	ctx.moveTo(xdata[0]*xRatio + width/2, ydata[0]*-yRatio + height + bottomShift);
	for (var i = 1; i <= pindex; i++){
		ctx.lineTo(xdata[i]*xRatio + width/2, ydata[i]*-yRatio + height + bottomShift);
	}
	--i;
	ctx.lineTo(xdata[i]*xRatio + width/2, height);
	ctx.lineTo(0,height);
	ctx.closePath();
	ctx.fill();
	ctx.stroke();

	ctx.fillStyle = strokeColor;
	ctx.font = "bold 20px serif";
	var textX = xdata[pindex]*xRatio + width/2 + paddingT - 50;
	if (textX < 0)
		textX = 0;
	ctx.fillText(Math.round(p*100) + "%", textX, height - paddingT*2 - 5);

    ctx.restore();
}
</script>
</head>

<body>
<div id="toTop">
	<section class="line arrow">
	</section>
	<section class="bodypart arrow">
	</section>
</div>
<h1 class="first" align="center">Longevity Report</h1>
<br/>
<div class="intro">
<p></p>
<p><This report provides data about your longevity genetics. Longevity and healthy lifespan are among the most complex phenotypes, with the heritability of age at death in adulthood being approximately 25 %. So, other 75 % is highly dependent on your lifestyle.
</p>
<p></p>
<p>This report is based on your raw file genome&rsquo;s data cross-referencing against available genetic databases and constructing a report based on interpretations in these databases. Variant annotations were made with 
	<a href="https://github.com/rkimoakbioinformatics/oakvar">OakVar</a>. Annotation databases that was used for report generating: cadd_exome, gnomAD, pubmed, clinpred, clinvar, ncbigene, omim, prec, provean, revel, sift, LongevityMap.
</p>
<p></p><div class="attention">
<p ><strong>Attention!</strong> The information provided is just for informational purposes. Its clinical implementation is possible just after consulting your healthcare practitioner. All drugs, vitamins and lifestyle changes may be prescribed just by your healthcare practitioner, based on clinical blood test results, family history etc. Genetical testing just provides additional information about possible health risk and risk management.
</p></div>
<p></p>
<p>The report consists of:</p>
<ul>
	<li><a href="#variants" class="li1 second_li">Longevity variants</a></li>
	<li><a href="#drugs" class="li1 third_li">Longevity drugs</a></li>
	<li><a href="#cancer" class="li1 forth_li">Major risks</a></li>
	<li><a href="#prs" class="li1 forth_li">All polygentic risk scores</a></li>
</ul>
<p></p>
</div>
<h2 id="variants"> Longevity variants</h2>
<button class="accordion accordion0 accordion1 panel-variants module-button"><div></div> Description</button>
<div class="board variants panel panel-description">
<p></p>
<p>This report estimates how many significant longevity variations you have and whether its &quot;a lot&quot; or &quot;not so&quot; for your population. The data for this report is obtained mainly by studying centenarians&rsquo; genomes.
</p>
<p></p>
<p>The longevity variants report is based on 1900 variants from LongevityMap (<a href="https://genomics.senescence.info/longevity/"> genomics.senescence.info/longevity </a>) and other data sources which are scored and prioritized according to multiple criteria. It also depends on ClinGene, dbSNP and ClinVar modules.</p>
<p></p>
<p>In the table you can see, how many significant for your population variants were found in your genome. Positively longevity-associated variants are marked with green, negatively associated - with red.
</p>
<p>We have analyzed genes that affect different longevity pathways in your genome. We have divided them according to the longevity pathways they contribute to. So you can see which of them “work well” and which do not so.</p>

<p>Longevity pathways encompass a fascinating realm of biological processes and signaling pathways that play a crucial role in regulating aging and lifespan. Recent advancements in our understanding of these pathways have yielded valuable insights, including the identification of specific genes that impact the aging process [PMID:<a href="https://pubmed.ncbi.nlm.nih.gov/33891896/">33891896</a>].</p>

<p>To help you better understand the role your genes play in longevity, we have categorized them into 11 distinct groups.</p>

<p> Our analysis has allowed us to identify which of your genes contribute to different longevity pathways, giving you a clearer picture of which genes are performing well and which ones may need additional attention.</p>
</div>
<button class="accordion accordion0 accordion1 prs-panel module-button"><div></div>Longevity Polygenic Risk Scores (PRSs)</button>
		<div class="board variants prs-panel panel">
			<p></p>
<p>There is data on <u>Polygenic risk scores (PRSs)</u> for gaining longevity.</p>
<p>PRSs aggregate the effect of many common genetic variants to estimate a person&rsquo;s chances of gaining extreme longevity. Each variant on its own tends little to the total outcome, but when added together, these differences can have a more significant impact.
</p>
<p></p>
<p>PRS results are highly dependent on the population. PPS is calculated as a weighted sum of trait-associated alleles. PRS gives an estimate of how likely an individual is to have a given trait
       <u>only based on genetics, without taking environmental factors into account</u>.</p>
<p></p>
<p class="how_to">How to read the result?</p>
<p></p>
<p>PRS is represented as a <u>percentile</u> within a given population. For example, you have the 95th percentile, which means your genetic chances to gain extreme longevity is higher than 95 out of every 100 people in a chosen population.
</p>
<p></p>
<p>So percentile &#65310;50 shows an increased genetic predisposition to longevity, &#65308;50 - decreased one.
</p>
<p></p>
<p></p>
<p></p>
<p class="how_to">PRS info</p>
<p>This PRS includes an analysis of 332 variants that significantly discriminated between centenarians and older adults.</p>
<p>PRS source: <a style="word-wrap: break-word;;" href="https://www.google.com/url?q=https://doi.org/10.1093/gerona/glaa289&amp;sa=D&amp;source=editors&amp;ust=1664211538645453&amp;usg=AOvVaw3xUB1M87YH6DdNAFu4Oipq">https://doi.org/10.1093/gerona/glaa289</a></p>
<p></p>

<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Longevity PRS (PRS5)</td><td rowspan="6"><canvas id="PRS5_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-0.24635944</td></tr>
    <tr><td>Count/total</td><td>156/330</td></tr>
    <tr><td>Average</td><td>-0.0007896135897435898</td></tr>
    <tr><td>Percentile</td><td>1%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PRS5_canvas", 0.01, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 1);
})
</script>
</div>
<button class="accordion accordion-heading">Genes involved in lipid transfer and lipid signaling</button >
<div class="basic-panel">
<button class="accordion accordion0 accordion1"><div></div> Description</button>
<div class="board variants panel panel-description">
	<p>Lipids play crucial roles in regulating aging and longevity. Lipids are key biological molecules that contribute to cellular and organismal functions in three principal ways. First, they are fundamental structural elements of cellular membranes. Second, they are key molecules in energy metabolism to fuel the cell. Third, they play roles by acting as signaling molecules. Lipid metabolism is not considered a separate longevity pathway, but genes that regulate lipid transfer, like APOE and CETP, show the strongest association with longevity. This is a list of genes we analyzed in your genotype related to lipid transfer: </p>
	</div>
<button class="accordion accordion0 accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longlipidsTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longlipidsTBody">
			
				
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs405509">rs405509</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APOE">APOE</a></td>
				<td>G/G</td><td>T</td><td>G</td><td>hom</td><td style="background-color:#ff8c8c">-0.3</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-286T>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.575188691314</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">The minor allele frequency of rs769449 in APOE was significantly decreased in the oldest-old. Haplotype case–control comparisons identified an haplotype with rs405509, rs440446 and rs769449 associated with longevity. Gene-based analysis confirmed the significant association of variations in APOE with longevity. The association of rs769449 was confirmed in the German population.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (APOE)</th><td>The protein encoded by this gene is a major apoprotein of the chylomicron. It binds to a specific liver and peripheral cell receptor, and is essential for the normal catabolism of triglyceride-rich lipoprotein constituents. This gene maps to chromosome 19 in a cluster with the related apolipoprotein C1 and C2 genes. Mutations in this gene result in familial dysbetalipoproteinemia, or type III hyperlipoproteinemia (HLP III), in which increased plasma cholesterol and triglycerides are the consequence of impaired clearance of chylomicron and VLDL remnants. [provided by RefSeq, Jun 2016]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7524519">rs7524519</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LYPLAL1">LYPLAL1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.245633048578</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9916344">rs9916344</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PITPNM3">PITPNM3</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.274+8634G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.508238600077</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PITPNM3)</th><td>This gene encodes a member of a family of membrane-associated phosphatidylinositol transfer domain-containing proteins. The calcium-binding protein has phosphatidylinositol (PI) transfer activity and interacts with the protein tyrosine kinase PTK2B (also known as PYK2). The protein is homologous to a Drosophila protein that is implicated in the visual transduction pathway in flies. Mutations in this gene result in autosomal dominant cone dystrophy. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Sep 2009]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs289714">rs289714</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CETP">CETP</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#8cff8c">0.3</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.930+29G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.757621061324</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">The minor allele frequency of rs9923854 in CETP was significantly associated with longevity. Haplotype case–control comparisons identified two haplotypes associated with longevity. Gene-based analysis confirmed the significant association of variations in CETP with longevity. The association of rs9923854 was borderline in the German population.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CETP)</th><td>The protein encoded by this gene is found in plasma, where it is involved in the transfer of cholesteryl ester from high density lipoprotein (HDL) to other lipoproteins. Defects in this gene are a cause of hyperalphalipoproteinemia 1 (HALP1). Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2013]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1800774">rs1800774</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CETP">CETP</a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#8cff8c">0.3</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1215-14C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.319643539417</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">The minor allele frequency of rs9923854 in CETP was significantly associated with longevity. Haplotype case–control comparisons identified two haplotypes associated with longevity. Gene-based analysis confirmed the significant association of variations in CETP with longevity. The association of rs9923854 was borderline in the German population.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CETP)</th><td>The protein encoded by this gene is found in plasma, where it is involved in the transfer of cholesteryl ester from high density lipoprotein (HDL) to other lipoproteins. Defects in this gene are a cause of hyperalphalipoproteinemia 1 (HALP1). Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2013]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs440446">rs440446</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APOE">APOE</a></td>
				<td>G/G</td><td>C</td><td>G</td><td>hom</td><td style="background-color:#8cff8c">0.3</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-24+69C>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.620829084772</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">The minor allele frequency of rs769449 in APOE was significantly decreased in the oldest-old. Haplotype case–control comparisons identified an haplotype with rs405509, rs440446 and rs769449 associated with longevity. Gene-based analysis confirmed the significant association of variations in APOE with longevity. The association of rs769449 was confirmed in the German population.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (APOE)</th><td>The protein encoded by this gene is a major apoprotein of the chylomicron. It binds to a specific liver and peripheral cell receptor, and is essential for the normal catabolism of triglyceride-rich lipoprotein constituents. This gene maps to chromosome 19 in a cluster with the related apolipoprotein C1 and C2 genes. Mutations in this gene result in familial dysbetalipoproteinemia, or type III hyperlipoproteinemia (HLP III), in which increased plasma cholesterol and triglycerides are the consequence of impaired clearance of chylomicron and VLDL remnants. [provided by RefSeq, Jun 2016]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>
	<button class="accordion0 accordion"><div></div>Recommendations</button>
<div class="board variants   panel">
	<p> From 6 gene variants involved in lipid metabolism and longevity pathways, you have <b>5</b> favorable and <b>1</b> unfavorable. </p>
	<br>
	<p>If your genetic testing reveals that you have unfavorable forms of genes involved in lipid and cholesterol transfer, there are several things you can do to manage your risk and improve your health: </p>
	<p><b>Monitor your health:</b> Regular check-ups and monitoring of your cholesterol levels, blood pressure, and other health parameters can help to identify any issues early and allow for prompt intervention. </p>
	<p><b>Biochemical markers for check-ups: </b>Triglycerides, Total cholesterol level, LDL and HDL cholesterol, and the atherogenicity index (AI). It is a good idea to monitor these markers on a regular basis, at least once per year.
 </p>
	<p><b>Adopt a healthy lifestyle:</b> it sounds not innovative, but getting regular exercise and maintaining a healthy weight can help to lower your risk of unfavorable lipid metabolism genes.
 </p>
	<p><b>Take medications if recommended:</b> Your healthcare professional may recommend medications such as statins, which can help to lower your cholesterol levels and reduce your risk of developing cardiovascular disease. Do not take any medicaments if they are not prescribed by a doctor. 
 </p>
	<p><b>Look at pharmacogenetics:</b> If your biochemical lipid profile is disturbed, consider that variations in these genes may be associated with differences in response to statin therapy.
 </p>
	</div>
	<div class="last inside"></div>
</div>
<button class="accordion accordion-heading">Genes involved in the insulin/IGF-1 signaling pathway
</button>
<div class="basic-panel">
<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>The insulin/insulin-like growth factor (IGF-1) signaling pathway is a key regulator of metabolism, growth, and aging. It has been extensively studied in various model organisms, including worms, flies, and mice, and is also thought to play an important role in human aging and longevity. This pathway is also involved in glucose metabolism. This is a list of genes we analyzed in your genotype related to insulin/IGF-1 signaling pathway: 
 </p>
	</div>
	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longinsulinTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longinsulinTBody">
			
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3762985">rs3762985</a></td><td>Korean</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PCSK1">PCSK1</a></td>
				<td>C/A</td><td>A</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-1981T>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.209226646248</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19641380</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">463 biomarkers in 176 diesase associated genes were examined in 137 long lived individuals (>90 y, 59.1% female, 96.73 ± 3.97 y) and 213 young controls (12-14 y, 39.4% female, 12.93 ± 0.80)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">PCSK1 was associated with longevity (P = 0.008) after correcting for multiple hypothesis testing</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PCSK1)</th><td>This gene encodes a member of the subtilisin-like proprotein convertase family, which includes proteases that process protein and peptide precursors trafficking through regulated or constitutive branches of the secretory pathway. The encoded protein undergoes an initial autocatalytic processing event in the ER to generate a heterodimer which exits the ER and sorts to subcellular compartments where a second autocatalytic even takes place and the catalytic activity is acquired. The protease is packaged into and activated in dense core secretory granules and expressed in the neuroendocrine system and brain. This gene encodes one of the seven basic amino acid-specific members which cleave their substrates at single or paired basic residues. It functions in the proteolytic activation of polypeptide hormones and neuropeptides precursors. Mutations in this gene have been associated with susceptibility to obesity and proprotein convertase 1/3 deficiency. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene [provided by RefSeq, Jan 2014]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9899404">rs9899404</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.618227072564</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs5758008">rs5758008</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=MKL1">MKL1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.275710809426</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (MKL1)</th><td>The protein encoded by this gene interacts with the transcription factor myocardin, a key regulator of smooth muscle cell differentiation. The encoded protein is predominantly nuclear and may help transduce signals from the cytoskeleton to the nucleus. This gene is involved in a specific translocation event that creates a fusion of this gene and the RNA-binding motif protein-15 gene. This translocation has been associated with acute megakaryocytic leukemia. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2013]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs572169">rs572169</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=GHSR">GHSR</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#d4ffd4">0.11</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.477G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.301744078789</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22406557</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Alleles in candidate pathways (GH/IGF1 signaling, DNA damage signaling and repair and pro/antioxidants) were investigated for association with longevity in 1089 oldest-old (age 92-93) and 736 middle-aged Danes</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Eleven SNPs (in GSR, KL, GHRHR, INS, GHSR, IGF2R, RAD52, WRN, RAD23B, POLB and NTLH1) were associated with longevity after correction for multiple hypothesis testing. No replications were observed in German and Dutch populations.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (GHSR)</th><td>This gene encodes a member of the G-protein coupled receptor family. The encoded protein may play a role in energy homeostasis and regulation of body weight. Two identified transcript variants are expressed in several tissues and are evolutionary conserved in fish and swine. One transcript, 1a, excises an intron and encodes the functional protein; this protein is the receptor for the Ghrelin ligand and defines a neuroendocrine pathway for growth hormone release. The second transcript (1b) retains the intron and does not function as a receptor for Ghrelin; however, it may function to attenuate activity of isoform 1a. Mutations in this gene are associated with autosomal idiopathic short stature.[provided by RefSeq, Apr 2010]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3842755">rs3842755</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=INS">INS</a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#d4ffd4">0.11</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-1768G>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.168081169038</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22406557</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Alleles in candidate pathways (GH/IGF1 signaling, DNA damage signaling and repair and pro/antioxidants) were investigated for association with longevity in 1089 oldest-old (age 92-93) and 736 middle-aged Danes</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Eleven SNPs (in GSR, KL, GHRHR, INS, GHSR, IGF2R, RAD52, WRN, RAD23B, POLB and NTLH1) were associated with longevity after correction for multiple hypothesis testing. No replications were observed in German and Dutch populations.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (INS)</th><td>This gene encodes a member of the insulin family of polypeptide growth factors, which are involved in development and growth. It is an imprinted gene, expressed only from the paternal allele, and epigenetic changes at this locus are associated with Wilms tumour, Beckwith-Wiedemann syndrome, rhabdomyosarcoma, and Silver-Russell syndrome. A read-through INS-IGF2 gene exists, whose 5&apos; region overlaps the INS gene and the 3&apos; region overlaps this gene. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2010]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs155979">rs155979</a></td><td>Korean</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PCSK1">PCSK1</a></td>
				<td>C/C</td><td>G</td><td>C</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-1152C>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.87862653829</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19641380</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">463 biomarkers in 176 diesase associated genes were examined in 137 long lived individuals (>90 y, 59.1% female, 96.73 ± 3.97 y) and 213 young controls (12-14 y, 39.4% female, 12.93 ± 0.80)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">PCSK1 was associated with longevity (P = 0.008) after correcting for multiple hypothesis testing</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PCSK1)</th><td>This gene encodes a member of the subtilisin-like proprotein convertase family, which includes proteases that process protein and peptide precursors trafficking through regulated or constitutive branches of the secretory pathway. The encoded protein undergoes an initial autocatalytic processing event in the ER to generate a heterodimer which exits the ER and sorts to subcellular compartments where a second autocatalytic even takes place and the catalytic activity is acquired. The protease is packaged into and activated in dense core secretory granules and expressed in the neuroendocrine system and brain. This gene encodes one of the seven basic amino acid-specific members which cleave their substrates at single or paired basic residues. It functions in the proteolytic activation of polypeptide hormones and neuropeptides precursors. Mutations in this gene have been associated with susceptibility to obesity and proprotein convertase 1/3 deficiency. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene [provided by RefSeq, Jan 2014]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs216493">rs216493</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PLEKHA7">PLEKHA7</a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.2007+629T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.831993620415</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PLEKHA7)</th><td>Enables delta-catenin binding activity. Involved in epithelial cell-cell adhesion; pore complex assembly; and zonula adherens maintenance. Located in several cellular components, including centrosome; nucleoplasm; and zonula adherens. Part of pore complex. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9456497">rs9456497</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=IGF2R">IGF2R</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#a8ffa8">0.225</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.514-2176A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.187328526766</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22406557</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Alleles in candidate pathways (GH/IGF1 signaling, DNA damage signaling and repair and pro/antioxidants) were investigated for association with longevity in 1089 oldest-old (age 92-93) and 736 middle-aged Danes</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Eleven SNPs (in GSR, KL, GHRHR, INS, GHSR, IGF2R, RAD52, WRN, RAD23B, POLB and NTLH1) were associated with longevity after correction for multiple hypothesis testing. No replications were observed in German and Dutch populations.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (IGF2R)</th><td>This gene encodes a receptor for both insulin-like growth factor 2 and mannose 6-phosphate. The binding sites for each ligand are located on different segments of the protein. This receptor has various functions, including in the intracellular trafficking of lysosomal enzymes, the activation of transforming growth factor beta, and the degradation of insulin-like growth factor 2. Mutation or loss of heterozygosity of this gene has been association with risk of hepatocellular carcinoma. The orthologous mouse gene is imprinted and shows exclusive expression from the maternal allele; however, imprinting of the human gene may be polymorphic, as only a minority of individuals showed biased expression from the maternal allele (PMID:8267611). [provided by RefSeq, Nov 2015]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs473268">rs473268</a></td><td>German </td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#79ff79">0.35</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.460768640491</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19196970</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">16 known FOXO3A SNPs were examined in 1,762 German centenarians/nonagenarians and younger controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Polymorphisms in FOXO3A were associated with longevity. The FOXO3A association was considerably stronger in centenarians than in nonagenarians, highlighting the importance of centenarians for genetic longevity research. Replication in 535 French centenarians (mean age: 103.8 years) and 553 younger controls (aged 18–70 years) generated a trend that supported the results, although it was not statistically significant.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7762395">rs7762395</a></td><td>multiple</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=FOXO3">FOXO3</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#54ff54">0.445</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.622-39551G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.170241286863</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>ID</th>
							<th>Association</th>
							<th>Population</th>
							<th>Identifier</th>
							<th>Gene</th>
							<th>Pubmed ID</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">2006</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">significant</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Danish</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs7762395</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">FOXO3</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20849522</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">insulin</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19196970</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">16 known FOXO3A SNPs were examined in 1,762 German centenarians/nonagenarians and younger controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Polymorphisms in FOXO3A were associated with longevity. The FOXO3A association was considerably stronger in centenarians than in nonagenarians, highlighting the importance of centenarians for genetic longevity research. Replication in 535 French centenarians (mean age: 103.8 years) and 553 younger controls (aged 18–70 years) generated a trend that supported the results, although it was not statistically significant.</th>
						</tr>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20849522</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">15 SNPs that covered FOXO3A common variation were examined in 1089 oldest old and 736 middle-aged for longevity association by longitudinal study design as well as case-control study design</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">4 SNPs previously reported to be associated with longevity were verified in this study. 4 novel SNPs (rs12206094, rs13220810, rs7762395, and rs9486902 (corrected P-values 0.001–0.044)) and two haplotypes(corrected P-values: 0.01–0.03) associated with longevity were found in this study. FOXO3A was a candidate longevity gene for survival from younger ages to old age, yet not during old age.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (FOXO3)</th><td>This gene belongs to the forkhead family of transcription factors which are characterized by a distinct forkhead domain. This gene likely functions as a trigger for apoptosis through expression of genes necessary for cell death. Translocation of this gene with the MLL gene is associated with secondary acute leukemia. Alternatively spliced transcript variants encoding the same protein have been observed. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6911407">rs6911407</a></td><td>German </td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/A</td><td>C</td><td>A</td><td>hom</td><td style="background-color:#3fff3f">0.5</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.468622318693</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19196970</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">16 known FOXO3A SNPs were examined in 1,762 German centenarians/nonagenarians and younger controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Polymorphisms in FOXO3A were associated with longevity. The FOXO3A association was considerably stronger in centenarians than in nonagenarians, highlighting the importance of centenarians for genetic longevity research. Replication in 535 French centenarians (mean age: 103.8 years) and 553 younger controls (aged 18–70 years) generated a trend that supported the results, although it was not statistically significant.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs479744">rs479744</a></td><td>multiple</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=multiple">multiple</a></td>
				<td>T/T</td><td>G</td><td>T</td><td>hom</td><td style="background-color:#00ff00">0.79</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.318117932665</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>ID</th>
							<th>Association</th>
							<th>Population</th>
							<th>Identifier</th>
							<th>Gene</th>
							<th>Pubmed ID</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">1992</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">significant</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Danish</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs479744</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">FOXO3</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20849522</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">insulin</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19196970</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">16 known FOXO3A SNPs were examined in 1,762 German centenarians/nonagenarians and younger controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Polymorphisms in FOXO3A were associated with longevity. The FOXO3A association was considerably stronger in centenarians than in nonagenarians, highlighting the importance of centenarians for genetic longevity research. Replication in 535 French centenarians (mean age: 103.8 years) and 553 younger controls (aged 18–70 years) generated a trend that supported the results, although it was not statistically significant.</th>
						</tr>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20849522</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">15 SNPs that covered FOXO3A common variation were examined in 1089 oldest old and 736 middle-aged for longevity association by longitudinal study design as well as case-control study design</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">4 SNPs previously reported to be associated with longevity were verified in this study. 4 novel SNPs (rs12206094, rs13220810, rs7762395, and rs9486902 (corrected P-values 0.001–0.044)) and two haplotypes(corrected P-values: 0.01–0.03) associated with longevity were found in this study. FOXO3A was a candidate longevity gene for survival from younger ages to old age, yet not during old age.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

<button class=" accordion0 accordion"><div></div>Recommendations</button>
<div class="board variants   panel">
	<p> From  12 gene variants involved in insulin/IGF-1 signaling pathways, you have <b>12</b> favorable and <b>0</b> unfavorable. </p>
	<br>
	<p>If you have unfavorable variants of  insulin/IGF-1 signaling pathway genes, there are several lifestyle changes you can make to help promote healthy aging and reduce the risk of age-related diseases:
 </p>
	<p><b>Diet:</b> A healthy, balanced diet that is low in sugar and processed foods can help to regulate insulin and IGF-1 levels, which may help to mitigate the effects of unfavorable genes. 
 </p>
	<p><b>Exercise:</b> Regular exercise has been shown to improve insulin sensitivity and promote healthy aging. Aim for at least 30 minutes of moderate-intensity exercise most days of the week, such as brisk walking, cycling, or swimming.
 </p>
	<p><b>Stress management:</b> Chronic stress can contribute to insulin resistance and age-related diseases. Incorporating stress-management techniques such as meditation, yoga, or deep breathing into your daily routine can help to reduce stress and promote healthy aging.
 </p>
	<p><b>Sleep:</b> Poor sleep has been linked to insulin resistance and other age-related diseases. Aim for at least 7-8 hours of sleep each night and establish a regular sleep routine to support healthy aging.
 </p>
	<p><b>Biochemical markers for check-ups:</b> insulin, fasting glucose level, Hemoglobin A1c (HbA1c), and C-peptide.  It is a good idea to monitor these markers on a regular basis, at least once per year.
 </p>
	</div>
	<div class="last inside"></div>
</div>

	
	<button class="accordion accordion-heading"  >Genes involved in antioxidant defense</button>
	<div class="basic-panel">
<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p> Antioxidant defense plays an important role in the aging process and longevity. Oxidative stress, which is caused by an imbalance between the production of reactive oxygen species (ROS) and the body's ability to neutralize them, is a major contributor to age-related diseases and the aging process.</p>
	<p>
The body has various mechanisms to defend against oxidative stress, including antioxidant enzymes such as superoxide dismutase (SOD), catalase, and glutathione peroxidase, as well as non-enzymatic antioxidants such as vitamins C and E, and glutathione. These antioxidants work together to neutralize ROS and prevent damage to cells and tissues.
</p>
	<p>Studies have shown that increased antioxidant defense can promote longevity and delay the onset of age-related diseases. This is a list of genes we analyzed in your genotype related to antioxidant defense: </p>
	</div>


<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longantioxidantTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longantioxidantTBody">
			
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7310815">rs7310815</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>G/C</td><td>C</td><td>G</td><td>het</td><td style="background-color:#d6ffd6">0.105</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.91+4177C>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.192236539934</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4964735">rs4964735</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#d6ffd6">0.105</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.304+4269G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.371068780363</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7301631">rs7301631</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#d6ffd6">0.105</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.305-9447T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.530190610209</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs17202060">rs17202060</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#d6ffd6">0.105</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1747-2071C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.290957378728</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7962759">rs7962759</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>G/C</td><td>C</td><td>G</td><td>het</td><td style="background-color:#d6ffd6">0.105</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1882-3115C>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.150736936132</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7318601">rs7318601</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ENOX1">ENOX1</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1036+1209C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.842905405405</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ENOX1)</th><td>The protein encoded by this gene is involved in plasma membrane electron transport pathways. The encoded protein has both a hydroquinone (NADH) oxidase activity and a protein disulfide-thiol interchange activity. The two activities cycle with a periodicity of 24 minutes, with one activity being at its peak when the other is at its lowest. [provided by RefSeq, Dec 2016]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10861169">rs10861169</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#aeffae">0.21</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.91+969C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.707852267653</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4964728">rs4964728</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#aeffae">0.21</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.244-2064G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.796053051074</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7310505">rs7310505</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>C/C</td><td>A</td><td>C</td><td>hom</td><td style="background-color:#aeffae">0.21</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.304+2691A>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.741557612512</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10778318">rs10778318</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TXNRD1">TXNRD1</a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#aeffae">0.21</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.305-11203A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.690747330961</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TXNRD1)</th><td>The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes an ubiquitously expressed, cytosolic form of TrxR, which functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternative splicing, primarily at the 5&apos; end, results in transcript variants encoding same or different isoforms, including a glutaredoxin-containing isoform that is predominantly expressed in testis. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

<button class="accordion0 accordion"><div></div>Recommendations</button>
<div class="board variants   panel">
	<p> From 10 gene variants involved in antioxidant defense, you have <b>10</b> favorable and <b>0</b> unfavorable. </p>
	<p>If you have unfavorable variants of  antioxidant system genes, there are several lifestyle changes you can make to help promote healthy aging and reduce the risk of age-related diseases:
</p>
	<p><b>Ensure the uptake of all components needed for your antioxidant system works appropriately: </b></p>
	<p><u>Glutathione</u> (which is an important component of your defense) is composed of the amino acids glutamate, cysteine, and glycine. Its’ synthesis requires magnesium, zinc, selenium, and vitamins B2, B6, and B12 as cofactors. Ensure you take all these components from food. NAC supplementation can be used to boost glutathione synthesis.</p>
	<p><u>Flavonoids:</u> They are a group of plant compounds that have antioxidant and anti-inflammatory properties. Good food sources of flavonoids include berries, apples, citrus fruits, and tea.
</p>
	<p><u>Selenium:</u> It is a mineral that is important for the production of antioxidant enzymes in the body. Good food sources of selenium include Brazil nuts, seafood, and organ meats.
</p>
	<p>Polyphenols are a group of naturally occurring compounds found in plant-based foods, including fruits, vegetables, nuts, seeds, and grains. They are known to have antioxidant properties, which means they can help protect the body against damage caused by free radicals.
</p>
	<p><u>Polyphenols</u> work by neutralizing free radicals and preventing them from causing oxidative damage to cells and tissues. They can also boost the activity of the body's own antioxidant enzymes, including glutathione peroxidase and superoxide dismutase.Some of the most commonly studied polyphenols include resveratrol, found in grapes and red wine, catechins, found in green tea, and quercetin, found in onions, apples, and berries. Other sources of polyphenols include cocoa, dark chocolate, coffee, and many types of fruits and vegetables.
</p>
	<p><b>Biochemical markers for a check-up:</b> oxidized low-density lipoprotein (oxLDL), Fe/Cu ratio, glutathione, 8-hydroxy-2’-deoxyguanosine (8-OHdG) (a marker of oxidative damage to DNA), malondialdehyde (MDA) (it is a byproduct of lipid peroxidation and is used as a marker of oxidative stress). It is a good idea to monitor these markers on a regular basis, at least once per year.
</p>
	</div>
	<div class="last inside"></div>
</div>
	
	<button class="accordion accordion-heading"  >Genes related to mitochondria function</button>
	<div class="basic-panel">
<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>Mitochondria are the powerhouses of the cell, generating the energy (in the form of ATP) needed for cellular processes. They also play a key role in apoptosis (programmed cell death) and other cellular processes. Mitochondrial dysfunction and increased oxidative stress are believed to play a role in the aging process and age-related diseases. Several genes involved in mitochondrial function have been implicated in longevity. They include UCP genes, genes involved in respiratory chain functioning, SIRT3, PGC1a, and some other genes. 
 </p>
 <p>In general, the form of these genes determines how well you are protected from oxidative stress and how effectively your cells generate energy. This is a list of genes we analyzed in your genotype related to mitochondria function: </p>
	</div>

<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longmitoTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longmitoTBody">
			
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6435326">rs6435326</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=NDUFS1">NDUFS1</a></td>
				<td>T/T</td><td>A</td><td>T</td><td>hom</td><td style="background-color:#ffb6b6">-0.19</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1392+650T>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.535661881978</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (NDUFS1)</th><td>The protein encoded by this gene belongs to the complex I 75 kDa subunit family. Mammalian complex I is composed of 45 different subunits. It locates at the mitochondrial inner membrane. This protein has NADH dehydrogenase activity and oxidoreductase activity. It transfers electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone. This protein is the largest subunit of complex I and it is a component of the iron-sulfur (IP) fragment of the enzyme. It may form part of the active site crevice where NADH is oxidized. Mutations in this gene are associated with complex I deficiency. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2011]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9557276">rs9557276</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CLYBL">CLYBL</a></td>
				<td>T/G</td><td>G</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.62+58077G>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.464685919191</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CLYBL)</th><td>Enables (S)-citramalyl-CoA lyase activity; magnesium ion binding activity; and malate synthase activity. Involved in protein homotrimerization and regulation of cobalamin metabolic process. Predicted to be located in mitochondrion. Predicted to be integral component of membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1042718">rs1042718</a></td><td>Chinese</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ADRB2">ADRB2</a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#d9ffd9">0.098</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.523C>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.229937976931</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">23020224</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A 10-year follow-up study of the Chinese longitudinal healthy longevity survey was performed consisting of 963 long-lived and 1028 geography-matched young individuals. Sixteen SNPs from ADRB1, ADRB2, ADCY5, ADCY6, and MAPK1 were selected and genotyped. </th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Two SNPs, rs1042718 (C/A) and rs1042719 (G/C), of ADRB2 in linkage disequilibrium were found to be associated with enhanced longevity in men in two geographically isolated populations. Men with haplotype A-C showed an increased probability to become centenarians , in contrast to those with haplotype C-G.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ADRB2)</th><td>This gene encodes beta-2-adrenergic receptor which is a member of the G protein-coupled receptor superfamily. This receptor is directly associated with one of its ultimate effectors, the class C L-type calcium channel Ca(V)1.2. This receptor-channel complex also contains a G protein, an adenylyl cyclase, cAMP-dependent kinase, and the counterbalancing phosphatase, PP2A. The assembly of the signaling complex provides a mechanism that ensures specific and rapid signaling by this G protein-coupled receptor. This receptor is also a transcription regulator of the alpha-synuclein gene, and together, both genes are believed to be associated with risk of Parkinson&apos;s Disease. This gene is intronless. Different polymorphic forms, point mutations, and/or downregulation of this gene are associated with nocturnal asthma, obesity, type 2 diabetes and cardiovascular disease. [provided by RefSeq, Oct 2019]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1042719">rs1042719</a></td><td>Chinese</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ADRB2">ADRB2</a></td>
				<td>C/G</td><td>G</td><td>C</td><td>het</td><td style="background-color:#d9ffd9">0.098</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1053G>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.334156454306</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">23020224</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A 10-year follow-up study of the Chinese longitudinal healthy longevity survey was performed consisting of 963 long-lived and 1028 geography-matched young individuals. Sixteen SNPs from ADRB1, ADRB2, ADCY5, ADCY6, and MAPK1 were selected and genotyped. </th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Two SNPs, rs1042718 (C/A) and rs1042719 (G/C), of ADRB2 in linkage disequilibrium were found to be associated with enhanced longevity in men in two geographically isolated populations. Men with haplotype A-C showed an increased probability to become centenarians , in contrast to those with haplotype C-G.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ADRB2)</th><td>This gene encodes beta-2-adrenergic receptor which is a member of the G protein-coupled receptor superfamily. This receptor is directly associated with one of its ultimate effectors, the class C L-type calcium channel Ca(V)1.2. This receptor-channel complex also contains a G protein, an adenylyl cyclase, cAMP-dependent kinase, and the counterbalancing phosphatase, PP2A. The assembly of the signaling complex provides a mechanism that ensures specific and rapid signaling by this G protein-coupled receptor. This receptor is also a transcription regulator of the alpha-synuclein gene, and together, both genes are believed to be associated with risk of Parkinson&apos;s Disease. This gene is intronless. Different polymorphic forms, point mutations, and/or downregulation of this gene are associated with nocturnal asthma, obesity, type 2 diabetes and cardiovascular disease. [provided by RefSeq, Oct 2019]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3019435">rs3019435</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PRKN">PRKN</a></td>
				<td>T/T</td><td>G</td><td>T</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.735-2679C>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.814891581633</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PRKN)</th><td>The precise function of this gene is unknown; however, the encoded protein is a component of a multiprotein E3 ubiquitin ligase complex that mediates the targeting of substrate proteins for proteasomal degradation. Mutations in this gene are known to cause Parkinson disease and autosomal recessive juvenile Parkinson disease. Alternative splicing of this gene produces multiple transcript variants encoding distinct isoforms. Additional splice variants of this gene have been described but currently lack transcript support. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11548670">rs11548670</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=NDUFS1">NDUFS1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#b6ffb6">0.19</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.414T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.0496733227741</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (NDUFS1)</th><td>The protein encoded by this gene belongs to the complex I 75 kDa subunit family. Mammalian complex I is composed of 45 different subunits. It locates at the mitochondrial inner membrane. This protein has NADH dehydrogenase activity and oxidoreductase activity. It transfers electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone. This protein is the largest subunit of complex I and it is a component of the iron-sulfur (IP) fragment of the enzyme. It may form part of the active site crevice where NADH is oxidized. Mutations in this gene are associated with complex I deficiency. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2011]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs15763">rs15763</a></td><td>Italian (Southern)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=UCP3">UCP3</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#a1ffa1">0.245</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.*980T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.707359773753</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22216339</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 598 subjects (293 men and 305 women, age range 64–105 years; mean ages 82.74 and 85.23 years, respectively) were analyzed for genetic variability in Uncoupling Proteins (UCPs). The set was divided into two age classes. The separation threshold for these two classes was 88 years for men and 91 years for women.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">The genetic variability of UCP2, UCP3 and UCP4 affects the individual’s chances of surviving up to a very old age. Using the minor allele for each SNP as reference and after adjusting for BMI and sex, the dominant model for the rs660339 (UCP2) and the rs1800849 (UCP3) resulted to be significantly associated with the longevity phenotype (p = 0.001 in both cases), while the recessive model was the most likely for SNPs rs15763 (UCP3), rs9472817 (UCP4) and rs2235800 (UCP5) (p < 0.05). After adjusting for multiple testing, all the previous associations remained statistically significant, except those for rs2235800 of the UCP5 gene (p= 0.058).</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (UCP3)</th><td>Mitochondrial uncoupling proteins (UCP) are members of the larger family of mitochondrial anion carrier proteins (MACP). UCPs separate oxidative phosphorylation from ATP synthesis with energy dissipated as heat, also referred to as the mitochondrial proton leak. UCPs facilitate the transfer of anions from the inner to the outer mitochondrial membrane and the return transfer of protons from the outer to the inner mitochondrial membrane. They also reduce the mitochondrial membrane potential in mammalian cells. The different UCPs have tissue-specific expression; this gene is primarily expressed in skeletal muscle. This gene&apos;s protein product is postulated to protect mitochondria against lipid-induced oxidative stress. Expression levels of this gene increase when fatty acid supplies to mitochondria exceed their oxidation capacity and the protein enables the export of fatty acids from mitochondria. UCPs contain the three solcar protein domains typically found in MACPs. Two splice variants have been found for this gene.[provided by RefSeq, Nov 2008]</td></tr>
				</table>
			</td></tr>
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6435324">rs6435324</a></td><td>Danish</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=NDUFS1">NDUFS1</a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#6dff6d">0.38</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1708+963T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.520315594455</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24462499</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">38 genes (311 SNPs) belonging to pro-antioxidant pathways were investigated for the association with physical and cognitive performances in a Cohort of 1089 Danish nonagenarians. For each gene analyzed in the pro-antioxidant pathway, the influence on longitudinal survival was tested.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">NDUFS1, TXNRD1, SOD2 and UCP3 were found significantly associated with lifespan in the female cohort. This result is consistent with their associations with physical functioning and suggests that the variability of genes in the pro-antioxidant pathway can influence survival through an effect on physical performances, at least in the analyzed cohort.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (NDUFS1)</th><td>The protein encoded by this gene belongs to the complex I 75 kDa subunit family. Mammalian complex I is composed of 45 different subunits. It locates at the mitochondrial inner membrane. This protein has NADH dehydrogenase activity and oxidoreductase activity. It transfers electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone. This protein is the largest subunit of complex I and it is a component of the iron-sulfur (IP) fragment of the enzyme. It may form part of the active site crevice where NADH is oxidized. Mutations in this gene are associated with complex I deficiency. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2011]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion"><div></div>Recommendations</button>
<div class="board variants   panel">
	<p>From 8 gene variants involved in mitochondrial function, you have <b>7</b> favorable and <b>1</b> unfavorable. </p>
	<br>
	<p>If you have unfavorable variants in genes involved in mitochondrial function, there are several steps you can take to improve your health outcomes potentially:</p>
	<br>
	<p><b>Biochemical markers for check-ups:</b> lactate, pyruvate, and creatine kinase, markers of oxidative stress, such as glutathione level and Fe/Cu ratio.</p>
	<p><b>Consider adequate uptake of trace elements and vitamins essential for mitochondria function:</b> magnesium, B vitamins (particularly thiamine (B1), riboflavin (B2), and niacin (B3)), selenium, iron, copper, and vitamin E.</p>
	<p><b>Exercise:</b> Regular exercise has been shown to increase mitochondrial biogenesis (the process of creating new mitochondria) and improve mitochondrial function. This is thought to be due to increased energy demand during exercise, which triggers the body to produce more mitochondria.</p>
	</div>
	<div class="last inside"></div>

</div>
	<button class="accordion accordion-heading"  >Genes related to the sirtuin pathway</button>
	<div class="basic-panel">
	<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p> The sirtuin genes (SIRT1-SIRT7) are a family of genes that are involved in regulating cellular processes such as DNA repair, metabolism, and stress response. The sirtuin pathway, which involves the activity of these genes, has been implicated in regulating longevity.
</p>
	<p>Studies have shown that activation of sirtuins can increase lifespan in several model organisms, including yeast, worms, and mice. This is thought to be due to the role of sirtuins in promoting cellular resilience and protecting cells from damage.
</p>
	<p>In humans, variations in the SIRT genes have been associated with age-related diseases such as Alzheimer's disease, cardiovascular disease, and cancer. Some studies have also suggested that increased activity of SIRT genes may be associated with increased lifespan and improved healthspan in humans.
</p>
	<p>This is a list of genes we analyzed in your genotype related to sirtuin pathway: 
</p>
	</div>

	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longsirtuinTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longsirtuinTBody">
			
						
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs107251">rs107251</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SIRT6">SIRT6</a></td>
				<td>C/C</td><td>T</td><td>C</td><td>hom</td><td style="background-color:#8cff8c">0.3</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.438-151A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.892813836237</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Gene-based analysis showed a significant association of variations in SIRT6 with longevity, though this seemed to reflect the association of a single SNP (rs107251)</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (SIRT6)</th><td>This gene encodes a member of the sirtuin family of NAD-dependent enzymes that are implicated in cellular stress resistance, genomic stability, aging and energy homeostasis. The encoded protein is localized to the nucleus, exhibits ADP-ribosyl transferase and histone deacetylase activities, and plays a role in DNA repair, maintenance of telomeric chromatin, inflammation, lipid and glucose metabolism. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Mar 2016]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion "><div></div>Recommendations</button>
<div class="board variants   panel">
	<p> From 1 gene variants involved in the sirtuin pathway, you have <b>1</b> favorable and <b>0</b> unfavorable. </p>
	<br>
	<p>There are several ways to activate the sirtuin pathway if your genes are not favorable:</p>
	<br>
	<p><b>Exercise:</b> Exercise has been shown to increase sirtuin activity in multiple tissues, including skeletal muscle, liver, and brain. Both endurance and resistance exercise may be effective in activating the sirtuin pathway.</p>
	<p><b>Supplementation:</b> Supplementation with compounds such as resveratrol and nicotinamide riboside has been shown to increase sirtuin activity in multiple tissues. These compounds are thought to work by mimicking the effects of caloric restriction on sirtuin activity. <u>Consider that in human trials effect of these compounds has not yet been shown. So before taking any supplements, consult your healthcare practitioner.</u> 
</p>
	<p><b>Intermittent fasting:</b> Intermittent fasting is a dietary strategy that involves cycling between periods of fasting and non-fasting. Intermittent fasting has been shown to increase sirtuin activity in multiple tissues and may activate the sirtuin pathway.
</p>
	</div>
	<div class="last inside"></div>
</div>

	
	<button class="accordion accordion-heading"  >Genes related to mTOR pathway</button>
	<div class="basic-panel">
<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>
MTOR, also known as the mechanistic target of rapamycin, is a gene that encodes for a protein kinase involved in multiple cellular processes, including growth, metabolism, and aging. The mTOR pathway plays a complex role in aging and longevity. On the one hand, activation of the mTOR pathway has been shown to promote cellular growth and proliferation, which may be beneficial for tissue repair and regeneration during early life stages. On the other hand, chronic activation of the mTOR pathway has been implicated in age-related diseases such as cancer, metabolic disorders, and neurodegeneration.
 </p>
	<p>Studies have shown that genetic or pharmacological inhibition of the mTOR pathway can extend the lifespan in a variety of model organisms, including mice, flies, and worms. Additionally, modulation of the mTOR pathway has been shown to improve age-related functional decline and delay the onset of age-related diseases in various animal models.
</p>
	<p>
Genetic variants in the MTOR gene may be associated with differences in lifespan and age-related disease risk. 
</p>
	</div>

	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longmTORTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longmTORTBody">
			
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4729049">rs4729049</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDK6">CDK6</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.370-22076A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.126115786789</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CDK6)</th><td>The protein encoded by this gene is a member of the CMGC family of serine/threonine protein kinases. This kinase is a catalytic subunit of the protein kinase complex that is important for cell cycle G1 phase progression and G1/S transition. The activity of this kinase first appears in mid-G1 phase, which is controlled by the regulatory subunits including D-type cyclins and members of INK4 family of CDK inhibitors. This kinase, as well as CDK4, has been shown to phosphorylate, and thus regulate the activity of, tumor suppressor protein Rb. Altered expression of this gene has been observed in multiple human cancers. A mutation in this gene resulting in reduced cell proliferation, and impaired cell motility and polarity, and has been identified in patients with primary microcephaly. [provided by RefSeq, Aug 2017]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion "><div></div>Recommendations</button>
<div class="board variants   panel">
	<p> From 1 gene variants involved in mTOR pathway, you have <b>1</b> favorable and <b>0</b> unfavorable. </p>

	<br>
	<p>There are several ways to influence mTOR pathway if your genes are not favorable:
</p>
	<br>
	<p><b>Fasting:</b> Intermittent fasting or periodic fasting has been shown to decrease mTOR activity.
</p>
	<p><b>Use spices:</b> Curcumin is a compound found in turmeric, which has been shown to inhibit mTOR activity.
</p>
	<p><b>Plant-based diet:</b> A diet rich in fruits, vegetables, whole grains, and legumes has been associated with decreased mTOR activity.
</p>
	</div>

	<div class="last inside"></div>
</div>
	<button class="accordion accordion-heading"  >Tumor-suppressor genes</button>
	<div class="basic-panel">
	<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>Tumor suppressor genes and cell cycle regulators are not typically considered as "longevity genes" per se, but they do play an important role in the aging process and the development of age-related diseases, including cancer.
 </p>
	<p>One well-known tumor suppressor gene is TP53, which has been shown to play a role in regulating cellular senescence and preventing the accumulation of damaged cells. 
Similarly, cell cycle regulators such as cyclin-dependent kinases (CDKs) are involved in regulating the timing and progression of cell division. The dysregulation of CDKs has been implicated in a variety of age-related diseases, including cancer and neurodegenerative disorders.
 </p>
	</div>


	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longTumorTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longTumorTBody">
			
				
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4977756">rs4977756</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDKN2B-AS1">CDKN2B-AS1</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#ffbaba">-0.18</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.635035098915</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CDKN2B-AS1)</th><td>This gene is located within the CDKN2B-CDKN2A gene cluster at chromosome 9p21. The gene product is a functional RNA molecule that interacts with polycomb repressive complex-1 (PRC1) and -2 (PRC2), leading to epigenetic silencing of other genes in this cluster. This region is a significant genetic susceptibility locus for cardiovascular disease, and has also been linked to a number of other pathologies, including several cancers, intracranial aneurysm, type-2 diabetes, periodontitis, Alzheimer&apos;s disease, endometriosis, frailty in the elderly, and glaucoma. Multiple alternatively processed transcript variants have been detected, some of which may take the form of circular RNA molecules (PMID:21151960). [provided by RefSeq, May 2014]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1063192">rs1063192</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDKN2B">CDKN2B</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#baffba">0.18</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.*2619C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.688862604693</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CDKN2B)</th><td>This gene lies adjacent to the tumor suppressor gene CDKN2A in a region that is frequently mutated and deleted in a wide variety of tumors. This gene encodes a cyclin-dependent kinase inhibitor, which forms a complex with CDK4 or CDK6, and prevents the activation of the CDK kinases, thus the encoded protein functions as a cell growth regulator that controls cell cycle G1 progression. The expression of this gene was found to be dramatically induced by TGF beta, which suggested its role in the TGF beta induced growth inhibition. Two alternatively spliced transcript variants of this gene, which encode distinct proteins, have been reported. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1412832">rs1412832</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDKN2B-AS1">CDKN2B-AS1</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#baffba">0.18</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.753685150916</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CDKN2B-AS1)</th><td>This gene is located within the CDKN2B-CDKN2A gene cluster at chromosome 9p21. The gene product is a functional RNA molecule that interacts with polycomb repressive complex-1 (PRC1) and -2 (PRC2), leading to epigenetic silencing of other genes in this cluster. This region is a significant genetic susceptibility locus for cardiovascular disease, and has also been linked to a number of other pathologies, including several cancers, intracranial aneurysm, type-2 diabetes, periodontitis, Alzheimer&apos;s disease, endometriosis, frailty in the elderly, and glaucoma. Multiple alternatively processed transcript variants have been detected, some of which may take the form of circular RNA molecules (PMID:21151960). [provided by RefSeq, May 2014]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2738679">rs2738679</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=WWOX">WWOX</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#b4ffb4">0.195</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1056+38349T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.426490277335</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (WWOX)</th><td>This gene encodes a member of the short-chain dehydrogenases/reductases (SDR) protein family. This gene spans the FRA16D common chromosomal fragile site and appears to function as a tumor suppressor gene. Expression of the encoded protein is able to induce apoptosis, while defects in this gene are associated with multiple types of cancer. Disruption of this gene is also associated with autosomal recessive spinocerebellar ataxia 12. Disruption of a similar gene in mouse results in impaired steroidogenesis, additionally suggesting a metabolic function for the protein. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9616906">rs9616906</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TP53">TP53</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#a3ffa3">0.24</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.374928082849</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs189037">rs189037</a></td><td>multiple</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ATM">ATM</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#80ff80">0.33</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-111G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.486967354501</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>ID</th>
							<th>Association</th>
							<th>Population</th>
							<th>Identifier</th>
							<th>Gene</th>
							<th>Pubmed ID</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">357</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">significant</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Italian</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs189037</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">ATM</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22960875</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">tumor-suppressor</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20816691</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">SNP rs189037 in the promoter region of ATM gene was studied in 789 long-long lived individuals and 886 controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A significant association between CT genotype and longevity was observed. The CT genotype showed higher frequency in long-lived individuals than in controls (53.0% versus 45.9%, P = 0.004).</th>
						</tr>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22960875</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs189037 C/T genotype was examined in 128 Italian long-lived individuals (78.1 % females, 98.7 ± 5.1 years; 52.3 % > 100 years) and 150 controls  (56.7 % females, 67.2 ± 9.1 years) to study the association with longevity</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">There was association between the rs189037 C/T genotype and longevity in Italian centenarians, with an odds ratio of 1.85 (95 % CI 0.99-3.45)</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ATM)</th><td>The protein encoded by this gene belongs to the PI3/PI4-kinase family. This protein is an important cell cycle checkpoint kinase that phosphorylates; thus, it functions as a regulator of a wide variety of downstream proteins, including tumor suppressor proteins p53 and BRCA1, checkpoint kinase CHK2, checkpoint proteins RAD17 and RAD9, and DNA repair protein NBS1. This protein and the closely related kinase ATR are thought to be master controllers of cell cycle checkpoint signaling pathways that are required for cell response to DNA damage and for genome stability. Mutations in this gene are associated with ataxia telangiectasia, an autosomal recessive disorder. [provided by RefSeq, Aug 2010]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs13008689">rs13008689</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TP53">TP53</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#47ff47">0.48</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.355504440611</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion "><div></div>Recommendations</button>
<div class="board variants   panel">
	<p>From 7 gene variants of tumor-suppressive genes, you have <b>6</b> favorable and <b>1</b> unfavorable <b>regarding their impact on longevity</b>*.  </p>
<br>
<p>If you have unfavorable variants in genes involved in genome maintenance, there are several steps you can take to improve your health outcomes potentially:
</p>
<br>
	<p><b>Ensure methylation processes in your body work well:</b> monitor blood levels of B12, B9, B2, B6, Zn, and Co[PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/31601260/">31601260</a>, PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/36203899/">36203899</a>]. These components are needed for methylation enzymes' appropriate work. Methylation is important for genome maintenance because it is involved in the regulation of gene expression and DNA repair [PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/34209979/">34209979</a>].</p>
	<p><b>Reduce exposure to environmental toxins</b>, such as pollution, pesticides, and chemicals. Air-control indoor systems can help with this point.</p>
	<p><b>Protect the skin</b> from UV radiation by wearing protective clothing and using sunscreen.</p>
	<p><b>Avoid tobacco:</b> Tobacco use is a leading cause of cancer, so avoiding tobacco in all forms, including smoking, vaping, and chewing, can significantly decrease the risk of cancer.</p>
	<p><i>*In our analysis, we have categorized tumor-suppressive gene variants based solely on their effect on longevity as either favorable or unfavorable. We would like to clarify that our report does not provide an assessment of whether these variants have an increased risk for cancer or not.</i></p>
	</div>
	<div class="last inside"></div>
</div>



	
	<button class="accordion accordion-heading"  >Genes of the renin-angiotensin system</button>
	<div class="basic-panel">
	<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>The renin-angiotensin system (RAS) plays an important role in the regulation of blood pressure, fluid balance, and electrolyte homeostasis. However, it is also involved in the aging process and the development of age-related diseases. </p>
	<p>One of the mechanisms by which RAS influences aging is through the activation of oxidative stress and inflammation. RAS also affects the cardiovascular system and is involved in the development of hypertension, a major risk factor for cardiovascular diseases such as heart attack and stroke. Furthermore, RAS activation has been linked to the development of insulin resistance and metabolic syndrome, which are also associated with aging and age-related diseases.
</p>
	<p>In addition, RAS has been implicated in the regulation of cellular senescence, a process by which cells lose their ability to divide and contribute to aging and age-related diseases. 
</p>
	</div>


	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longreninTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longreninTBody">
			
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs275653">rs275653</a></td><td>Italian (Northern) and Japanese</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=AGTR1">AGTR1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#a8ffa8">0.225</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-501A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.188410892355</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22569962</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">AGTR1 was sequenced in 173 Italian centenarians (99-106 y, mean 100.9 ± 1.7 y, 83% female) and 376 younger controls (49-78 y, mean 65.9 ± 9.1 y, 43 % females). The results were then verified in 589 Japanese cases (99-115 y, mean 104.5 ± 3 y, 84% femal) and 422 controls (69-72 y, mean 70.1 ± 0.9 y, 52 % female).</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs422858, rs275653 were significantly associated with extreme longevity (P = 0.004). Combining the Italian and Japanese cohorts, rs275653 was associated to extreme longevity either at recessive model (P = 0.007, odds ratio (OR) 3.57) or at genotype level (P = 0.015).</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (AGTR1)</th><td>Angiotensin II is a potent vasopressor hormone and a primary regulator of aldosterone secretion. It is an important effector controlling blood pressure and volume in the cardiovascular system. It acts through at least two types of receptors. This gene encodes the type 1 receptor which is thought to mediate the major cardiovascular effects of angiotensin II. This gene may play a role in the generation of reperfusion arrhythmias following restoration of blood flow to ischemic or infarcted myocardium. It was previously thought that a related gene, denoted as AGTR1B, existed; however, it is now believed that there is only one type 1 receptor gene in humans. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Aug 2020]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion "><div></div>Recommendations</button>
<div class="board variants panel">
	<p>From 1 gene variants of genes related to the renin-angiotensin system, you have <b>1</b> favorable and <b>0</b> unfavorable regarding their impact on longevity. 
 </p>
 <br>
	<p>There are several ways to activate the sirtuin pathway if your genes are not favorable:
</p>
<br>
	<p><b>Control your blood pressure level
</b></p>
	<p><b>Look at pharmacogenetics:</b> you may have a low response to angiotensin-converting enzyme (ACE) inhibitors, depending on the variants carrying,  so talk to your healthcare practitioner. 
</p>
	<p><b>Diet:</b> A healthy diet can help prevent hypertension and improve overall cardiovascular health. Here are some dietary tips to follow: limit processed and high-fat foods, as well as sugary drinks and snacks; reduce your salt intake by using herbs and spices to flavor your food instead of salt, and avoid high-sodium packaged and processed foods; incorporate calcium-rich foods such as low-fat dairy products, tofu, and leafy greens into your diet.
</p>
	</div>
	<div class="last inside"></div>
	</div>


	
	<button class="accordion accordion-heading"  >Heat-shock protein genes</button>
	<div class="basic-panel">
	<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>The HSP (heat shock protein) genes are a group of genes that encode heat shock proteins, which are a class of chaperone proteins that help to protect cells from stress-induced damage. HSPs have been shown to play a role in a variety of cellular processes, including protein folding, DNA repair, and apoptosis. While the exact mechanisms by which HSPA genes influence aging and longevity are not yet fully understood, it is believed that they may help to protect cells from the accumulation of damage caused by stress and other environmental factors. As such, HSPs and their associated genes may be a promising target for interventions aimed at promoting healthy aging and extending lifespan. Some unfavorable variants in HSP genes have been shown to reduce the ability of cells to respond to stress, leading to increased damage and inflammation.
 </p>
	</div>

	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longHeatTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longHeatTBody">
			
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9787671">rs9787671</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HSPA14">HSPA14</a></td>
				<td>G/C</td><td>C</td><td>G</td><td>het</td><td style="background-color:#c5ffc5">0.15</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.993+3987C>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.194613268505</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Gene-based analysis showed a significant association of variations in HSPA14 with longevity</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (HSPA14)</th><td>Predicted to enable several functions, including ATP binding activity; misfolded protein binding activity; and unfolded protein binding activity. Predicted to be involved in several processes, including cellular response to unfolded protein; chaperone cofactor-dependent protein refolding; and protein refolding. Located in cytosol. Colocalizes with ribosome. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7905174">rs7905174</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HSPA14">HSPA14</a></td>
				<td>T/A</td><td>A</td><td>T</td><td>het</td><td style="background-color:#c1ffc1">0.16</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.994-841A>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.399553001277</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Gene-based analysis showed a significant association of variations in HSPA14 with longevity</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (HSPA14)</th><td>Predicted to enable several functions, including ATP binding activity; misfolded protein binding activity; and unfolded protein binding activity. Predicted to be involved in several processes, including cellular response to unfolded protein; chaperone cofactor-dependent protein refolding; and protein refolding. Located in cytosol. Colocalizes with ribosome. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs17268499">rs17268499</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HSPA14">HSPA14</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#bfffbf">0.165</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.994-511C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.118405103668</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Gene-based analysis showed a significant association of variations in HSPA14 with longevity</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (HSPA14)</th><td>Predicted to enable several functions, including ATP binding activity; misfolded protein binding activity; and unfolded protein binding activity. Predicted to be involved in several processes, including cellular response to unfolded protein; chaperone cofactor-dependent protein refolding; and protein refolding. Located in cytosol. Colocalizes with ribosome. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12770830">rs12770830</a></td><td>Danish, German, Dutch</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HSPA14">HSPA14</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#aaffaa">0.22</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.221+2353G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.0753025282934</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22234866</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">102 SNPs from 16 longevity candidate genes were examined in Danish. 1089 individuals (ages 92.2-93.8, mean age 93.2, 71.3 female) and 736 middle-aged controls (46-55 y, mean age 50.6, 49.6% female) were involved in this case-control study. Then the results were replicated in a German cohort of 1613 individuals (95-110 y, 73.2% female) and 1104 middle-aged controls (mean age 67.2, SD 4.07, 74.3% female). A 11 years study was introduced in Danish cohort to identify the SNPs associated with longevity, then the results were verified in Dutch longitudinal cohort.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Gene-based analysis showed a significant association of variations in HSPA14 with longevity</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (HSPA14)</th><td>Predicted to enable several functions, including ATP binding activity; misfolded protein binding activity; and unfolded protein binding activity. Predicted to be involved in several processes, including cellular response to unfolded protein; chaperone cofactor-dependent protein refolding; and protein refolding. Located in cytosol. Colocalizes with ribosome. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion "><div></div>Recommendations</button>
<div class="board variants   panel">
	<p> From 4 gene variants related to heat-shock proteins, you have <b>4</b> favorable and <b>0</b> unfavorable. 
</p><br>
<p>If you have unfavorable heat shock protein (HSP) gene variants, there are several things you can do to reduce cell stress:
</p><br>
<p><b>Environmental factors:</b> Avoiding exposure to environmental toxins, such as cigarette smoke, pollution, and pesticides, can help reduce cellular stress and inflammation.
</p>
<p><b>Sleep:</b> Getting adequate sleep is important for reducing cellular stress and promoting cellular repair and regeneration.
</p>
	</div>
	<div class="last inside"></div>
</div>

	
	<button class="accordion accordion-heading"  >Inflammation and related pathways</button>
	<div class="basic-panel">
	<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p> Chronic inflammation is a major contributor to the aging process. Inflammation is a natural response of the immune system to harmful stimuli, such as pathogens or tissue damage. However, if this response becomes chronic, it can lead to tissue damage and the development of age-related diseases such as Alzheimer's disease, cardiovascular disease, and cancer.
</p>
<p>Chronic inflammation is characterized by the sustained activation of the immune system and the release of pro-inflammatory molecules, such as cytokines, chemokines, and reactive oxygen species (ROS). These molecules can damage cellular components, including DNA, proteins, and lipids, leading to cellular dysfunction and death.
</p>
<p>In addition, chronic inflammation can also activate other pathways involved in aging, such as the mTOR pathway and the senescence-associated secretory phenotype (SASP), which further exacerbate inflammation and tissue damage. Therefore, reducing chronic inflammation is a promising strategy to promote healthy aging and prevent age-related diseases.
</p>
<p>Chronic inflammation can have a genetic component, with certain genes being associated with increased inflammation levels.
</p>
	</div>


	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longInflamTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longInflamTBody">
			
				
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs507879">rs507879</a></td><td>Russian</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CASP5">CASP5</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#ffe2e2">-0.075</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.316A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.439117161658</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20434535</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">510 subjects were analyzed for the genotype distribution of CASP5 Ala90Thr polymorphism.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Heterozygosity for CASP5 Ala90Thr polymorphism was negatively associated with life expectancy.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CASP5)</th><td>This gene encodes a member of the cysteine-aspartic acid protease (caspase) family. Sequential activation of caspases plays a central role in the execution-phase of cell apoptosis. Caspases exist as inactive proenzymes which undergo proteolytic processing at conserved aspartic residues to produce two subunits, large and small, that dimerize to form the active enzyme. Overexpression of the active form of this enzyme induces apoptosis in fibroblasts. Max, a central component of the Myc/Max/Mad transcription regulation network important for cell growth, differentiation, and apoptosis, is cleaved by this protein; this process requires Fas-mediated dephosphorylation of Max. The expression of this gene is regulated by interferon-gamma and lipopolysaccharide. Alternatively spliced transcript variants have been identified for this gene. [provided by RefSeq, Aug 2010]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3804474">rs3804474</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LY86">LY86</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.137-3651C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.530360107266</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LY86)</th><td>Acts upstream of or within positive regulation of lipopolysaccharide-mediated signaling pathway. Predicted to be located in extracellular space. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7013313">rs7013313</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CSMD3">CSMD3</a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1859+13337G>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.301847589822</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CSMD3)</th><td>Predicted to be involved in regulation of dendrite development. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1450741">rs1450741</a></td><td>Korean</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LYN">LYN</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#dcffdc">0.09</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-509T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.693521541515</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19641380</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">463 biomarkers in 176 diesase associated genes were examined in 137 long lived individuals (>90 y, 59.1% female, 96.73 ± 3.97 y) and 213 young controls (12-14 y, 39.4% female, 12.93 ± 0.80)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">LYN was associated with longevity (P = 0.002) after correcting for multiple hypothesis testing</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LYN)</th><td>This gene encodes a tyrosine protein kinase, which maybe involved in the regulation of mast cell degranulation, and erythroid differentiation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1980042">rs1980042</a></td><td>Korean</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LYN">LYN</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#dcffdc">0.09</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-6+19980A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.419678202018</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19641380</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">463 biomarkers in 176 diesase associated genes were examined in 137 long lived individuals (>90 y, 59.1% female, 96.73 ± 3.97 y) and 213 young controls (12-14 y, 39.4% female, 12.93 ± 0.80)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">LYN was associated with longevity (P = 0.002) after correcting for multiple hypothesis testing</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LYN)</th><td>This gene encodes a tyrosine protein kinase, which maybe involved in the regulation of mast cell degranulation, and erythroid differentiation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1979128">rs1979128</a></td><td>Korean</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LYN">LYN</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#dcffdc">0.09</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-6+21348A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.420131570544</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19641380</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">463 biomarkers in 176 diesase associated genes were examined in 137 long lived individuals (>90 y, 59.1% female, 96.73 ± 3.97 y) and 213 young controls (12-14 y, 39.4% female, 12.93 ± 0.80)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">LYN was associated with longevity (P = 0.002) after correcting for multiple hypothesis testing</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LYN)</th><td>This gene encodes a tyrosine protein kinase, which maybe involved in the regulation of mast cell degranulation, and erythroid differentiation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1546518">rs1546518</a></td><td>Korean</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LYN">LYN</a></td>
				<td>C/G</td><td>G</td><td>C</td><td>het</td><td style="background-color:#dcffdc">0.09</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-5-29520G>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.666166517686</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19641380</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">463 biomarkers in 176 diesase associated genes were examined in 137 long lived individuals (>90 y, 59.1% female, 96.73 ± 3.97 y) and 213 young controls (12-14 y, 39.4% female, 12.93 ± 0.80)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">LYN was associated with longevity (P = 0.002) after correcting for multiple hypothesis testing</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LYN)</th><td>This gene encodes a tyrosine protein kinase, which maybe involved in the regulation of mast cell degranulation, and erythroid differentiation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3968371">rs3968371</a></td><td>Korean</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LYN">LYN</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#dcffdc">0.09</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-5-4279A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.292929615213</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">19641380</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">463 biomarkers in 176 diesase associated genes were examined in 137 long lived individuals (>90 y, 59.1% female, 96.73 ± 3.97 y) and 213 young controls (12-14 y, 39.4% female, 12.93 ± 0.80)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">LYN was associated with longevity (P = 0.002) after correcting for multiple hypothesis testing</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LYN)</th><td>This gene encodes a tyrosine protein kinase, which maybe involved in the regulation of mast cell degranulation, and erythroid differentiation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2251252">rs2251252</a></td><td>Italian (Southern)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SDC4">SDC4</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#dcffdc">0.09</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.60+5608C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.434739533397</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">26254886</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">11 SNPs within the SDC4 gene were tested for association with longevity in a cohort of 624 unrelated subjects (aged 64-107, 287 males and 337 females, 350 subjects > 85 years old). Then the identified longevity-associated SNPs were correlated with metabolic parameters in the subgroup of 64-85 years old.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs1981429 was negatively associated with longevity (adjusted p = 0.028), high levels of triglycerides (adjusted p = 0.028) and low levels of LDL-C (adjusted p = 0.009). rs2251252 was positively correlated with longevity (adjusted p = 0.018) and high LDL-C (adjusted p = 0.022).</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (SDC4)</th><td>The protein encoded by this gene is a transmembrane (type I) heparan sulfate proteoglycan that functions as a receptor in intracellular signaling. The encoded protein is found as a homodimer and is a member of the syndecan proteoglycan family. This gene is found on chromosome 20, while a pseudogene has been found on chromosome 22. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1981429">rs1981429</a></td><td>Italian (Southern)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SDC4">SDC4</a></td>
				<td>T/G</td><td>G</td><td>T</td><td>het</td><td style="background-color:#dcffdc">0.09</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.60+1272C>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.527770684372</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">26254886</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">11 SNPs within the SDC4 gene were tested for association with longevity in a cohort of 624 unrelated subjects (aged 64-107, 287 males and 337 females, 350 subjects > 85 years old). Then the identified longevity-associated SNPs were correlated with metabolic parameters in the subgroup of 64-85 years old.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs1981429 was negatively associated with longevity (adjusted p = 0.028), high levels of triglycerides (adjusted p = 0.028) and low levels of LDL-C (adjusted p = 0.009). rs2251252 was positively correlated with longevity (adjusted p = 0.018) and high LDL-C (adjusted p = 0.022).</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (SDC4)</th><td>The protein encoded by this gene is a transmembrane (type I) heparan sulfate proteoglycan that functions as a receptor in intracellular signaling. The encoded protein is found as a homodimer and is a member of the syndecan proteoglycan family. This gene is found on chromosome 20, while a pseudogene has been found on chromosome 22. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs282070">rs282070</a></td><td>Italian (Southern)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=MAP3K7">MAP3K7</a></td>
				<td>G/C</td><td>C</td><td>G</td><td>het</td><td style="background-color:#cbffcb">0.135</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.120+63G>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.677975314958</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22576335</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A two-stage case-control study was performed to identify the association between longevity and variation of in homeostasis regulation pathway genes. 317 SNPs in 104 genes were analyzed in 78 cases (≥90 years, median age 98 years, 42 females) and 71 controls (<90 years, median age 67 years, 32 females) in stage 1. Then, 31 candidate SNPs identified in stage 1 (π markers = 0.1) were analyzed in an independent sample composed by 288 cases (≥90 years, median age 92 years, 163 females) and 554 controls (<90 years, median age 67 years, 277 females).</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">After adjustment for multiple testing, rs282070 was significantly associated with longevity (P  = 1.1 × 10−3). The C allele carriers showed an increased chance to attain longevity</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (MAP3K7)</th><td>The protein encoded by this gene is a member of the serine/threonine protein kinase family. This kinase mediates the signaling transduction induced by TGF beta and morphogenetic protein (BMP), and controls a variety of cell functions including transcription regulation and apoptosis. In response to IL-1, this protein forms a kinase complex including TRAF6, MAP3K7P1/TAB1 and MAP3K7P2/TAB2; this complex is required for the activation of nuclear factor kappa B. This kinase can also activate MAPK8/JNK, MAP2K4/MKK4, and thus plays a role in the cell response to environmental stresses. Four alternatively spliced transcript variants encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs205499">rs205499</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TRIM25">TRIM25</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1363+303C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.178947704489</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TRIM25)</th><td>The protein encoded by this gene is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. The protein is an RNA binding protein, functions as a ubiquitin E3 ligase and is involved in multiple cellular processes, including regulation of antiviral innate immunity. [provided by RefSeq, Sep 2021]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion "><div></div>Recommendations</button>
<div class="board variants   panel">
	<p>From 12 gene variants related to inflammation, you have <b>11</b> favorable and <b>1</b> unfavorable.  </p>
	<br>
	<p>If you have unfavorable variants in genes involved in chronic inflammation, there are several steps you can take to improve your health outcomes potentially:
</p>
	<br>
	<p><b>Biochemical markers for check-ups:</b> C-reactive protein (CRP), ferritin, tumor necrosis factor-alpha (TNF-alpha).
</p>
	<p><b>Diet:</b> Consuming a balanced diet that is rich in anti-inflammatory foods such as fruits, vegetables, whole grains, and fatty fish can help reduce inflammation. Some herbs and spices have an antiinflammatory action so that they can be added to the diet: garlic, ginger, turmeric, cinnamon, and oregano.
</p>
	<p><b>Dentist check-ups:</b> regular dental check-ups and good oral hygiene practices can help prevent chronic inflammation in the gums and other oral tissues. Periodontal disease, which is caused by bacteria and leads to inflammation and tissue destruction in the gums, is a common cause of chronic inflammation.
</p>
<p><b>Air cleaning:</b> air pollutants are a common source of chronic inflammation. Control your air quality indoors, and use filter systems if needed.

	</p>
	</div>
	<div class="last inside"></div>
</div>
	
	<button class="accordion accordion-heading"  >Genome maintenance and post-transcriptional processes
</button>
<div class="basic-panel">
<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>Genome maintenance and post-transcriptional processes are both important for maintaining cellular and organismal homeostasis, and both have been implicated in the regulation of longevity.  </p>
	<p>Genome maintenance is essential for preventing DNA damage and mutations that can lead to age-related diseases and shortened lifespans. Post-transcriptional processes, including RNA splicing, translation, and decay, are important for regulating gene expression and ensuring that proteins are produced at the appropriate levels and times. Dysregulation of these processes can lead to age-related diseases and shortened lifespans.
		</p>
	</div>
	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longGenomeTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longGenomeTBody">
			
				
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2485662">rs2485662</a></td><td>American (Caucasians), Italian (Southern), French, Ashkenazi Jewish </td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LMNA">LMNA</a></td>
				<td>C/C</td><td>T</td><td>C</td><td>hom</td><td style="background-color:#ff9797">-0.27</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-1242T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.65033286391</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22340368</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">After 16 SNPs over LMNA gene were genotyped in 873 long-lived individuals and 443 controls for association with longevity, a haplotype was identified. Then, the result was tested in 3619 subjects from four independent samples (738 New England Centenarian Study samples, 905 Southern Italian Centenarian Study samples, 1103 France samples and 702 Einstein Ashkenazi Longevity Study). </th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A meta-analysis combining results from 5 sets of samples showed the haplotype based on 4 SNPs (rs915179, rs2485662, rs4641, rs1468772) from LMNA was significantly associated with longevity (P=7.5 × 10(-4), multiple-testing-adjusted P=0.037)</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LMNA)</th><td>The protein encoded by this gene is part of the nuclear lamina, a two-dimensional matrix of proteins located next to the inner nuclear membrane. The lamin family of proteins make up the matrix and are highly conserved in evolution. During mitosis, the lamina matrix is reversibly disassembled as the lamin proteins are phosphorylated. Lamin proteins are thought to be involved in nuclear stability, chromatin structure and gene expression. Vertebrate lamins consist of two types, A and B. Alternative splicing results in multiple transcript variants. Mutations in this gene lead to several diseases: Emery-Dreifuss muscular dystrophy, familial partial lipodystrophy, limb girdle muscular dystrophy, dilated cardiomyopathy, Charcot-Marie-Tooth disease, and Hutchinson-Gilford progeria syndrome. [provided by RefSeq, May 2022]</td></tr>
				</table>
			</td></tr>
				
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4641">rs4641</a></td><td>American (Caucasians), Italian (Southern), French, Ashkenazi Jewish </td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LMNA">LMNA</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#ffcbcb">-0.135</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1698C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.251489380073</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22340368</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">After 16 SNPs over LMNA gene were genotyped in 873 long-lived individuals and 443 controls for association with longevity, a haplotype was identified. Then, the result was tested in 3619 subjects from four independent samples (738 New England Centenarian Study samples, 905 Southern Italian Centenarian Study samples, 1103 France samples and 702 Einstein Ashkenazi Longevity Study). </th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A meta-analysis combining results from 5 sets of samples showed the haplotype based on 4 SNPs (rs915179, rs2485662, rs4641, rs1468772) from LMNA was significantly associated with longevity (P=7.5 × 10(-4), multiple-testing-adjusted P=0.037)</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LMNA)</th><td>The protein encoded by this gene is part of the nuclear lamina, a two-dimensional matrix of proteins located next to the inner nuclear membrane. The lamin family of proteins make up the matrix and are highly conserved in evolution. During mitosis, the lamina matrix is reversibly disassembled as the lamin proteins are phosphorylated. Lamin proteins are thought to be involved in nuclear stability, chromatin structure and gene expression. Vertebrate lamins consist of two types, A and B. Alternative splicing results in multiple transcript variants. Mutations in this gene lead to several diseases: Emery-Dreifuss muscular dystrophy, familial partial lipodystrophy, limb girdle muscular dystrophy, dilated cardiomyopathy, Charcot-Marie-Tooth disease, and Hutchinson-Gilford progeria syndrome. [provided by RefSeq, May 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9397084">rs9397084</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE1">SYNE1</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.24129+1237A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.204432397959</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (SYNE1)</th><td>This gene encodes a spectrin repeat containing protein expressed in skeletal and smooth muscle, and peripheral blood lymphocytes, that localizes to the nuclear membrane. Mutations in this gene have been associated with autosomal recessive spinocerebellar ataxia 8, also referred to as autosomal recessive cerebellar ataxia type 1 or recessive ataxia of Beauce. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3817530">rs3817530</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=UBE2H">UBE2H</a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.53+5119A>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.129039468642</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (UBE2H)</th><td>The modification of proteins with ubiquitin is an important cellular mechanism for targeting abnormal or short-lived proteins for degradation. Ubiquitination involves at least three classes of enzymes: ubiquitin-activating enzymes, or E1s, ubiquitin-conjugating enzymes, or E2s, and ubiquitin-protein ligases, or E3s. This gene encodes a member of the E2 ubiquitin-conjugating enzyme family. The encoded protein sequence is 100% identical to the mouse homolog and 98% identical to the frog and zebrafish homologs. Three alternatively spliced transcript variants have been found for this gene and they encode distinct isoforms. [provided by RefSeq, Feb 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4785393">rs4785393</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PAPD5">PAPD5</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1800+287A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.181774656878</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PAPD5)</th><td>Enables guanylyltransferase activity and polynucleotide adenylyltransferase activity. Involved in several processes, including RNA metabolic process; negative regulation of telomere maintenance via telomerase; and regulation of mRNA stability. Located in cytoplasm and nucleolus. Part of TRAMP complex. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1800392">rs1800392</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=WRN">WRN</a></td>
				<td>T/G</td><td>G</td><td>T</td><td>het</td><td style="background-color:#e1ffe1">0.078</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.2361G>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.451206734207</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (WRN)</th><td>This gene encodes a member of the RecQ subfamily of DNA helicase proteins. The encoded nuclear protein is important in the maintenance of genome stability and plays a role in DNA repair, replication, transcription and telomere maintenance. This protein contains a N-terminal 3&apos; to 5&apos; exonuclease domain, an ATP-dependent helicase domain and RQC (RecQ helicase conserved region) domain in its central region, and a C-terminal HRDC (helicase RNase D C-terminal) domain and nuclear localization signal. Defects in this gene are the cause of Werner syndrome, an autosomal recessive disorder characterized by accelerated aging and an elevated risk for certain cancers. [provided by RefSeq, Aug 2017]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9876781">rs9876781</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ATRIP">ATRIP</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e0ffe0">0.08</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-912A>G</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ATRIP)</th><td>This gene encodes an essential component of the DNA damage checkpoint. The encoded protein binds to single-stranded DNA coated with replication protein A. The protein also interacts with the ataxia telangiectasia and Rad3 related protein kinase, resulting in its accumulation at intranuclear foci induced by DNA damage. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2012]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs915179">rs915179</a></td><td>multiple</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LMNA">LMNA</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#cbffcb">0.135</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.482187400064</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>ID</th>
							<th>Association</th>
							<th>Population</th>
							<th>Identifier</th>
							<th>Gene</th>
							<th>Pubmed ID</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">1457</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">significant</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">American (Caucasian)</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs915179</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">LMNA</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">genome_maintenance</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22340368</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">After 16 SNPs over LMNA gene were genotyped in 873 long-lived individuals and 443 controls for association with longevity, a haplotype was identified. Then, the result was tested in 3619 subjects from four independent samples (738 New England Centenarian Study samples, 905 Southern Italian Centenarian Study samples, 1103 France samples and 702 Einstein Ashkenazi Longevity Study). </th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A meta-analysis combining results from 5 sets of samples showed the haplotype based on 4 SNPs (rs915179, rs2485662, rs4641, rs1468772) from LMNA was significantly associated with longevity (P=7.5 × 10(-4), multiple-testing-adjusted P=0.037)</th>
						</tr>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LMNA)</th><td>The protein encoded by this gene is part of the nuclear lamina, a two-dimensional matrix of proteins located next to the inner nuclear membrane. The lamin family of proteins make up the matrix and are highly conserved in evolution. During mitosis, the lamina matrix is reversibly disassembled as the lamin proteins are phosphorylated. Lamin proteins are thought to be involved in nuclear stability, chromatin structure and gene expression. Vertebrate lamins consist of two types, A and B. Alternative splicing results in multiple transcript variants. Mutations in this gene lead to several diseases: Emery-Dreifuss muscular dystrophy, familial partial lipodystrophy, limb girdle muscular dystrophy, dilated cardiomyopathy, Charcot-Marie-Tooth disease, and Hutchinson-Gilford progeria syndrome. [provided by RefSeq, May 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6443429">rs6443429</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TBL1XR1">TBL1XR1</a></td>
				<td>C/C</td><td>A</td><td>C</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-122+45411T>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.65558536118</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TBL1XR1)</th><td>This gene is a member of the WD40 repeat-containing gene family and shares sequence similarity with transducin (beta)-like 1X-linked (TBL1X). The protein encoded by this gene is thought to be a component of both nuclear receptor corepressor (N-CoR) and histone deacetylase 3 (HDAC 3) complexes, and is required for transcriptional activation by a variety of transcription factors. Mutations in these gene have been associated with some autism spectrum disorders, and one finding suggests that haploinsufficiency of this gene may be a cause of intellectual disability with dysmorphism. Mutations in this gene as well as recurrent translocations involving this gene have also been observed in some tumors. [provided by RefSeq, Mar 2016]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1033686">rs1033686</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=RAD51B">RAD51B</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#bfffbf">0.165</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.756+8589G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.0965270684372</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (RAD51B)</th><td>The protein encoded by this gene is a member of the RAD51 protein family. RAD51 family members are evolutionarily conserved proteins essential for DNA repair by homologous recombination. This protein has been shown to form a stable heterodimer with the family member RAD51C, which further interacts with the other family members, such as RAD51, XRCC2, and XRCC3. Overexpression of this gene was found to cause cell cycle G1 delay and cell apoptosis, which suggested a role of this protein in sensing DNA damage. Rearrangements between this locus and high mobility group AT-hook 2 (HMGA2, GeneID 8091) have been observed in uterine leiomyomata. [provided by RefSeq, Mar 2016]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2387653">rs2387653</a></td><td>Italian, Ashkenazi Jewish and Japanese</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ADARB2">ADARB2</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#bcffbc">0.175</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.188-1714G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.251275510204</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20011587</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Association study of RNA editing genes ADARB1 and ADARB2 with exceptional longevity in the New England Centenarian Study (NECS), with more than 1,500 US individuals of primarily North European ancestry, the Southern Italian Centenarian Study (SICS), the Ashkenazi Jewish Centenarian Study (AJCS), a study of approximately 300 nonagenarians and centenarians, and the Japanese Centenarian Study (JCS), a study of Japanese centenarians which focuses on \semi-supercentenarians\" subjects living past 105 years"</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">SNPs in the RNA editing genes ADARB1 and ADARB2 were significantly associated with extreme old age in the New England Centenarian Study. These findings were replicated in different genetic backgrounds (Italian, Ashkenazi Jewish and Japanese) with mixed results.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ADARB2)</th><td>This gene encodes a member of the double-stranded RNA adenosine deaminase family of RNA-editing enzymes and may play a regulatory role in RNA editing. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2805543">rs2805543</a></td><td>Italian, Ashkenazi Jewish and Japanese</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ADARB2">ADARB2</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#bcffbc">0.175</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.101-43537A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.430131562141</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20011587</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Association study of RNA editing genes ADARB1 and ADARB2 with exceptional longevity in the New England Centenarian Study (NECS), with more than 1,500 US individuals of primarily North European ancestry, the Southern Italian Centenarian Study (SICS), the Ashkenazi Jewish Centenarian Study (AJCS), a study of approximately 300 nonagenarians and centenarians, and the Japanese Centenarian Study (JCS), a study of Japanese centenarians which focuses on \semi-supercentenarians\" subjects living past 105 years"</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">SNPs in the RNA editing genes ADARB1 and ADARB2 were significantly associated with extreme old age in the New England Centenarian Study. These findings were replicated in different genetic backgrounds (Italian, Ashkenazi Jewish and Japanese) with mixed results.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ADARB2)</th><td>This gene encodes a member of the double-stranded RNA adenosine deaminase family of RNA-editing enzymes and may play a regulatory role in RNA editing. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2676192">rs2676192</a></td><td>Italian, Ashkenazi Jewish and Japanese</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ADARB2">ADARB2</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#bcffbc">0.175</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.101-84119G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.585761758691</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20011587</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Association study of RNA editing genes ADARB1 and ADARB2 with exceptional longevity in the New England Centenarian Study (NECS), with more than 1,500 US individuals of primarily North European ancestry, the Southern Italian Centenarian Study (SICS), the Ashkenazi Jewish Centenarian Study (AJCS), a study of approximately 300 nonagenarians and centenarians, and the Japanese Centenarian Study (JCS), a study of Japanese centenarians which focuses on \semi-supercentenarians\" subjects living past 105 years"</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">SNPs in the RNA editing genes ADARB1 and ADARB2 were significantly associated with extreme old age in the New England Centenarian Study. These findings were replicated in different genetic backgrounds (Italian, Ashkenazi Jewish and Japanese) with mixed results.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ADARB2)</th><td>This gene encodes a member of the double-stranded RNA adenosine deaminase family of RNA-editing enzymes and may play a regulatory role in RNA editing. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3788157">rs3788157</a></td><td>Italian, Ashkenazi Jewish and Japanese</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ADARB1">ADARB1</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#bcffbc">0.175</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-220+16000T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.154692082111</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20011587</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Association study of RNA editing genes ADARB1 and ADARB2 with exceptional longevity in the New England Centenarian Study (NECS), with more than 1,500 US individuals of primarily North European ancestry, the Southern Italian Centenarian Study (SICS), the Ashkenazi Jewish Centenarian Study (AJCS), a study of approximately 300 nonagenarians and centenarians, and the Japanese Centenarian Study (JCS), a study of Japanese centenarians which focuses on \semi-supercentenarians\" subjects living past 105 years"</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">SNPs in the RNA editing genes ADARB1 and ADARB2 were significantly associated with extreme old age in the New England Centenarian Study. These findings were replicated in different genetic backgrounds (Italian, Ashkenazi Jewish and Japanese) with mixed results.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ADARB1)</th><td>This gene encodes the enzyme responsible for pre-mRNA editing of the glutamate receptor subunit B by site-specific deamination of adenosines. Studies in rat found that this enzyme acted on its own pre-mRNA molecules to convert an AA dinucleotide to an AI dinucleotide which resulted in a new splice site. Alternative splicing of this gene results in several transcript variants, some of which have been characterized by the presence or absence of an ALU cassette insert and a short or long C-terminal region. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3024239">rs3024239</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=WRN">WRN</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#b4ffb4">0.195</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.3138+6C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.464721614909</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (WRN)</th><td>This gene encodes a member of the RecQ subfamily of DNA helicase proteins. The encoded nuclear protein is important in the maintenance of genome stability and plays a role in DNA repair, replication, transcription and telomere maintenance. This protein contains a N-terminal 3&apos; to 5&apos; exonuclease domain, an ATP-dependent helicase domain and RQC (RecQ helicase conserved region) domain in its central region, and a C-terminal HRDC (helicase RNase D C-terminal) domain and nuclear localization signal. Defects in this gene are the cause of Werner syndrome, an autosomal recessive disorder characterized by accelerated aging and an elevated risk for certain cancers. [provided by RefSeq, Aug 2017]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>

	<button class="accordion0 accordion "><div></div>Recommendations</button>
<div class="board variants   panel">
	<p>From 15 gene variants related to genome maintenance and post-transcriptional processes, you have <b>13</b> favorable and <b>2</b> unfavorable. 
 </p>
	<br>
	<p>If you have unfavorable variants in genes involved in genome maintenance, there are several steps you can take to improve your health outcomes potentially:
</p>
	<br>
	<p><b>Ensure methylation processes in your body work well:</b> monitor blood levels of B12, B9, B2, B6, Zn, and Co[PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/31601260/">31601260</a>, PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/36203899/">36203899</a>]. These components are needed for methylation enzymes' appropriate work. Methylation is important for genome maintenance because it is involved in the regulation of gene expression and DNA repair [PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/34209979/">34209979</a>].</p>
	<p><b>Reduce exposure to environmental toxins</b>, such as pollution, pesticides, and chemicals. Air-control indoor systems can help with this point. 
</p>
<p><b>Check vitamin D level: </b>Vitamin D plays a role in DNA repair and has been shown to have a protective effect regarding genome maintenance [PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/11295155/">11295155</a>].

	<p><b>Protect the skin</b> from UV radiation by wearing protective clothing and using sunscreen.
</p>
	<br>
	<p>
Although genes associated with longevity can be classified into 11 definite pathways, there are other genes that do not fall into these categories, which are presented in this table.
</p>
	</div>
	<div class="last inside"></div>
</div>




	<button class="accordion accordion-heading"  >Other genes associated with longevity</button>
	<div class="basic-panel">
	<button class="accordion0 accordion accordion1"><div></div>Description</button>
<div class="board variants panel panel-description">
	<p>Although many genes associated with longevity can be classified into 11 definite pathways, there are other genes that do not fall into these categories, which are presented in this table.
 </p>
	</div>


	<button class="accordion0 accordion accordion-result"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui selectable celled teal table table variants">
		<thead>
			<tr><th class="clickable title" onclick="collapseExpandAll(this, 'longOtherTBody')"><div class="ui tiny circular compact positive button">+</div></th>
				<th>RSID</th><th>Population</th><th>Gene</th>
				<th>Your genotype</th><th>Ref allele</th><th>Alt allele</th><th>Zygosity</th>
				<th>Weight</th>
			</tr>
		</thead>
		<tbody id="longOtherTBody">
			
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1733676">rs1733676</a></td><td>American</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#ebffeb">0.05</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.234608506834</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">25682868</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A genome-wide linkage scan was performed for healthy aging in family study data, followed by fine-mapping by association in a separate population. A frailty index, called FI34, was introduced as a quantitative measure of healthy aging.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A single linkage peak was found  on chromosome 12  in the genome-wide linkage scan. Fine-scale association mapping identified three sites associated with healthy aging in intergenic regions at 12q13-14. </th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1834497">rs1834497</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CLSTN2">CLSTN2</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e6ffe6">0.065</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.109+72942A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.560059964277</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs, including in CDH4, SVEP1, CACNA1C, CARS, STK24, C7orf50, PARVG, NCAM2, PPP2R2C, NLRC5, BTBD9, RAC2, TGFA, KIAA0649, ABCC4, CLSTN2, FAM19A5 and RUNX3, were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CLSTN2)</th><td>Predicted to enable calcium ion binding activity. Predicted to be involved in positive regulation of synapse assembly and positive regulation of synaptic transmission. Predicted to be located in postsynaptic density. Predicted to be active in cell surface; glutamatergic synapse; and postsynaptic membrane. Predicted to be integral component of postsynaptic density membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10256972">rs10256972</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=C7orf50">C7orf50</a></td>
				<td>C/A</td><td>A</td><td>C</td><td>het</td><td style="background-color:#e6ffe6">0.065</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.405+1103T>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.605605925169</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs, including in CDH4, SVEP1, CACNA1C, CARS, STK24, C7orf50, PARVG, NCAM2, PPP2R2C, NLRC5, BTBD9, RAC2, TGFA, KIAA0649, ABCC4, CLSTN2, FAM19A5 and RUNX3, were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (C7orf50)</th><td>Enables RNA binding activity. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2370413">rs2370413</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CACNA1C">CACNA1C</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e6ffe6">0.065</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.477+125274T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.583679203116</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs, including in CDH4, SVEP1, CACNA1C, CARS, STK24, C7orf50, PARVG, NCAM2, PPP2R2C, NLRC5, BTBD9, RAC2, TGFA, KIAA0649, ABCC4, CLSTN2, FAM19A5 and RUNX3, were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CACNA1C)</th><td>This gene encodes an alpha-1 subunit of a voltage-dependent calcium channel. Calcium channels mediate the influx of calcium ions into the cell upon membrane polarization. The alpha-1 subunit consists of 24 transmembrane segments and forms the pore through which ions pass into the cell. The calcium channel consists of a complex of alpha-1, alpha-2/delta, beta, and gamma subunits in a 1:1:1:1 ratio. There are multiple isoforms of each of these proteins, either encoded by different genes or the result of alternative splicing of transcripts. The protein encoded by this gene binds to and is inhibited by dihydropyridine. Alternative splicing results in many transcript variants encoding different proteins. Some of the predicted proteins may not produce functional ion channel subunits. [provided by RefSeq, Oct 2012]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2826891">rs2826891</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=NCAM2">NCAM2</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e6ffe6">0.065</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.2403-51C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.439181784975</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs, including in CDH4, SVEP1, CACNA1C, CARS, STK24, C7orf50, PARVG, NCAM2, PPP2R2C, NLRC5, BTBD9, RAC2, TGFA, KIAA0649, ABCC4, CLSTN2, FAM19A5 and RUNX3, were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (NCAM2)</th><td>The protein encoded by this gene belongs to the immunoglobulin superfamily. It is a type I membrane protein and may function in selective fasciculation and zone-to-zone projection of the primary olfactory axons. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11211037">rs11211037</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=BTBD19">BTBD19</a></td>
				<td>C/A</td><td>A</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.87-334A>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.452681992337</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10923673">rs10923673</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.211086261981</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10210870">rs10210870</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LINC01250">LINC01250</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.271712079638</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1505696">rs1505696</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.384630114253</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1974676">rs1974676</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HPCAL1">HPCAL1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.484+699A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.491507566567</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (HPCAL1)</th><td>The protein encoded by this gene is a member of neuron-specific calcium-binding proteins family found in the retina and brain. It is highly similar to human hippocalcin protein and nearly identical to the rat and mouse hippocalcin like-1 proteins. It may be involved in the calcium-dependent regulation of rhodopsin phosphorylation and may be of relevance for neuronal signalling in the central nervous system. Several alternatively spliced transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Apr 2012]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11687681">rs11687681</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.454319095959</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1581035">rs1581035</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.421882984159</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3769583">rs3769583</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TTC27">TTC27</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1452+6238C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.161240903868</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2901840">rs2901840</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.145500957243</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4363980">rs4363980</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC105373714">LOC105373714</a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-20+16800T>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.646105760031</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LOC105373714)</th><td>Enables deoxyuridine phosphorylase activity; identical protein binding activity; and uridine phosphorylase activity. Involved in dCMP catabolic process and uridine catabolic process. Located in type III intermediate filament. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12623542">rs12623542</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LINC01237">LINC01237</a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.470718836476</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12634249">rs12634249</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SUMF1">SUMF1</a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.519+4647G>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.199240684022</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (SUMF1)</th><td>This gene encodes an enzyme that catalyzes the hydrolysis of sulfate esters by oxidizing a cysteine residue in the substrate sulfatase to an active site 3-oxoalanine residue, which is also known as C-alpha-formylglycine. Mutations in this gene cause multiple sulfatase deficiency, a lysosomal storage disorder. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2009]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12489296">rs12489296</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.253404949166</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs15606">rs15606</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LINC02032">LINC02032</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.473861449647</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1456669">rs1456669</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LINC02045">LINC02045</a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.131817601226</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs767478">rs767478</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.226646248086</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11943045">rs11943045</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.229692680439</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs765850">rs765850</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APBB2">APBB2</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1530-9773A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.569340736486</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (APBB2)</th><td>The protein encoded by this gene interacts with the cytoplasmic domains of amyloid beta (A4) precursor protein and amyloid beta (A4) precursor-like protein 2. This protein contains two phosphotyrosine binding (PTB) domains, which are thought to function in signal transduction. Polymorphisms in this gene have been associated with Alzheimer&apos;s disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2009]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2660342">rs2660342</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.732846109657</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2553377">rs2553377</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.140164561806</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1002098">rs1002098</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.236981324495</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1189843">rs1189843</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.231792091837</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2331101">rs2331101</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.243328651685</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1490813">rs1490813</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.493229432805</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs158807">rs158807</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.722675939514</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs924584">rs924584</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.680108556833</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs147295">rs147295</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.*5314A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.608634563801</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description ()</th><td>Involved in snRNA transcription by RNA polymerase II. Located in nucleoplasm. Part of transcription elongation factor complex. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9324976">rs9324976</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/A</td><td>A</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.114768740032</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs923520">rs923520</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC105377739">LOC105377739</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.327665411855</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs524533">rs524533</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TMEM151B">TMEM151B</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.136-244C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.740280086968</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TMEM151B)</th><td>Predicted to be integral component of membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs514217">rs514217</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.5170850099</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4245543">rs4245543</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.675370275792</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6946852">rs6946852</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.581673052363</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4141153">rs4141153</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.220810724545</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10263457">rs10263457</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.262499123484</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4240673">rs4240673</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=XKR6">XKR6</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.765-5272A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.549658735727</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (XKR6)</th><td>Predicted to be involved in apoptotic process involved in development; engulfment of apoptotic cell; and phosphatidylserine exposure on apoptotic cell surface. Predicted to be integral component of membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6984496">rs6984496</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=XKR6">XKR6</a></td>
				<td>T/G</td><td>G</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.765-13753C>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.557087242326</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (XKR6)</th><td>Predicted to be involved in apoptotic process involved in development; engulfment of apoptotic cell; and phosphatidylserine exposure on apoptotic cell surface. Predicted to be integral component of membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6991271">rs6991271</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=KIF13B">KIF13B</a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.150-9456A>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.521461420542</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (KIF13B)</th><td>Enables 14-3-3 protein binding activity and protein kinase binding activity. Involved in regulation of axonogenesis. Located in axon and cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1812736">rs1812736</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.846347526772</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10505561">rs10505561</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.124824684432</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs829751">rs829751</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.808460409622</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs997675">rs997675</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LINC00032">LINC00032</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.430239979576</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2383722">rs2383722</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=EQTN">EQTN</a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.481+775A>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.458985497988</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (EQTN)</th><td>Predicted to be involved in acrosomal vesicle exocytosis; endocytosis; and fusion of sperm to egg plasma membrane involved in single fertilization. Predicted to be located in acrosomal membrane; early endosome; and nucleus. Predicted to be active in inner acrosomal membrane; outer acrosomal membrane; and plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7857137">rs7857137</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PTAR1">PTAR1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.*4531T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.265137966275</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PTAR1)</th><td>Predicted to enable protein prenyltransferase activity. Predicted to be involved in protein prenylation. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12341685">rs12341685</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TMEM2">TMEM2</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.2986-1662A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.156862745098</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TMEM2)</th><td>This gene encodes a type II transmembrane protein that belongs to the interferon-induced transmembrane (IFITM) protein superfamily. The encoded protein functions as a cell surface hyaluronidase that cleaves extracellular high molecular weight hyaluronan into intermediate size fragments before internalization and degradation in the lysosome. It also has an interferon-mediated antiviral function in humans through activation of the JAK STAT signaling pathway. The activation of this gene by transcription factor SOX4 in breast cancer cells has been shown to mediate the pathological effects of SOX4 on cancer progression. Naturally occurring mutations in this gene are associated with autosomal recessive non-syndromic hearing loss. [provided by RefSeq, Mar 2017]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1016013">rs1016013</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.662411438055</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1867102">rs1867102</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=C9orf3">C9orf3</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1365-58501A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.426917187699</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (C9orf3)</th><td>This gene encodes a member of the M1 zinc aminopeptidase family. The encoded protein is a zinc-dependent metallopeptidase that catalyzes the removal of an amino acid from the amino terminus of a protein or peptide. This protein may play a role in the generation of angiotensin IV. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Oct 2010]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs303542">rs303542</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.0761112173968</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10512392">rs10512392</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=EPB41L4B">EPB41L4B</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.307-13858G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.236772486772</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (EPB41L4B)</th><td>Predicted to be a structural constituent of cytoskeleton. Involved in several processes, including positive regulation of cell adhesion; positive regulation of keratinocyte migration; and wound healing. Acts upstream of or within actomyosin structure organization. Located in apical part of cell; cytosol; and plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4933309">rs4933309</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.734939374601</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11601702">rs11601702</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.250909090909</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description ()</th><td>The protein encoded by this gene forms a receptor-activated calcium channel in the cell membrane. The channel is activated by diacylglycerol and is thought to be under the control of a phosphatidylinositol second messenger system. Activation of this channel occurs independently of protein kinase C and is not triggered by low levels of intracellular calcium. Defects in this gene are a cause of focal segmental glomerulosclerosis 2 (FSGS2). [provided by RefSeq, Mar 2009]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2660888">rs2660888</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.43997188678</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1340573">rs1340573</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.910237625024</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12585627">rs12585627</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=RNF219-AS1">RNF219-AS1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.14021551999</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9652250">rs9652250</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.114420562536</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs17702471">rs17702471</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=GPC6">GPC6</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-2593A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.164636091089</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (GPC6)</th><td>The glypicans comprise a family of glycosylphosphatidylinositol-anchored heparan sulfate proteoglycans, and they have been implicated in the control of cell growth and cell division. The glypican encoded by this gene is a putative cell surface coreceptor for growth factors, extracellular matrix proteins, proteases and anti-proteases. Mutations in this gene are associated with omodysplasia 1. [provided by RefSeq, Nov 2016]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10498342">rs10498342</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.280691679428</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7149754">rs7149754</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC105370656">LOC105370656</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.419491795952</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1280396">rs1280396</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CGNL1">CGNL1</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1531A>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.846012868144</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CGNL1)</th><td>This gene encodes a member of the cingulin family. The encoded protein localizes to both adherens and tight cell-cell junctions and mediates junction assembly and maintenance by regulating the activity of the small GTPases RhoA and Rac1. Heterozygous chromosomal rearrangements resulting in association of the promoter for this gene with the aromatase gene are a cause of aromatase excess syndrome. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, Nov 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12446827">rs12446827</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.266447368421</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs933664">rs933664</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=GSG1L">GSG1L</a></td>
				<td>C/A</td><td>A</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.397+38210T>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.37378516624</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (GSG1L)</th><td>Predicted to be involved in regulation of AMPA receptor activity. Predicted to be located in postsynaptic density. Predicted to be active in Schaffer collateral - CA1 synapse; glutamatergic synapse; and plasma membrane. Predicted to be integral component of postsynaptic density membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2334207">rs2334207</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.481352130727</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2304965">rs2304965</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.370928598799</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1005321">rs1005321</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HS3ST3B1">HS3ST3B1</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.554+16112G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.449229982747</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (HS3ST3B1)</th><td>The protein encoded by this gene is a type II integral membrane protein that belongs to the 3-O-sulfotransferases family. These proteins catalyze the addition of sulfate groups at the 3-OH position of glucosamine in heparan sulfate. The substrate specificity of individual members of the family is based on prior modification of the heparan sulfate chain, thus allowing different members of the family to generate binding sites for different proteins on the same heparan sulfate chain. Following treatment with a histone deacetylase inhibitor, expression of this gene is activated in a pancreatic cell line. The increased expression results in promotion of the epithelial-mesenchymal transition. In addition, the modification catalyzed by this protein allows herpes simplex virus membrane fusion and penetration. A very closely related homolog with an almost identical sulfotransferase domain maps less than 1 Mb away. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7207422">rs7207422</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ASIC2">ASIC2</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.709-15303T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.256399616981</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (ASIC2)</th><td>This gene encodes a member of the degenerin/epithelial sodium channel (DEG/ENaC) superfamily. The members of this family are amiloride-sensitive sodium channels that contain intracellular N and C termini, 2 hydrophobic transmembrane regions, and a large extracellular loop, which has many cysteine residues with conserved spacing. The member encoded by this gene may play a role in neurotransmission. In addition, a heteromeric association between this member and acid-sensing (proton-gated) ion channel 3 has been observed to co-assemble into proton-gated channels sensitive to gadolinium. Alternative splicing has been observed at this locus and two variants, encoding distinct isoforms, have been identified. [provided by RefSeq, Feb 2012]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2880540">rs2880540</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.15198954215</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description ()</th><td>This gene is a member of the DEAH box polypeptide family. The encoded protein contains the DEAH (Asp-Glu-Ala-His) motif which is characteristic of all DEAH box proteins, and is thought to function as an ATP-dependent RNA helicase that regulates the release of spliced mRNAs from spliceosomes prior to their export from the nucleus. This protein may be required for the replication of human immunodeficiency virus type 1 (HIV-1). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2014]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7505612">rs7505612</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/A</td><td>A</td><td>C</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.720105921388</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1368774">rs1368774</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.392800102217</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2061450">rs2061450</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC105372161">LOC105372161</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.278721502012</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7246865">rs7246865</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=MYO9B">MYO9B</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.840+5738G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.333205619413</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (MYO9B)</th><td>This gene encodes a member of the myosin family of actin-based molecular motor heavy chain proteins. The protein represents an unconventional myosin; it should not be confused with the conventional non-muscle myosin-9 (MYH9). The protein has four IQ motifs located in the neck domain that bind calmodulin, which serves as a light chain. The protein complex has a single-headed structure and exhibits processive movement on actin filaments toward the minus-end. The protein also has rho-GTPase activity. Polymorphisms in this gene are associated with celiac disease and ulcerative colitis susceptibility. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4802234">rs4802234</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC107985305">LOC107985305</a></td>
				<td>G/T</td><td>T</td><td>G</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6115865">rs6115865</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=C20orf194">C20orf194</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.284-2354G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.354577982238</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (C20orf194)</th><td>This gene encodes an uncharacterized protein with a C-terminal coiled-coil region. The gene is located on chromosome 20p13 in a 1.8 Mb region linked to a spinocerebellar ataxia phenotype, but this gene does not appear to be a disease candidate. [provided by RefSeq, Dec 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4596036">rs4596036</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PREX1">PREX1</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.3775+93C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.241085666901</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PREX1)</th><td>The protein encoded by this gene acts as a guanine nucleotide exchange factor for the RHO family of small GTP-binding proteins (RACs).  It has been shown to bind to and activate RAC1 by exchanging bound GDP for free GTP. The encoded protein, which is found mainly in the cytoplasm, is activated by phosphatidylinositol-3,4,5-trisphosphate and the beta-gamma subunits of heterotrimeric G proteins. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2253363">rs2253363</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.726544037811</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9977638">rs9977638</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=BTG3">BTG3</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#e4ffe4">0.07</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.520-1670G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.511192120747</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (BTG3)</th><td>The protein encoded by this gene is a member of the BTG/Tob family. This family has structurally related proteins that appear to have antiproliferative properties. This encoded protein might play a role in neurogenesis in the central nervous system. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1503434">rs1503434</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HTR1F">HTR1F</a></td>
				<td>C/T</td><td>T</td><td>C</td><td>het</td><td style="background-color:#e2ffe2">0.075</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-43+39114T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.159273678836</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (HTR1F)</th><td>Enables G protein-coupled serotonin receptor activity and serotonin binding activity. Involved in adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway. Is integral component of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs432203">rs432203</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TGFA">TGFA</a></td>
				<td>A/C</td><td>C</td><td>A</td><td>het</td><td style="background-color:#e0ffe0">0.08</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.40+16172G>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.589627455984</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs, including in CDH4, SVEP1, CACNA1C, CARS, STK24, C7orf50, PARVG, NCAM2, PPP2R2C, NLRC5, BTBD9, RAC2, TGFA, KIAA0649, ABCC4, CLSTN2, FAM19A5 and RUNX3, were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (TGFA)</th><td>This gene encodes a growth factor that is a ligand for the epidermal growth factor receptor, which activates a signaling pathway for cell proliferation, differentiation and development. This protein may act as either a transmembrane-bound ligand or a soluble ligand. This gene has been associated with many types of cancers, and it may also be involved in some cases of cleft lip/palate. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2149954">rs2149954</a></td><td>European</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LINC02227">LINC02227</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#deffde">0.085</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.361480439785</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">24688116</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A genome-wide association meta-analysis was performed in 7729 long-lived individuals of European descent (>=85 years) and 16,121 younger controls ( <65 years) followed by replication in an additional set of 13,060 long-lived individuals and 61 156 controls.</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">rs2149954 and rs4420638 were observed significantly associated with longevity. Meta-analysis showed rs2149954 (T) associates with increased survival.</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7799039">rs7799039</a></td><td>Jordanian</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC105375494">LOC105375494</a></td>
				<td>A/G</td><td>G</td><td>A</td><td>het</td><td style="background-color:#d3ffd3">0.115</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.370785440613</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">20201642</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">SNPs rs266729 (-11377G/C), rs2241766 (+45T/G), and rs1501299 (+276 G/T) of adiponectin gene (ADIPQO) and rs7799039 (-2548C/A) SNP of leptin (LEP) gene were genotyped in 110 randomly selected elderly subjects (>85 years old) with mean age of 90.2 years, and 120 young control subjects (range from 20 to 50 years) with mean age of 32.0 years</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">No significant differences were detected in the genotype and allele frequencies of examined gene variants between the two groups (p > 0.05). However, when gender was considered, genotypes and alleles frequencies of rs1501299 SNP in ADIPOQ gene and rs7799039 in LEP gene were significantly associated with longevity in men (p < 0.02) but not in women (p > 0.05).</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2032563">rs2032563</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CAMTA1">CAMTA1</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.4182+2746G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.149107142857</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CAMTA1)</th><td>The protein encoded by this gene contains a CG1 DNA-binding domain, a transcription factor immunoglobulin domain, ankyrin repeats, and calmodulin-binding IQ motifs. The encoded protein is thought to be a transcription factor and may be a tumor suppressor. However, a translocation event is sometimes observed between this gene and the WWTR1 gene, with the resulting WWTR1-CAMTA1 oncoprotein leading to epithelioid hemangioendothelioma, a malignant vascular cancer. [provided by RefSeq, Mar 2017]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12129750">rs12129750</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.*4747A>G</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description ()</th><td>This gene encodes a protein belonging to the vacuolar-protein-sorting-13 gene family. In yeast, vacuolar-protein-sorting-13 proteins are involved in trafficking of membrane proteins between the trans-Golgi network and the prevacuolar compartment. While several transcript variants may exist for this gene, the full-length natures of only two have been described to date. These two represent the major variants of this gene and encode distinct isoforms. [provided by RefSeq, Jul 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1463990">rs1463990</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC107985988">LOC107985988</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.801530612245</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs922943">rs922943</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=RARB">RARB</a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.179-93936G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.419250687296</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (RARB)</th><td>This gene encodes retinoic acid receptor beta, a member of the thyroid-steroid hormone receptor superfamily of nuclear transcriptional regulators. This receptor localizes to the cytoplasm and to subnuclear compartments. It binds retinoic acid, the biologically active form of vitamin A which mediates cellular signalling in embryonic morphogenesis, cell growth and differentiation. It is thought that this protein limits growth of many cell types by regulating gene expression. The gene was first identified in a hepatocellular carcinoma where it flanks a hepatitis B virus integration site. Alternate promoter usage and differential splicing result in multiple transcript variants. [provided by RefSeq, Mar 2014]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4461634">rs4461634</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.810652021202</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs975072">rs975072</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.790423361387</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2312243">rs2312243</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LOC102723654">LOC102723654</a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.1112456968</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6580511">rs6580511</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>A/A</td><td>G</td><td>A</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.561290322581</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs303006">rs303006</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CMAHP">CMAHP</a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.849801202642</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1858897">rs1858897</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.818506472801</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs567971">rs567971</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/C</td><td>T</td><td>C</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.799317122982</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2247549">rs2247549</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.885039872408</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2722222">rs2722222</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.843863563915</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs10483493">rs10483493</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=TTC6">TTC6</a></td>
				<td>G/G</td><td>T</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.4030-524T>G</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.230248883216</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2014547">rs2014547</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.814844348048</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7155817">rs7155817</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.877779901867</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2277509">rs2277509</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CCDC88C">CCDC88C</a></td>
				<td>A/A</td><td>C</td><td>A</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.4630+78G>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.668734015345</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CCDC88C)</th><td>This gene encodes a ubiquitously expressed coiled-coil domain-containing protein that interacts with the dishevelled protein and is a negative regulator of the Wnt signalling pathway. The protein encoded by this gene has a PDZ-domain binding motif in its C-terminus with which it interacts with the dishevelled protein. Dishevelled is a scaffold protein involved in the regulation of the Wnt signaling pathway. The Wnt signaling pathway plays an important role in embryonic development, tissue maintenance, and cancer progression. Mutations in this gene cause autosomal recessive, primary non-syndromic congenital hydrocephalus; a condition characterized by excessive accumulation of cerebrospinal fluid in the ventricles of the brain. [provided by RefSeq, Jan 2013]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs8073559">rs8073559</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>A</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.811945914918</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs8098316">rs8098316</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>G/G</td><td>T</td><td>G</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.808856394866</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs9304496">rs9304496</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=KCTD1">KCTD1</a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1809+17895G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.737238386932</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (KCTD1)</th><td>This gene encodes a protein containing a BTB (Broad-complex, tramtrack and bric a brac), also known as a POZ (POxvirus and zinc finger) protein-protein interaction domain. The encoded protein negatively regulates the AP-2 family of transcription factors and the Wnt signaling pathway. A mechanism for the modulation of Wnt signaling has been proposed in which the encoded protein enhances ubiquitination and degradation of the beta-catenin protein. Mutations in this gene have been identified in Scalp-ear-nipple (SEN) syndrome. [provided by RefSeq, May 2017]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2927261">rs2927261</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
				<td>C/C</td><td>T</td><td>C</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.818485949149</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11086106">rs11086106</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PGPEP1">PGPEP1</a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.88-5247C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.860891719745</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (PGPEP1)</th><td>The gene encodes a cysteine protease and member of the peptidase C15 family of proteins. The encoded protein cleaves amino terminal pyroglutamate residues from protein substrates including thyrotropin-releasing hormone and other neuropeptides. Expression of this gene may be downregulated in colorectal cancer, while activity of the encoded protein may be negatively correlated with cancer progression in colorectal cancer patients. Activity of the encoded protease may also be altered in other disease states including in liver cirrhosis, which is associated with reduced protease activity, and in necrozoospermia, which is associated with elevated protease activity. [provided by RefSeq, Jul 2016]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs5754682">rs5754682</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LARGE1">LARGE1</a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#c9ffc9">0.14</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.-82-44739G>A</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.904059393321</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (LARGE1)</th><td>This gene encodes a member of the N-acetylglucosaminyltransferase gene family. It encodes a glycosyltransferase which participates in glycosylation of alpha-dystroglycan, and may carry out the synthesis of glycoprotein and glycosphingolipid sugar chains. It may also be involved in the addition of a repeated disaccharide unit. The protein encoded by this gene is the glycotransferase that adds the final xylose and glucuronic acid to alpha-dystroglycan and thereby allows alpha-dystroglycan to bind ligands including laminin 211 and neurexin. Mutations in this gene cause several forms of congenital muscular dystrophy characterized by cognitive disability and abnormal glycosylation of alpha-dystroglycan. Alternative splicing of this gene results in multiple transcript variants that encode the same protein. [provided by RefSeq, May 2018]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs13053175">rs13053175</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SSTR3">SSTR3</a></td>
				<td>T/T</td><td>C</td><td>T</td><td>hom</td><td style="background-color:#c1ffc1">0.16</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>Substitution frequency (GnomAD)</th><td>0.322686662412</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22533364</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study using 1,471 genotyped participants from the Framingham Heart Study, of which 1,173 individuals had known lifespans, plus 517 individuals from the Offspring cohort for validating connections between longevity and genetic variants</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">A total of 27 SNPs, including in CDH4, SVEP1, CACNA1C, CARS, STK24, C7orf50, PARVG, NCAM2, PPP2R2C, NLRC5, BTBD9, RAC2, TGFA, KIAA0649, ABCC4, CLSTN2, FAM19A5 and RUNX3, were identified at the intersection of various statistical procedures</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7656234">rs7656234</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SLC4A4">SLC4A4</a></td>
				<td>T/C</td><td>C</td><td>T</td><td>het</td><td style="background-color:#b8ffb8">0.185</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.1974+3411C>T</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.590667775361</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (SLC4A4)</th><td>This gene encodes a sodium bicarbonate cotransporter (NBC) involved in the regulation of bicarbonate secretion and absorption and intracellular pH. Mutations in this gene are associated with proximal renal tubular acidosis. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2008]</td></tr>
				</table>
			</td></tr>
					
			<tr class="clickable" onclick="doOnClick(this)">
				<td><div class="ui tiny circular compact positive button">+</div></td>
				<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1860242">rs1860242</a></td><td>American (Caucasian)</td>
				<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CTNND2">CTNND2</a></td>
				<td>G/A</td><td>A</td><td>G</td><td>het</td><td style="background-color:#b0ffb0">0.205</td>
			</tr>
			<tr class="hidden" style="display:none;"><td colspan="8">
				<table class="ui padded table">
				<tr><th>cDNA change</th><td>c.2788+13539T>C</td></tr>
				<tr><th>Substitution frequency (GnomAD)</th><td>0.740380320337</td></tr>
				<tr><td colspan="2">
				</td></tr>
				<tr><td colspan="2">
				<table class="ui selectable celled teal table table variants">
					<thead>
						<tr>
							<th>Pubmed ID</th>
							<th>Study design</th>
							<th>Conclusions</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">22279548</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">Genome-wide association study in 801 centenarians and 914 healthy controls</th>
							<th style="font-weight: normal; background-color: rgb(255, 251, 251)smoke;">281 SNPs were found to discriminate between cases and controls</th>
						</tr>
					</tbody>
				</table>
				</td></tr>
					<tr><th>Gene Description (CTNND2)</th><td>This gene encodes an adhesive junction associated protein of the armadillo/beta-catenin superfamily and is implicated in brain and eye development and cancer formation. The protein encoded by this gene promotes the disruption of E-cadherin based adherens junction to favor cell spreading upon stimulation by hepatocyte growth factor. This gene is overexpressed in prostate adenocarcinomas and is associated with decreased expression of tumor suppressor E-cadherin in this tissue. This gene resides in a region of the short arm of chromosome 5 that is deleted in Cri du Chat syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2013]</td></tr>
				</table>
			</td></tr>
		</tbody>
	</table>
	</div>
	<div class="last inside"></div>
</div>
	<p></p>
	<h2 id="drugs" style="color:rgb(57, 145, 123);">Longevity drugs</h2>
	<button class="accordion accordion0 accordion1 module-button"><div></div>Description</button>
<div class="board variants panel panel-description accord_drugs">

	<p>Plenty of drugs are frequently prescribed at an older age. Moreover, some drugs are known to have geroprotective action (e.g. statins, metformin, rapamycin, etc.). Drug metabolism to a large extent depends on a person&rsquo;s genetic polymorphisms, affecting the activity of xenobiotics-transforming enzymes. So it&rsquo;s a common situation when individual dose correction is needed, or even another drug has to be taken in order to avoid adverse effects. Longevity-drugs report is mainly based on data from PharmGKB database (
			<a href="https://www.google.com/url?q=https://www.pharmgkb.org/&amp;sa=D&amp;source=editors&amp;ust=1664211538648790&amp;usg=AOvVaw0_aiXtrV8uDwT1_Vh57zSO">https://www.pharmgkb.org/</a>
			) and DrugAge.</p>
	<p></p>
	<p>In the table you can find data about your genome&rsquo;s gene variants, associated with a changed response to the drugs.
	</p>
</div>
	<button class="accordion accordion0 accordion-result accord_drugs module-button"><div></div>Results</button>
	<div class="content content-variants">
	<table class="ui celled teal table table drugs">
		<thead>
		<tr><th>#</th>
				<th>Variant/</br>Haplotypes</th><th>Drug(s)</th><th>Phenotype Category</th><th>Significance</th><th>Sentence</th><th>Allele Of Frequency In Cases</th>
				<th>Allele Of Frequency In Controls</th><th>Ratio Stat Type</th><th>Effect</th></tr>
		</thead>
		<tbody id="drugsTBody">
            
			<tr><td>1</td><td>rs2284018</td>
				<td>lithium</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype TT is associated with decreased response to lithium in people with Bipolar Disorder as compared to genotypes CC + CT.</td><td></td><td></td>
				<td>OR</td><td>9.091</td></tr>
				
			<tr><td>2</td><td>rs708272</td>
				<td>rosuvastatin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to rosuvastatin as compared to allele G.</td><td>A</td><td></td>
				<td>OR</td><td>8.05</td></tr>
				
			<tr><td>3</td><td>rs7316769</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>7.692</td></tr>
				
			<tr><td>4</td><td>rs10771999</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>7.692</td></tr>
				
			<tr><td>5</td><td>rs5441</td>
				<td>sertraline</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to sertraline in people with Depressive Disorder, Major as compared to genotypes AA + AG.</td><td>G</td><td></td>
				<td>OR</td><td>5.9</td></tr>
				
			<tr><td>6</td><td>rs7306991</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>5.882</td></tr>
				
			<tr><td>7</td><td>rs10771997</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>5.882</td></tr>
				
			<tr><td>8</td><td>rs10771998</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>5.882</td></tr>
				
			<tr><td>9</td><td>rs12595802</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>5.556</td></tr>
				
			<tr><td>10</td><td>rs7787082</td>
				<td>clozapine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to clozapine in people with Schizophrenia as compared to allele A.</td><td>A</td><td></td>
				<td>OR</td><td>4.87</td></tr>
				
			<tr><td>11</td><td>rs1042713</td>
				<td>tiotropium</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with increased response to tiotropium in people with Asthma as compared to genotypes AG + GG.</td><td>G</td><td></td>
				<td>OR</td><td>4.762</td></tr>
				
			<tr><td>12</td><td>rs3087403</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CT + TT are associated with increased response to cisplatin in people with Osteosarcoma as compared to genotype CC.</td><td>T</td><td></td>
				<td>OR</td><td>4.44</td></tr>
				
			<tr><td>13</td><td>rs1056827</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is associated with decreased response to risperidone in people with Schizophrenia as compared to allele C.</td><td>A</td><td>A</td>
				<td>OR</td><td>4.37</td></tr>
				
			<tr><td>14</td><td>rs622342</td>
				<td>metformin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AC + CC is associated with decreased response to metformin in people with Diabetes Mellitus, Type 2 as compared to genotype AA.</td><td>A</td><td></td>
				<td>OR</td><td>4.1</td></tr>
				
			<tr><td>15</td><td>rs11591741</td>
				<td>ustekinumab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CC + CG is associated with increased response to ustekinumab in people with Psoriasis as compared to genotype GG.</td><td></td><td></td>
				<td>OR</td><td>33.333</td></tr>
				
			<tr><td>16</td><td>rs4437856</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>3.704</td></tr>
				
			<tr><td>17</td><td>rs2419128</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>3.704</td></tr>
				
			<tr><td>18</td><td>rs12094644</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>3.704</td></tr>
				
			<tr><td>19</td><td>rs1883112</td>
				<td>idarubicin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with increased response to idarubicin in people with Leukemia, Myeloid, Acute as compared to genotype GG.</td><td>A</td><td></td>
				<td>OR</td><td>3.7</td></tr>
				
			<tr><td>20</td><td>rs4633</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is associated with decreased response to risperidone in people with Schizophrenia as compared to allele C.</td><td>T</td><td>T</td>
				<td>OR</td><td>3.56</td></tr>
				
			<tr><td>21</td><td>rs4639250</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>3.333</td></tr>
				
			<tr><td>22</td><td>rs12657120</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>3.226</td></tr>
				
			<tr><td>23</td><td>rs10012</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is associated with decreased response to risperidone in people with Schizophrenia as compared to allele G.</td><td>C</td><td>C</td>
				<td>OR</td><td>3.16</td></tr>
				
			<tr><td>24</td><td>rs3124955</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>3.125</td></tr>
				
			<tr><td>25</td><td>rs3128624</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to duloxetine in people with Depressive Disorder, Major as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>3.125</td></tr>
				
			<tr><td>26</td><td>rs6923761</td>
				<td>liraglutide</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AA + AG are associated with increased response to liraglutide in women with Obesity and Polycystic Ovary Syndrome as compared to genotype GG.</td><td>A</td><td>G</td>
				<td>OR</td><td>3.06</td></tr>
				
			<tr><td>27</td><td>rs2270007</td>
				<td>citalopram</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CC + CG are associated with decreased response to citalopram in people with Depressive Disorder, Major as compared to genotype GG.</td><td>C</td><td>C</td>
				<td>OR</td><td>2.93</td></tr>
				
			<tr><td>28</td><td>rs6498169</td>
				<td>glatiramer acetate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased response to glatiramer acetate in people with Multiple Sclerosis as compared to allele G.</td><td>A</td><td>A</td>
				<td>OR</td><td>2.86</td></tr>
				
			<tr><td>29</td><td>rs1803274</td>
				<td>rivastigmine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CT + TT is associated with decreased response to rivastigmine in people with Alzheimer Disease as compared to genotype CC.</td><td>C</td><td>T</td>
				<td>OR</td><td>2.857</td></tr>
				
			<tr><td>30</td><td>rs316019</td>
				<td>metformin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AC is associated with increased response to metformin in people with Diabetes Mellitus, Type 2 as compared to genotype CC.</td><td>A</td><td>A</td>
				<td>OR</td><td>2.857</td></tr>
				
			<tr><td>31</td><td>rs2284017</td>
				<td>lithium</td><td>Efficacy</td>
				<td>not stated</td>
				<td>Allele C is associated with increased response to lithium in people with Bipolar Disorder as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>2.857</td></tr>
				
			<tr><td>32</td><td>rs1695</td>
				<td>cyclophosphamide</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AG + GG is associated with decreased response to cyclophosphamide in people with Lupus Nephritis as compared to genotype AA.</td><td>G</td><td>G</td>
				<td>OR</td><td>2.8</td></tr>
				
			<tr><td>33</td><td>rs13058338</td>
				<td>idarubicin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AT is associated with increased response to idarubicin in people with Leukemia, Myeloid, Acute as compared to genotype TT.</td><td>A</td><td></td>
				<td>OR</td><td>2.8</td></tr>
				
			<tr><td>34</td><td>rs5333</td>
				<td>prednisone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CC + CT is associated with increased resistance to prednisone in children with Nephrotic Syndrome as compared to genotype TT.</td><td>C</td><td>C</td>
				<td>OR</td><td>2.74</td></tr>
				
			<tr><td>35</td><td>rs13064530</td>
				<td>clozapine</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with response to clozapine in people with Schizophrenia as compared to allele A.</td><td>A</td><td></td>
				<td>OR</td><td>2.6</td></tr>
				
			<tr><td>36</td><td>rs2072671</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes AC + CC.</td><td>C</td><td></td>
				<td>OR</td><td>2.5</td></tr>
				
			<tr><td>37</td><td>rs362267</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is associated with decreased response to risperidone in people with Schizophrenia as compared to allele C.</td><td>T</td><td>T</td>
				<td>OR</td><td>2.5</td></tr>
				
			<tr><td>38</td><td>rs2372536</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to methotrexate in people with Arthritis, Rheumatoid as compared to genotypes CC + CG.</td><td>G</td><td>G</td>
				<td>OR</td><td>2.4</td></tr>
				
			<tr><td>39</td><td>rs6277</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype GG is associated with decreased response to methadone in people with Substance-Related Disorders.</td><td>A</td><td></td>
				<td>OR</td><td>2.4</td></tr>
				
			<tr><td>40</td><td>rs11869731</td>
				<td>lithium</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to lithium in people with Bipolar Disorder as compared to genotypes CG + GG.</td><td>C</td><td>C</td>
				<td>OR</td><td>2.39</td></tr>
				
			<tr><td>41</td><td>rs9904341</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CG is associated with increased response to methotrexate in people with Arthritis, Rheumatoid as compared to genotype GG.</td><td>C</td><td></td>
				<td>OR</td><td>2.34</td></tr>
				
			<tr><td>42</td><td>rs1800469</td>
				<td>glatiramer acetate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to glatiramer acetate in people with Multiple Sclerosis as compared to allele G.</td><td>A</td><td>A</td>
				<td>OR</td><td>2.326</td></tr>
				
			<tr><td>43</td><td>rs362306</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is associated with decreased response to risperidone in people with Schizophrenia as compared to allele G.</td><td>A</td><td>A</td>
				<td>OR</td><td>2.22</td></tr>
				
			<tr><td>44</td><td>rs165599</td>
				<td>bupropion</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AG are associated with increased response to bupropion in smokers as compared to genotype GG.</td><td>A</td><td></td>
				<td>OR</td><td>2.21</td></tr>
				
			<tr><td>45</td><td>rs1801058</td>
				<td>metoprolol</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CT is associated with decreased response to metoprolol in women with hypertensive nephrosclerosis as compared to genotype CC.</td><td></td><td></td>
				<td>HR</td><td>2.174</td></tr>
				
			<tr><td>46</td><td>rs616147</td>
				<td>creatine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AG are associated with increased response to creatine in people with Amyotrophic Lateral Sclerosis as compared to genotype GG.</td><td>A</td><td></td>
				<td>HR</td><td>2.083</td></tr>
				
			<tr><td>47</td><td>rs680244</td>
				<td>Drugs used in nicotine dependence</td><td>Other</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to Drugs used in nicotine dependence in people with Tobacco Use Disorder.</td><td></td><td></td>
				<td>HR</td><td>2.083</td></tr>
				
			<tr><td>48</td><td>rs10161126</td>
				<td>alendronate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to alendronate in women with Bone Diseases as compared to genotypes AA + AG.</td><td>G</td><td></td>
				<td>OR</td><td>2.06</td></tr>
				
			<tr><td>49</td><td>rs1495741</td>
				<td>iguratimod</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AG + GG is associated with decreased clinical benefit to iguratimod in people with Arthritis, Rheumatoid as compared to genotype AA.</td><td>G</td><td>G</td>
				<td>OR</td><td>2.008</td></tr>
				
			<tr><td>50</td><td>rs1012335</td>
				<td>glatiramer acetate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to glatiramer acetate in people with Multiple Sclerosis as compared to allele C.</td><td>G</td><td>G</td>
				<td>OR</td><td>13.889</td></tr>
				
			<tr><td>51</td><td>rs9923231</td>
				<td>acenocoumarol</td><td>Dosage</td>
				<td>yes</td>
				<td>Genotype TT is associated with decreased dose of acenocoumarol as compared to genotypes CC + CT.</td><td>T</td><td></td>
				<td>OR</td><td>11.6</td></tr>
				
			<tr><td>52</td><td>rs2241766</td>
				<td>pioglitazone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GT is associated with increased response to pioglitazone in people with Diabetes Mellitus as compared to genotype TT.</td><td>G</td><td>T</td>
				<td>OR</td><td>11.483</td></tr>
				
			<tr><td>53</td><td>rs1329428</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.961</td></tr>
				
			<tr><td>54</td><td>rs2753326</td>
				<td>lamotrigine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased clinical benefit to lamotrigine in children with Epilepsy as compared to allele G.</td><td>A</td><td>A</td>
				<td>OR</td><td>1.923</td></tr>
				
			<tr><td>55</td><td>rs2753325</td>
				<td>lamotrigine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased clinical benefit to lamotrigine in children with Epilepsy as compared to allele G.</td><td>A</td><td>A</td>
				<td>OR</td><td>1.923</td></tr>
				
			<tr><td>56</td><td>rs10033900</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.887</td></tr>
				
			<tr><td>57</td><td>rs3763980</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to methotrexate in children with Arthritis, Juvenile Rheumatoid as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.887</td></tr>
				
			<tr><td>58</td><td>rs3781719</td>
				<td>botulinum toxin type a</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to botulinum toxin type a in women with Migraine NOS as compared to allele A.</td><td>G</td><td>G</td>
				<td>OR</td><td>1.88</td></tr>
				
			<tr><td>59</td><td>rs1126535</td>
				<td>adalimumab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to adalimumab in people with Arthritis, Rheumatoid as compared to allele C.</td><td>C</td><td></td>
				<td>OR</td><td>1.871</td></tr>
				
			<tr><td>60</td><td>rs2631370</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele C is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>1.87</td></tr>
				
			<tr><td>61</td><td>rs2010963</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele C is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele G.</td><td>C</td><td></td>
				<td>OR</td><td>1.852</td></tr>
				
			<tr><td>62</td><td>rs2804402</td>
				<td>methotrexate</td><td>Toxicity</td>
				<td>no</td>
				<td>Allele A is not associated with discontinuation of methotrexate in people with Arthritis, Rheumatoid as compared to allele G.</td><td>G</td><td></td>
				<td>HR</td><td>1.79</td></tr>
				
			<tr><td>63</td><td>rs1800544</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with decreased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>1.786</td></tr>
				
			<tr><td>64</td><td>rs8109525</td>
				<td>bupropion</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to bupropion in people with Tobacco Use Disorder as compared to genotypes AA + AG.</td><td>G</td><td></td>
				<td>OR</td><td>1.78</td></tr>
				
			<tr><td>65</td><td>rs7164594</td>
				<td>varenicline</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CT + TT are associated with increased response to varenicline in people with Tobacco Use Disorder as compared to genotype CC.</td><td>T</td><td></td>
				<td>OR</td><td>1.76</td></tr>
				
			<tr><td>66</td><td>rs1410996</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>1.754</td></tr>
				
			<tr><td>67</td><td>rs2032582</td>
				<td>imatinib</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to imatinib in people with Leukemia, Myelogenous, Chronic, BCR-ABL Positive as compared to allele A.</td><td>G</td><td>A</td>
				<td>OR</td><td>1.754</td></tr>
				
			<tr><td>68</td><td>rs7574865</td>
				<td>adalimumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to adalimumab in people with Arthritis, Rheumatoid as compared to allele G.</td><td>T</td><td></td>
				<td>OR</td><td>1.724</td></tr>
				
			<tr><td>69</td><td>rs628031</td>
				<td>imatinib</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AG is associated with decreased response to imatinib in people with Leukemia, Myelogenous, Chronic, BCR-ABL Positive as compared to genotype GG.</td><td></td><td></td>
				<td>OR</td><td>1.724</td></tr>
				
			<tr><td>70</td><td>rs6966038</td>
				<td>citalopram</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with increased response to citalopram in people with Depressive Disorder, Major as compared to allele A.</td><td>G</td><td>G</td>
				<td>OR</td><td>1.64</td></tr>
				
			<tr><td>71</td><td>rs10737680</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>1.613</td></tr>
				
			<tr><td>72</td><td>rs1800813</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele G is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele A.</td><td>G</td><td></td>
				<td>OR</td><td>1.587</td></tr>
				
			<tr><td>73</td><td>rs2236259</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methadone in people with Heroin Dependence as compared to allele T.</td><td>T</td><td></td>
				<td>OR</td><td>1.587</td></tr>
				
			<tr><td>74</td><td>rs12231740</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with decreased response to methotrexate in children with Arthritis, Juvenile Rheumatoid as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>1.587</td></tr>
				
			<tr><td>75</td><td>rs9937</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with decreased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes AG + GG.</td><td>G</td><td></td>
				<td>HR</td><td>1.53</td></tr>
				
			<tr><td>76</td><td>rs809736</td>
				<td>citalopram</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with increased response to citalopram in people with Depressive Disorder, Major as compared to allele A.</td><td>G</td><td>G</td>
				<td>OR</td><td>1.52</td></tr>
				
			<tr><td>77</td><td>rs28656907</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Genotype TT is associated with increased dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to genotypes CC + CT.</td><td>T</td><td></td>
				<td>OR</td><td>1.493</td></tr>
				
			<tr><td>78</td><td>rs249429</td>
				<td>metformin</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to metformin in people with Diabetes Mellitus as compared to allele C.</td><td>C</td><td>T</td>
				<td>OR</td><td>1.471</td></tr>
				
			<tr><td>79</td><td>rs7905446</td>
				<td>nortriptyline</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype TT is not associated with response to nortriptyline in people with Depression as compared to genotypes GG + GT.</td><td></td><td></td>
				<td>OR</td><td>1.441</td></tr>
				
			<tr><td>80</td><td>rs274717</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AG + GG are associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotype AA.</td><td></td><td></td>
				<td>HR</td><td>1.408</td></tr>
				
			<tr><td>81</td><td>rs73568641</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is associated with increased response to methadone in women with Opioid-Related Disorders as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.408</td></tr>
				
			<tr><td>82</td><td>rs3211371</td>
				<td>bupropion</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to bupropion in people with Tobacco Use Disorder as compared to allele C.</td><td>C</td><td>T</td>
				<td>OR</td><td>1.408</td></tr>
				
			<tr><td>83</td><td>rs2960306</td>
				<td>metoprolol</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype TT is not associated with decreased response to metoprolol in women with hypertensive nephrosclerosis as compared to genotype GG.</td><td></td><td></td>
				<td>HR</td><td>1.389</td></tr>
				
			<tr><td>84</td><td>rs4673990</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>1.37</td></tr>
				
			<tr><td>85</td><td>rs2236256</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methadone in people with Heroin Dependence as compared to allele A.</td><td>A</td><td></td>
				<td>OR</td><td>1.35</td></tr>
				
			<tr><td>86</td><td>rs3736544</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele G.</td><td>A</td><td></td>
				<td>OR</td><td>1.333</td></tr>
				
			<tr><td>87</td><td>rs2082940</td>
				<td>pioglitazone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype CC are not associated with response to pioglitazone in people with Diabetes Mellitus as compared to genotypes CT + TT.</td><td>C</td><td>T</td>
				<td>OR</td><td>1.303</td></tr>
				
			<tr><td>88</td><td>rs2306283</td>
				<td>atorvastatin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with increased response to atorvastatin in people with Hypercholesterolemia as compared to genotypes AG + GG.</td><td>G</td><td></td>
				<td>OR</td><td>1.29</td></tr>
				
			<tr><td>89</td><td>rs1800797</td>
				<td>peginterferon alfa-2a</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to peginterferon alfa-2a in people with Hepatitis C, Chronic as compared to allele A.</td><td></td><td></td>
				<td>RR</td><td>1.282</td></tr>
				
			<tr><td>90</td><td>rs1800795</td>
				<td>peginterferon alfa-2a</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to peginterferon alfa-2a in people with Hepatitis C, Chronic as compared to allele C.</td><td></td><td></td>
				<td>RR</td><td>1.282</td></tr>
				
			<tr><td>91</td><td>rs868755</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele G.</td><td>T</td><td></td>
				<td>OR</td><td>1.266</td></tr>
				
			<tr><td>92</td><td>rs1063538</td>
				<td>pioglitazone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CC + CT are not associated with response to pioglitazone in people with Diabetes Mellitus as compared to genotype TT.</td><td>C</td><td>T</td>
				<td>OR</td><td>1.255</td></tr>
				
			<tr><td>93</td><td>rs1800532</td>
				<td>venlafaxine</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with response to venlafaxine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.25</td></tr>
				
			<tr><td>94</td><td>rs3753394</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.235</td></tr>
				
			<tr><td>95</td><td>rs2740574</td>
				<td>amlodipine</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CT + TT are not associated with response to amlodipine in people with Hypertension as compared to genotype CC.</td><td></td><td></td>
				<td>HR</td><td>1.235</td></tr>
				
			<tr><td>96</td><td>rs762551</td>
				<td>imatinib</td><td>Dosage</td>
				<td>yes</td>
				<td>Genotype CC is associated with decreased dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to genotypes AA + AC.</td><td>C</td><td></td>
				<td>OR</td><td>1.235</td></tr>
				
			<tr><td>97</td><td>rs2295553</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.235</td></tr>
				
			<tr><td>98</td><td>rs6280</td>
				<td>clozapine</td><td>Efficacy</td>
				<td>not stated</td>
				<td>Allele T is not associated with response to clozapine in people with Schizophrenia as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>1.22</td></tr>
				
			<tr><td>99</td><td>rs17035723</td>
				<td>acamprosate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to acamprosate in people with Alcoholism as compared to allele C.</td><td></td><td></td>
				<td>HR</td><td>1.22</td></tr>
				
			<tr><td>100</td><td>rs3774261</td>
				<td>pioglitazone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AG + GG are not associated with response to pioglitazone in people with Diabetes Mellitus as compared to genotype AA.</td><td>G</td><td>A</td>
				<td>OR</td><td>1.209</td></tr>
				
			<tr><td>101</td><td>rs1329424</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.205</td></tr>
				
			<tr><td>102</td><td>rs2241767</td>
				<td>pioglitazone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype AA are not associated with response to pioglitazone in people with Diabetes Mellitus as compared to genotypes AG + GG.</td><td>G</td><td>A</td>
				<td>OR</td><td>1.201</td></tr>
				
			<tr><td>103</td><td>rs6356</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of methadone in people with Heroin Dependence as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>1.19</td></tr>
				
			<tr><td>104</td><td>rs2278749</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>1.18</td></tr>
				
			<tr><td>105</td><td>rs11212617</td>
				<td>metformin</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to metformin in people with Diabetes Mellitus, Type 2 as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>1.163</td></tr>
				
			<tr><td>106</td><td>rs2231142</td>
				<td>sulfasalazine</td><td>Other</td>
				<td>no</td>
				<td>Allele T is not associated with discontinuation of sulfasalazine in people with Arthritis, Rheumatoid as compared to allele G.</td><td>T</td><td></td>
				<td>HR</td><td>1.15</td></tr>
				
			<tr><td>107</td><td>rs683369</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele G is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele C.</td><td>G</td><td></td>
				<td>OR</td><td>1.136</td></tr>
				
			<tr><td>108</td><td>rs1042522</td>
				<td>oxaliplatin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CG + GG is associated with decreased response to oxaliplatin in people with Colorectal Neoplasms as compared to genotype CC.</td><td>G</td><td></td>
				<td>HR</td><td>1.136</td></tr>
				
			<tr><td>109</td><td>rs3821799</td>
				<td>pioglitazone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CT + TT are not associated with response to pioglitazone in people with Diabetes Mellitus as compared to genotype CC.</td><td>C</td><td>T</td>
				<td>OR</td><td>1.131</td></tr>
				
			<tr><td>110</td><td>rs6506569</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>1.124</td></tr>
				
			<tr><td>111</td><td>rs228697</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele C.</td><td>G</td><td></td>
				<td>OR</td><td>1.12</td></tr>
				
			<tr><td>112</td><td>rs293983</td>
				<td>belimumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to belimumab in people with Lupus erythematosus as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>1.111</td></tr>
				
			<tr><td>113</td><td>rs667282</td>
				<td>nicotine</td><td>Other</td>
				<td>no</td>
				<td>Allele C is not associated with exposure to nicotine in men as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>1.1</td></tr>
				
			<tr><td>114</td><td>rs6495307</td>
				<td>nicotine</td><td>Other</td>
				<td>no</td>
				<td>Allele T is not associated with exposure to nicotine in men as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>1.1</td></tr>
				
			<tr><td>115</td><td>rs6347</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Allele C is not associated with dose of methadone in people with Heroin Dependence as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>1.09</td></tr>
				
			<tr><td>116</td><td>rs2013169</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is associated with decreased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>1.087</td></tr>
				
			<tr><td>117</td><td>rs3808627</td>
				<td>heroin</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of heroin in people with Heroin Dependence as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>1.082</td></tr>
				
			<tr><td>118</td><td>rs9394992</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype CC is not associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes CT + TT.</td><td>T</td><td></td>
				<td>HR</td><td>1.08</td></tr>
				
			<tr><td>119</td><td>rs1801260</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele A.</td><td>G</td><td></td>
				<td>OR</td><td>1.07</td></tr>
				
			<tr><td>120</td><td>rs6902403</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methadone in people with Heroin Dependence as compared to allele T.</td><td>T</td><td></td>
				<td>OR</td><td>1.053</td></tr>
				
			<tr><td>121</td><td>rs25648</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>1.05</td></tr>
				
			<tr><td>122</td><td>rs3212964</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to cisplatin in women with Ovarian Neoplasms as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>1.05</td></tr>
				
			<tr><td>123</td><td>rs4483927</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with response to risperidone in people with Schizophrenia as compared to allele T.</td><td>G</td><td>G</td>
				<td>OR</td><td>1.033</td></tr>
				
			<tr><td>124</td><td>rs2071559</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele A is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele G.</td><td>A</td><td></td>
				<td>OR</td><td>1.02</td></tr>
				
			<tr><td>125</td><td>rs3393</td>
				<td>methotrexate</td><td>Toxicity</td>
				<td>no</td>
				<td>Allele C is not associated with discontinuation of methotrexate in people with Arthritis, Rheumatoid as compared to allele T.</td><td>C</td><td></td>
				<td>HR</td><td>1.01</td></tr>
				
			<tr><td>126</td><td>rs11568817</td>
				<td>venlafaxine</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to venlafaxine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>1.01</td></tr>
				
			<tr><td>127</td><td>rs5836788</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele del is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>1.0</td></tr>
				
			<tr><td>128</td><td>rs2230199</td>
				<td>eculizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CC + CG is not associated with response to eculizumab in people with paroxysmal nocturnal hemoglobinuria as compared to genotype GG.</td><td>C</td><td></td>
				<td>OR</td><td>1.0</td></tr>
				
			<tr><td>129</td><td>rs2072660</td>
				<td>varenicline</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CT + TT are not associated with response to varenicline in people with Tobacco Use Disorder as compared to genotype CC.</td><td></td><td></td>
				<td>OR</td><td>0.99</td></tr>
				
			<tr><td>130</td><td>rs1024323</td>
				<td>metoprolol</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype CC is not associated with decreased response to metoprolol in women with hypertensive nephrosclerosis as compared to genotype TT.</td><td></td><td></td>
				<td>HR</td><td>0.98</td></tr>
				
			<tr><td>131</td><td>rs1805054</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to risperidone in people with Schizophrenia as compared to allele C.</td><td>T</td><td>T</td>
				<td>OR</td><td>0.973</td></tr>
				
			<tr><td>132</td><td>rs2236225</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>0.971</td></tr>
				
			<tr><td>133</td><td>rs3212961</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to cisplatin in women with Ovarian Neoplasms as compared to allele G.</td><td>A</td><td></td>
				<td>OR</td><td>0.962</td></tr>
				
			<tr><td>134</td><td>rs743572</td>
				<td>abiraterone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AG + GG are not associated with response to abiraterone in men with Prostatic Neoplasms as compared to genotype AA.</td><td>G</td><td></td>
				<td>HR</td><td>0.96</td></tr>
				
			<tr><td>135</td><td>rs2298881</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to cisplatin in women with Ovarian Neoplasms as compared to allele C.</td><td>A</td><td></td>
				<td>OR</td><td>0.96</td></tr>
				
			<tr><td>136</td><td>rs3394</td>
				<td>methotrexate</td><td>Toxicity</td>
				<td>no</td>
				<td>Allele C is not associated with discontinuation of methotrexate in people with Arthritis, Rheumatoid as compared to allele T.</td><td>T</td><td></td>
				<td>HR</td><td>0.952</td></tr>
				
			<tr><td>137</td><td>rs228666</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>0.92</td></tr>
				
			<tr><td>138</td><td>rs4694362</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype TT is not associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes CT + TT.</td><td>T</td><td></td>
				<td>HR</td><td>0.92</td></tr>
				
			<tr><td>139</td><td>rs2236257</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methadone in people with Heroin Dependence as compared to allele G.</td><td>C</td><td></td>
				<td>OR</td><td>0.92</td></tr>
				
			<tr><td>140</td><td>rs615470</td>
				<td>nicotine</td><td>Other</td>
				<td>no</td>
				<td>Allele T is not associated with exposure to nicotine in men as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>0.909</td></tr>
				
			<tr><td>141</td><td>rs228642</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>0.901</td></tr>
				
			<tr><td>142</td><td>rs2279287</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>0.901</td></tr>
				
			<tr><td>143</td><td>rs2631367</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele C is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele G.</td><td>G</td><td></td>
				<td>OR</td><td>0.9</td></tr>
				
			<tr><td>144</td><td>rs130058</td>
				<td>venlafaxine</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to venlafaxine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.893</td></tr>
				
			<tr><td>145</td><td>rs4810083</td>
				<td>metformin</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to metformin in people with Diabetes Mellitus as compared to allele C.</td><td>C</td><td>T</td>
				<td>OR</td><td>0.89</td></tr>
				
			<tr><td>146</td><td>rs7624766</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.885</td></tr>
				
			<tr><td>147</td><td>rs1136287</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.885</td></tr>
				
			<tr><td>148</td><td>rs4343</td>
				<td>sertraline</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype GG is not associated with response to sertraline in people with Depressive Disorder.</td><td>G</td><td>G</td>
				<td>OR</td><td>0.885</td></tr>
				
			<tr><td>149</td><td>rs2640909</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>0.88</td></tr>
				
			<tr><td>150</td><td>rs228729</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with increased response to lithium in people with Bipolar Disorder as compared to allele C.</td><td>T</td><td></td>
				<td>OR</td><td>0.877</td></tr>
				
			<tr><td>151</td><td>rs1045280</td>
				<td>nicotine</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to nicotine in people with Tobacco Use Disorder as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>0.877</td></tr>
				
			<tr><td>152</td><td>rs9322453</td>
				<td>heroin</td><td>Dosage</td>
				<td>no</td>
				<td>Allele C is not associated with dose of heroin in people with Heroin Dependence as compared to allele G.</td><td>C</td><td></td>
				<td>OR</td><td>0.865</td></tr>
				
			<tr><td>153</td><td>rs129915</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Allele G is not associated with dose of methadone in people with Heroin Dependence as compared to allele A.</td><td>G</td><td></td>
				<td>OR</td><td>0.861</td></tr>
				
			<tr><td>154</td><td>rs6064463</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.855</td></tr>
				
			<tr><td>155</td><td>rs10011796</td>
				<td>allopurinol</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CT + TT are not associated with response to allopurinol in people with Gout as compared to genotype CC.</td><td>C</td><td>T</td>
				<td>OR</td><td>0.85</td></tr>
				
			<tr><td>156</td><td>rs735482</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to cisplatin in women with Ovarian Neoplasms as compared to allele A.</td><td>C</td><td></td>
				<td>OR</td><td>0.85</td></tr>
				
			<tr><td>157</td><td>rs4982133</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.84</td></tr>
				
			<tr><td>158</td><td>rs35592</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.84</td></tr>
				
			<tr><td>159</td><td>rs316009</td>
				<td>metformin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype TT is associated with increased response to metformin in people with Diabetes Mellitus as compared to genotypes CC + CT.</td><td></td><td></td>
				<td>OR</td><td>0.826</td></tr>
				
			<tr><td>160</td><td>rs1061115</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.82</td></tr>
				
			<tr><td>161</td><td>rs1870377</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele A.</td><td>T</td><td></td>
				<td>OR</td><td>0.813</td></tr>
				
			<tr><td>162</td><td>rs3842727</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of methadone in people with Heroin Dependence as compared to allele G.</td><td>G</td><td></td>
				<td>OR</td><td>0.807</td></tr>
				
			<tr><td>163</td><td>rs12979860</td>
				<td>peginterferon alfa-2b</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype CC is not associated with increased response to peginterferon alfa-2b in people with Hepatitis B, Chronic as compared to genotype CT.</td><td></td><td></td>
				<td>OR</td><td>0.806</td></tr>
				
			<tr><td>164</td><td>rs12693402</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to lithium in people with Bipolar Disorder as compared to allele T.</td><td>C</td><td>C</td>
				<td>OR</td><td>0.8</td></tr>
				
			<tr><td>165</td><td>rs4680</td>
				<td>levodopa</td><td>Dosage</td>
				<td>no</td>
				<td>Genotype AA is not associated with dose of levodopa in people with Parkinson Disease as compared to genotype GG.</td><td>A</td><td></td>
				<td>OR</td><td>0.8</td></tr>
				
			<tr><td>166</td><td>rs2107506</td>
				<td>lithium</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to lithium in people with Bipolar Disorder as compared to allele G.</td><td>C</td><td>C</td>
				<td>OR</td><td>0.787</td></tr>
				
			<tr><td>167</td><td>rs2650972</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.781</td></tr>
				
			<tr><td>168</td><td>rs7993418</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele G is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele A.</td><td>G</td><td></td>
				<td>OR</td><td>0.769</td></tr>
				
			<tr><td>169</td><td>rs10770140</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of methadone in people with Heroin Dependence as compared to allele C.</td><td>C</td><td></td>
				<td>OR</td><td>0.766</td></tr>
				
			<tr><td>170</td><td>rs2160734</td>
				<td>acamprosate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to acamprosate in people with Alcoholism as compared to allele T.</td><td>C</td><td>C</td>
				<td>HR</td><td>0.763</td></tr>
				
			<tr><td>171</td><td>rs2160733</td>
				<td>acamprosate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to acamprosate in people with Alcoholism as compared to allele A.</td><td>C</td><td>A</td>
				<td>HR</td><td>0.76</td></tr>
				
			<tr><td>172</td><td>rs3787429</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele T is not associated with response to risperidone in people with Schizophrenia as compared to allele C.</td><td>T</td><td>T</td>
				<td>OR</td><td>0.756</td></tr>
				
			<tr><td>173</td><td>rs362272</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.746</td></tr>
				
			<tr><td>174</td><td>rs162040</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with response to methotrexate in people with Arthritis, Rheumatoid as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.741</td></tr>
				
			<tr><td>175</td><td>rs10770141</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Allele G is not associated with dose of methadone in people with Heroin Dependence as compared to allele A.</td><td>A</td><td></td>
				<td>OR</td><td>0.73</td></tr>
				
			<tr><td>176</td><td>rs2058878</td>
				<td>acamprosate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased response to acamprosate in people with Alcoholism as compared to allele T.</td><td>A</td><td>T</td>
				<td>HR</td><td>0.72</td></tr>
				
			<tr><td>177</td><td>rs929740</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.714</td></tr>
				
			<tr><td>178</td><td>rs2236258</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to methadone in people with Heroin Dependence as compared to allele T.</td><td>C</td><td></td>
				<td>OR</td><td>0.714</td></tr>
				
			<tr><td>179</td><td>rs6973474</td>
				<td>buprenorphine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to buprenorphine in people with Opioid-Related Disorders as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.712</td></tr>
				
			<tr><td>180</td><td>rs4291</td>
				<td>sertraline</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype TT is not associated with response to sertraline in people with Depressive Disorder.</td><td>T</td><td>T</td>
				<td>OR</td><td>0.709</td></tr>
				
			<tr><td>181</td><td>rs1050152</td>
				<td>imatinib</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype TT is not associated with response to imatinib in people with Leukemia, Myelogenous, Chronic, BCR-ABL Positive as compared to genotypes CC + CT.</td><td>T</td><td></td>
				<td>HR</td><td>0.7</td></tr>
				
			<tr><td>182</td><td>rs1042028</td>
				<td>tamoxifen</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype TT is associated with decreased response to tamoxifen in women with Breast Neoplasms as compared to genotypes CC + CT.</td><td>T</td><td></td>
				<td>HR</td><td>0.7</td></tr>
				
			<tr><td>183</td><td>rs1933437</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele G is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele A.</td><td>G</td><td></td>
				<td>OR</td><td>0.699</td></tr>
				
			<tr><td>184</td><td>rs1061735</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.685</td></tr>
				
			<tr><td>185</td><td>rs11553441</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.685</td></tr>
				
			<tr><td>186</td><td>rs11782370</td>
				<td>buprenorphine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to buprenorphine in people with Opioid-Related Disorders as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.674</td></tr>
				
			<tr><td>187</td><td>rs274713</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes GT + TT.</td><td></td><td></td>
				<td>HR</td><td>0.671</td></tr>
				
			<tr><td>188</td><td>rs465646</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AG + GG are not associated with increased response to cisplatin in people with Osteosarcoma as compared to genotype AA.</td><td>G</td><td></td>
				<td>OR</td><td>0.667</td></tr>
				
			<tr><td>189</td><td>rs1799836</td>
				<td>levodopa</td><td>Dosage</td>
				<td>no</td>
				<td>Genotype TT is not associated with dose of levodopa in women with Parkinson Disease as compared to genotype CC.</td><td>T</td><td></td>
				<td>OR</td><td>0.667</td></tr>
				
			<tr><td>190</td><td>rs9032</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.658</td></tr>
				
			<tr><td>191</td><td>rs1143634</td>
				<td>pantoprazole</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype AA is not associated with increased response to pantoprazole in people with Helicobacter Infections as compared to genotypes AG + GG.</td><td></td><td></td>
				<td>OR</td><td>0.654</td></tr>
				
			<tr><td>192</td><td>rs1061170</td>
				<td>photodynamic therapy</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype TT is not associated with response to photodynamic therapy in people with Macular Degeneration as compared to genotype CC.</td><td>C</td><td></td>
				<td>OR</td><td>0.637</td></tr>
				
			<tr><td>193</td><td>rs17655</td>
				<td>platinum</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to platinum in people with Carcinoma, Non-Small-Cell Lung as compared to genotype CC.</td><td></td><td></td>
				<td>OR</td><td>0.637</td></tr>
				
			<tr><td>194</td><td>rs3803300</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to risperidone in people with Schizophrenia as compared to allele T.</td><td>C</td><td>C</td>
				<td>OR</td><td>0.628</td></tr>
				
			<tr><td>195</td><td>rs610604</td>
				<td>ustekinumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype GG is not associated with response to ustekinumab in people with Psoriasis as compared to genotype TT.</td><td>G</td><td></td>
				<td>OR</td><td>0.625</td></tr>
				
			<tr><td>196</td><td>rs12248560</td>
				<td>clopidogrel</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CT + TT is not associated with response to clopidogrel in people with Coronary Artery Disease as compared to genotype CC.</td><td>T</td><td></td>
				<td>OR</td><td>0.62</td></tr>
				
			<tr><td>197</td><td>rs11559290</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.617</td></tr>
				
			<tr><td>198</td><td>rs2229193</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype CC is not associated with response to methylphenidate in children with Attention Deficit Disorder with Hyperactivity as compared to genotype CT.</td><td>C</td><td></td>
				<td>OR</td><td>0.617</td></tr>
				
			<tr><td>199</td><td>rs7205113</td>
				<td>buprenorphine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to buprenorphine in people with Opioid-Related Disorders as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.607</td></tr>
				
			<tr><td>200</td><td>rs4562</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.606</td></tr>
				
			<tr><td>201</td><td>rs2242446</td>
				<td>venlafaxine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to venlafaxine in people with Depressive Disorder, Major as compared to genotypes CT + TT.</td><td>C</td><td></td>
				<td>OR</td><td>0.599</td></tr>
				
			<tr><td>202</td><td>rs1044396</td>
				<td>varenicline</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with decreased response to varenicline in people with Tobacco Use Disorder as compared to genotypes AA + AG.</td><td></td><td></td>
				<td>OR</td><td>0.599</td></tr>
				
			<tr><td>203</td><td>rs4358872</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Genotype GG is associated with decreased dose of methadone in people with Heroin Dependence as compared to genotypes GT + TT.</td><td></td><td></td>
				<td>OR</td><td>0.59</td></tr>
				
			<tr><td>204</td><td>rs1045642</td>
				<td>sunitinib</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with decreased response to sunitinib in people with Carcinoma, Renal Cell as compared to genotypes AG + GG.</td><td>A</td><td></td>
				<td>HR</td><td>0.588</td></tr>
				
			<tr><td>205</td><td>rs581111</td>
				<td>buprenorphine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AG is associated with decreased response to buprenorphine in women with Opioid-Related Disorders as compared to genotype GG.</td><td></td><td></td>
				<td>RR</td><td>0.581</td></tr>
				
			<tr><td>206</td><td>rs944050</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype TT is associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes CC + CT.</td><td></td><td></td>
				<td>HR</td><td>0.581</td></tr>
				
			<tr><td>207</td><td>rs4805162</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.581</td></tr>
				
			<tr><td>208</td><td>rs2336219</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AG + GG are associated with increased response to cisplatin in women with Ovarian Neoplasms as compared to genotype AA.</td><td>A</td><td></td>
				<td>OR</td><td>0.56</td></tr>
				
			<tr><td>209</td><td>rs529520</td>
				<td>buprenorphine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with decreased response to buprenorphine in women with Opioid-Related Disorders as compared to genotype CC.</td><td></td><td></td>
				<td>RR</td><td>0.556</td></tr>
				
			<tr><td>210</td><td>rs675026</td>
				<td>methadone</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele G is not associated with response to methadone in people with Heroin Dependence as compared to allele A.</td><td>A</td><td></td>
				<td>OR</td><td>0.552</td></tr>
				
			<tr><td>211</td><td>rs7719775</td>
				<td>platinum</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype AA is associated with decreased response to platinum in people with Carcinoma, Non-Small-Cell Lung as compared to genotype GG.</td><td></td><td></td>
				<td>OR</td><td>0.549</td></tr>
				
			<tr><td>212</td><td>rs1751034</td>
				<td>tenofovir</td><td>Toxicity</td>
				<td>yes</td>
				<td>Genotype TT is associated with increased concentrations of tenofovir in people with HIV Infections as compared to genotypes CC + CT.</td><td></td><td></td>
				<td>OR</td><td>0.541</td></tr>
				
			<tr><td>213</td><td>rs1800810</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele G is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele C.</td><td>G</td><td></td>
				<td>OR</td><td>0.54</td></tr>
				
			<tr><td>214</td><td>rs4673993</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with increased response to methotrexate in people with Arthritis, Rheumatoid as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.532</td></tr>
				
			<tr><td>215</td><td>rs1044457</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes CT + TT.</td><td></td><td></td>
				<td>HR</td><td>0.526</td></tr>
				
			<tr><td>216</td><td>rs2153628</td>
				<td>indomethacin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to indomethacin as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.521</td></tr>
				
			<tr><td>217</td><td>rs3087386</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AA + AG are not associated with increased response to cisplatin in people with Osteosarcoma as compared to genotype GG.</td><td>A</td><td></td>
				<td>OR</td><td>0.51</td></tr>
				
			<tr><td>218</td><td>rs6265</td>
				<td>olanzapine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to olanzapine in people with Schizophrenia as compared to genotypes CT + TT.</td><td></td><td></td>
				<td>OR</td><td>0.505</td></tr>
				
			<tr><td>219</td><td>rs1800812</td>
				<td>imatinib</td><td>Dosage</td>
				<td>no</td>
				<td>Allele T is not associated with dose of imatinib in people with Gastrointestinal Stromal Tumors as compared to allele G.</td><td>T</td><td></td>
				<td>OR</td><td>0.5</td></tr>
				
			<tr><td>220</td><td>rs2305948</td>
				<td>sunitinib</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype CT is not associated with response to sunitinib in people with Carcinoma, Renal Cell as compared to genotype CC.</td><td>T</td><td>C</td>
				<td>OR</td><td>0.5</td></tr>
				
			<tr><td>221</td><td>rs1128503</td>
				<td>sunitinib</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype AA is not associated with response to sunitinib in people with Carcinoma, Renal Cell as compared to genotypes AG + GG.</td><td>G</td><td>A</td>
				<td>OR</td><td>0.5</td></tr>
				
			<tr><td>222</td><td>rs6878291</td>
				<td>platinum</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with decreased response to platinum in people with Carcinoma, Non-Small-Cell Lung as compared to genotype AA.</td><td>G</td><td>G</td>
				<td>OR</td><td>0.49</td></tr>
				
			<tr><td>223</td><td>rs776746</td>
				<td>sunitinib</td><td>Dosage</td>
				<td>yes</td>
				<td>Allele T is associated with decreased dose of sunitinib in people with Carcinoma, Renal Cell as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.49</td></tr>
				
			<tr><td>224</td><td>rs2053044</td>
				<td>ramipril</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AG is associated with increased response to ramipril in people with Hypertension as compared to genotype GG.</td><td></td><td></td>
				<td>HR</td><td>0.478</td></tr>
				
			<tr><td>225</td><td>rs10835210</td>
				<td>methadone</td><td>Dosage</td>
				<td>no</td>
				<td>Genotype CC is associated with increased dose of methadone in people with Heroin Dependence as compared to genotypes AA + AC.</td><td></td><td></td>
				<td>OR</td><td>0.467</td></tr>
				
			<tr><td>226</td><td>rs10465180</td>
				<td>clozapine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with decreased response to clozapine in people with Schizophrenia as compared to genotypes CT + TT.</td><td></td><td></td>
				<td>OR</td><td>0.465</td></tr>
				
			<tr><td>227</td><td>rs4149117</td>
				<td>imatinib</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes GG + GT is associated with decreased response to imatinib in people with Leukemia, Myelogenous, Chronic, BCR-ABL Positive as compared to genotype TT.</td><td></td><td></td>
				<td>OR</td><td>0.461</td></tr>
				
			<tr><td>228</td><td>rs7311358</td>
				<td>imatinib</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AG is associated with decreased response to imatinib in people with Leukemia, Myelogenous, Chronic, BCR-ABL Positive as compared to genotype GG.</td><td></td><td></td>
				<td>OR</td><td>0.461</td></tr>
				
			<tr><td>229</td><td>rs1051266</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CT + TT is associated with increased response to methotrexate in people with Arthritis, Rheumatoid as compared to genotype CC.</td><td>T</td><td></td>
				<td>OR</td><td>0.455</td></tr>
				
			<tr><td>230</td><td>rs2299214</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CC + CT is associated with decreased response to risperidone in people with Schizophrenia as compared to genotype TT.</td><td></td><td></td>
				<td>OR</td><td>0.444</td></tr>
				
			<tr><td>231</td><td>rs2886059</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is associated with increased response to methylphenidate in people with Attention Deficit Disorder with Hyperactivity.</td><td></td><td></td>
				<td>OR</td><td>0.435</td></tr>
				
			<tr><td>232</td><td>rs523349</td>
				<td>abiraterone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to abiraterone in men with Prostatic Neoplasms as compared to genotypes CG + GG.</td><td>C</td><td></td>
				<td>HR</td><td>0.43</td></tr>
				
			<tr><td>233</td><td>rs2838958</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with decreased response to methotrexate in children with Precursor Cell Lymphoblastic Leukemia-Lymphoma as compared to genotypes AG + GG.</td><td></td><td></td>
				<td>HR</td><td>0.427</td></tr>
				
			<tr><td>234</td><td>rs6545816</td>
				<td>hydroxyurea</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele A is not associated with increased response to hydroxyurea in people with beta-Thalassemia.</td><td>A</td><td>A</td>
				<td>OR</td><td>0.417</td></tr>
				
			<tr><td>235</td><td>rs10485057</td>
				<td>ethanol</td><td>Dosage</td>
				<td>yes</td>
				<td>Genotype AA is associated with increased dose of ethanol as compared to genotype AG.</td><td></td><td></td>
				<td>OR</td><td>0.417</td></tr>
				
			<tr><td>236</td><td>rs1799983</td>
				<td>salvianolic acid b</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to salvianolic acid b in people with Coronary Disease as compared to genotypes GT + TT.</td><td>T</td><td></td>
				<td>OR</td><td>0.408</td></tr>
				
			<tr><td>237</td><td>rs3025039</td>
				<td>ranibizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to ranibizumab in people with Macular Degeneration as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.407</td></tr>
				
			<tr><td>238</td><td>rs4938013</td>
				<td>clozapine</td><td>Efficacy</td>
				<td>no</td>
				<td>Allele C is not associated with response to clozapine in people with Schizophrenia as compared to allele A.</td><td>A</td><td></td>
				<td>OR</td><td>0.402</td></tr>
				
			<tr><td>239</td><td>rs12610827</td>
				<td>olanzapine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased clinical benefit to olanzapine in people with Schizophrenia as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>0.402</td></tr>
				
			<tr><td>240</td><td>rs12507552</td>
				<td>gemcitabine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to gemcitabine in people with Pancreatic Neoplasms as compared to genotypes CT + TT.</td><td></td><td></td>
				<td>HR</td><td>0.395</td></tr>
				
			<tr><td>241</td><td>rs462779</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AG + GG are associated with decreased response to cisplatin in people with Osteosarcoma as compared to genotype AA.</td><td>G</td><td></td>
				<td>OR</td><td>0.385</td></tr>
				
			<tr><td>242</td><td>rs1800470</td>
				<td>rituximab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AG is associated with increased response to rituximab in people with Arthritis, Rheumatoid as compared to genotype AA.</td><td></td><td></td>
				<td>OR</td><td>0.385</td></tr>
				
			<tr><td>243</td><td>rs2233945</td>
				<td>etanercept</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with decreased response to etanercept in people with Arthritis, Rheumatoid as compared to allele A.</td><td>A</td><td></td>
				<td>OR</td><td>0.37</td></tr>
				
			<tr><td>244</td><td>rs7703002</td>
				<td>platinum</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype AA is associated with decreased response to platinum in people with Carcinoma, Non-Small-Cell Lung as compared to genotype CC.</td><td></td><td></td>
				<td>OR</td><td>0.366</td></tr>
				
			<tr><td>245</td><td>rs3077</td>
				<td>peginterferon alfa-2b</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to peginterferon alfa-2b in people with Hepatitis B, Chronic as compared to genotypes AA + AG.</td><td></td><td></td>
				<td>OR</td><td>0.36</td></tr>
				
			<tr><td>246</td><td>rs1731017</td>
				<td>valproic acid</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes AA + AG is associated with increased resistance to valproic acid in people with Epilepsy as compared to genotype GG.</td><td>A</td><td>A</td>
				<td>OR</td><td>0.358</td></tr>
				
			<tr><td>247</td><td>rs3761548</td>
				<td>methotrexate</td><td>Other</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to methotrexate in people with Psoriasis as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.355</td></tr>
				
			<tr><td>248</td><td>rs11615</td>
				<td>cisplatin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to cisplatin in women with Ovarian Neoplasms as compared to allele G.</td><td>A</td><td></td>
				<td>OR</td><td>0.348</td></tr>
				
			<tr><td>249</td><td>rs6785930</td>
				<td>clopidogrel</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AG is associated with increased resistance to clopidogrel as compared to genotype GG.</td><td></td><td></td>
				<td>OR</td><td>0.346</td></tr>
				
			<tr><td>250</td><td>rs324026</td>
				<td>olanzapine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with increased clinical benefit to olanzapine in people with Schizophrenia as compared to allele T.</td><td>C</td><td>C</td>
				<td>OR</td><td>0.345</td></tr>
				
			<tr><td>251</td><td>rs28386840</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AT + TT are associated with increased response to methylphenidate in children with Attention Deficit Disorder with Hyperactivity as compared to genotype AA.</td><td></td><td></td>
				<td>OR</td><td>0.341</td></tr>
				
			<tr><td>252</td><td>rs1063537</td>
				<td>pioglitazone</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotype CC are not associated with response to pioglitazone in people with Diabetes Mellitus as compared to genotypes CT + TT.</td><td>C</td><td>T</td>
				<td>OR</td><td>0.336</td></tr>
				
			<tr><td>253</td><td>rs1232027</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased response to methotrexate in people with Arthritis, Psoriatic as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>0.334</td></tr>
				
			<tr><td>254</td><td>rs12708954</td>
				<td>atomoxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased response to atomoxetine in children with Attention Deficit Disorder with Hyperactivity as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.323</td></tr>
				
			<tr><td>255</td><td>rs2108622</td>
				<td>acenocoumarol</td><td>Dosage</td>
				<td>yes</td>
				<td>Allele T is associated with increased dose of acenocoumarol in people with Atrial Fibrillation as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.321</td></tr>
				
			<tr><td>256</td><td>rs4253778</td>
				<td>fenofibrate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with increased response to fenofibrate in people with Diabetes Mellitus as compared to genotypes CC + CG.</td><td></td><td></td>
				<td>OR</td><td>0.313</td></tr>
				
			<tr><td>257</td><td>rs1544105</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CT + TT are associated with decreased response to methotrexate in people with Arthritis, Rheumatoid as compared to genotype CC.</td><td></td><td></td>
				<td>OR</td><td>0.288</td></tr>
				
			<tr><td>258</td><td>rs7079</td>
				<td>benazepril</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with response to benazepril in women with Hypertension as compared to genotype TT.</td><td></td><td></td>
				<td>OR</td><td>0.286</td></tr>
				
			<tr><td>259</td><td>rs2640543</td>
				<td>benazepril</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with response to benazepril in women with Hypertension as compared to genotype AA.</td><td></td><td></td>
				<td>OR</td><td>0.286</td></tr>
				
			<tr><td>260</td><td>rs4646</td>
				<td>tamoxifen</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with decreased response to tamoxifen in women with Breast Neoplasms and Menopause as compared to genotypes AC + CC.</td><td>A</td><td></td>
				<td>HR</td><td>0.277</td></tr>
				
			<tr><td>261</td><td>rs6809699</td>
				<td>clopidogrel</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AA + AC is associated with increased resistance to clopidogrel as compared to genotype CC.</td><td></td><td></td>
				<td>OR</td><td>0.272</td></tr>
				
			<tr><td>262</td><td>rs3792269</td>
				<td>metformin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to metformin in people with Diabetes Mellitus as compared to allele A.</td><td>G</td><td>A</td>
				<td>OR</td><td>0.27</td></tr>
				
			<tr><td>263</td><td>rs79663003</td>
				<td>lithium</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to lithium in people with Bipolar Disorder as compared to allele C.</td><td></td><td></td>
				<td>HR</td><td>0.262</td></tr>
				
			<tr><td>264</td><td>rs78015114</td>
				<td>lithium</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to lithium in people with Bipolar Disorder as compared to allele C.</td><td></td><td></td>
				<td>HR</td><td>0.262</td></tr>
				
			<tr><td>265</td><td>rs74795342</td>
				<td>lithium</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to lithium in people with Bipolar Disorder as compared to allele A.</td><td></td><td></td>
				<td>HR</td><td>0.262</td></tr>
				
			<tr><td>266</td><td>rs75222709</td>
				<td>lithium</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to lithium in people with Bipolar Disorder as compared to allele G.</td><td></td><td></td>
				<td>HR</td><td>0.262</td></tr>
				
			<tr><td>267</td><td>rs11269124</td>
				<td>clonidine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes GGGGAGCTTTCCCAGAGACCC/del + del/del are associated with increased response to clonidine in people with Liver Cirrhosis as compared to genotype GGGGAGCTTTCCCAGAGACCC/GGGGAGCTTTCCCAGAGACCC.</td><td></td><td></td>
				<td>OR</td><td>0.258</td></tr>
				
			<tr><td>268</td><td>rs1043550</td>
				<td>acenocoumarol</td><td>Dosage</td>
				<td>yes</td>
				<td>Allele G is associated with increased dose of acenocoumarol in people with Atrial Fibrillation as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.253</td></tr>
				
			<tr><td>269</td><td>rs10248420</td>
				<td>clozapine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with decreased response to clozapine in people with Schizophrenia as compared to allele G.</td><td>A</td><td></td>
				<td>OR</td><td>0.244</td></tr>
				
			<tr><td>270</td><td>rs7625956</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.243</td></tr>
				
			<tr><td>271</td><td>rs4858478</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.243</td></tr>
				
			<tr><td>272</td><td>rs10124893</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.239</td></tr>
				
			<tr><td>273</td><td>rs25487</td>
				<td>fluorouracil</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to fluorouracil in people with Rectal Neoplasms as compared to genotype CT.</td><td></td><td></td>
				<td>OR</td><td>0.239</td></tr>
				
			<tr><td>274</td><td>rs3784864</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AG + GG are associated with decreased response to methotrexate in people with Arthritis, Rheumatoid as compared to genotype AA.</td><td></td><td></td>
				<td>OR</td><td>0.236</td></tr>
				
			<tr><td>275</td><td>rs2070762</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with decreased response to methylphenidate in children with Attention Deficit Disorder with Hyperactivity as compared to genotypes AA + AG.</td><td></td><td></td>
				<td>OR</td><td>0.231</td></tr>
				
			<tr><td>276</td><td>rs12995526</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CT + TT is associated with decreased response to methotrexate in people with Arthritis, Rheumatoid as compared to genotype CC.</td><td>C</td><td>C</td>
				<td>OR</td><td>0.23</td></tr>
				
			<tr><td>277</td><td>rs12502866</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.229</td></tr>
				
			<tr><td>278</td><td>rs3811381</td>
				<td>eculizumab</td><td>Efficacy</td>
				<td>no</td>
				<td>Genotypes CG + GG is not associated with response to eculizumab in people with paroxysmal nocturnal hemoglobinuria as compared to genotype CC.</td><td>C</td><td></td>
				<td>OR</td><td>0.227</td></tr>
				
			<tr><td>279</td><td>rs10123866</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.227</td></tr>
				
			<tr><td>280</td><td>rs7472</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.227</td></tr>
				
			<tr><td>281</td><td>rs7035619</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.227</td></tr>
				
			<tr><td>282</td><td>rs6479008</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.227</td></tr>
				
			<tr><td>283</td><td>rs10989064</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.227</td></tr>
				
			<tr><td>284</td><td>rs9873889</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.216</td></tr>
				
			<tr><td>285</td><td>rs9879065</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.216</td></tr>
				
			<tr><td>286</td><td>rs3213422</td>
				<td>leflunomide</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased clinical benefit to leflunomide in people with Arthritis, Rheumatoid as compared to genotypes AA + AC.</td><td>A</td><td></td>
				<td>OR</td><td>0.212</td></tr>
				
			<tr><td>287</td><td>rs5744174</td>
				<td>ustekinumab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AG + GG is associated with increased response to ustekinumab in people with Psoriasis as compared to genotype AA.</td><td>A</td><td></td>
				<td>OR</td><td>0.19</td></tr>
				
			<tr><td>288</td><td>rs11231809</td>
				<td>methotrexate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AT + TT are associated with decreased response to methotrexate in people with Arthritis, Rheumatoid as compared to genotype AA.</td><td></td><td></td>
				<td>OR</td><td>0.186</td></tr>
				
			<tr><td>289</td><td>rs7627954</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to risperidone in people with Schizophrenia as compared to genotypes CT + TT.</td><td>C</td><td>C</td>
				<td>OR</td><td>0.18</td></tr>
				
			<tr><td>290</td><td>rs2088885</td>
				<td>risperidone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with increased response to risperidone in people with Schizophrenia as compared to genotypes AC + CC.</td><td>A</td><td>A</td>
				<td>OR</td><td>0.17</td></tr>
				
			<tr><td>291</td><td>rs1427407</td>
				<td>hydroxyurea</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to hydroxyurea in people with beta-Thalassemia as compared to allele G.</td><td>T</td><td>T</td>
				<td>OR</td><td>0.164</td></tr>
				
			<tr><td>292</td><td>rs6729738</td>
				<td>cannabidiol</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to cannabidiol in people with Epilepsy as compared to genotypes AA + AC.</td><td></td><td></td>
				<td>OR</td><td>0.149</td></tr>
				
			<tr><td>293</td><td>rs2290573</td>
				<td>imatinib</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype GG is associated with decreased response to imatinib in people with Leukemia, Myelogenous, Chronic, BCR-ABL Positive as compared to genotypes AA + AG.</td><td></td><td></td>
				<td>OR</td><td>0.149</td></tr>
				
			<tr><td>294</td><td>rs2274567</td>
				<td>eculizumab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes AG + GG is associated with decreased response to eculizumab in people with paroxysmal nocturnal hemoglobinuria as compared to genotype AA.</td><td>A</td><td></td>
				<td>OR</td><td>0.138</td></tr>
				
			<tr><td>295</td><td>rs9310657</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>296</td><td>rs9310658</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>297</td><td>rs7616119</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>298</td><td>rs7653345</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>299</td><td>rs9819548</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>300</td><td>rs13093500</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>301</td><td>rs9824595</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele T.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>302</td><td>rs4334661</td>
				<td>duloxetine</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele T is associated with increased response to duloxetine in people with Depressive Disorder, Major as compared to allele C.</td><td></td><td></td>
				<td>OR</td><td>0.115</td></tr>
				
			<tr><td>303</td><td>rs766432</td>
				<td>hydroxyurea</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with increased response to hydroxyurea in people with beta-Thalassemia as compared to allele A.</td><td>C</td><td>C</td>
				<td>OR</td><td>0.11</td></tr>
				
			<tr><td>304</td><td>rs10210302</td>
				<td>adalimumab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CT + TT is associated with increased response to adalimumab in people with Crohn Disease as compared to genotype CC.</td><td>C</td><td></td>
				<td>OR</td><td>0.106</td></tr>
				
			<tr><td>305</td><td>rs5882</td>
				<td>rosuvastatin</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with increased response to rosuvastatin as compared to allele A.</td><td>G</td><td></td>
				<td>OR</td><td>0.075</td></tr>
				
			<tr><td>306</td><td>rs2284411</td>
				<td>methylphenidate</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype CC is associated with increased response to methylphenidate in children with Attention Deficit Disorder with Hyperactivity as compared to genotypes CT + TT.</td><td>C</td><td></td>
				<td>OR</td><td>0.07</td></tr>
				
			<tr><td>307</td><td>rs9934438</td>
				<td>warfarin</td><td>Dosage</td>
				<td>yes</td>
				<td>Genotype AG is associated with increased dose of warfarin in people with Atrial Fibrillation, Cardiomyopathies, heart valve replacement, Peripheral Vascular Diseases, Pulmonary Embolism and Venous Thrombosis as compared to genotype GG.</td><td>A</td><td></td>
				<td>OR</td><td>0.06</td></tr>
				
			<tr><td>308</td><td>rs1570360</td>
				<td>sildenafil</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with decreased response to sildenafil in men with Erectile Dysfunction as compared to genotypes AG + GG.</td><td>A</td><td>A</td>
				<td>OR</td><td>0.05</td></tr>
				
			<tr><td>309</td><td>rs1967554</td>
				<td>methadone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with decreased response to methadone in people with Opioid-Related Disorders as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.049</td></tr>
				
			<tr><td>310</td><td>rs988748</td>
				<td>methadone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele C is associated with decreased response to methadone in people with Opioid-Related Disorders as compared to allele G.</td><td></td><td></td>
				<td>OR</td><td>0.049</td></tr>
				
			<tr><td>311</td><td>rs2030324</td>
				<td>methadone</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele G is associated with decreased response to methadone in people with Opioid-Related Disorders as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.049</td></tr>
				
			<tr><td>312</td><td>rs749671</td>
				<td>warfarin</td><td>Dosage</td>
				<td>yes</td>
				<td>Allele G is associated with increased dose of warfarin as compared to allele A.</td><td></td><td></td>
				<td>OR</td><td>0.049</td></tr>
				
			<tr><td>313</td><td>rs4671393</td>
				<td>hydroxyurea</td><td>Efficacy</td>
				<td>yes</td>
				<td>Allele A is associated with increased response to hydroxyurea in people with beta-Thalassemia as compared to allele G.</td><td>A</td><td>A</td>
				<td>OR</td><td>0.047</td></tr>
				
			<tr><td>314</td><td>rs396991</td>
				<td>rituximab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotype AA is associated with decreased response to rituximab in people with Neuromyelitis Optica as compared to genotype CC.</td><td>A</td><td></td>
				<td>OR</td><td>0.04</td></tr>
				
			<tr><td>315</td><td>rs191190</td>
				<td>ustekinumab</td><td>Efficacy</td>
				<td>yes</td>
				<td>Genotypes CC + CT is associated with decreased response to ustekinumab in people with Psoriasis as compared to genotype TT.</td><td></td><td></td>
				<td>OR</td><td>0.035</td></tr>
				
		</tbody>
	</table>
	<p></p>
</div>
	<h2 id="cancer" style="color:rgb(57, 145, 123);;">Major risks</h2>
	<button class="accordion accordion0 accordion1 major-risk module-button"><div></div> Description</button>
<div class="board variants panel panel-description major-risk">
	<p></p>
	<p>It is not enough to have &quot;centenarian&quot; genes to become one: You have to cut down Your health risks before. This section contains information about the risk of developing chronic age-related diseases: cardiovascular, cancer, mental health, chronic inflammation, bone and muscle diseases, and lung diseases.
	</p>
</div>
	<button class="accordion accordion-heading major-risk"  >Cancer Report</button> 

	<div class="basic-panel major-risk">
	<button class="accordion accordion0 accordion1 major-risk"><div></div>Description</button>
	<div class="board variants panel panel-description major-risk">
		<p>Hereditary Cancers are caused by a genetic defect that determines a higher-than-normal risk of developing cancer. Hereditary cancers can have an impact on longevity, as individuals with certain genetic mutations that increase the risk of cancer may have a shorter life expectancy. The report includes about  300 genes, related to cancer predisposition,  progression and tumor cell motility.</p>
		</div>

		<button class="accordion0 accordion accordion-result major-risk"><div></div>Results</button>
		<div class="content content-variants major-risk">
	<table class="ui selectable celled teal table table cancer cancer1">
		<thead>
		<tr><th style="width:1%" class="clickable title" onclick="collapseExpandAll(this, 'cancTBody')"><div class="ui tiny circular compact positive button">+</div></th><th>#</th>
				<th>Chrom</th><th>Position</th><th>Gene</th><th>RSID</th><th>cDNA Change</th><th>Zygosity</th>
				<th>Allele Frequency</th><th>Phenotype Name</th><th>Significance</th></tr>
		</thead>
		<tbody id="cancTBody">
			
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>1</td><td>chrX</td>
					<td>71132768</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=MED12">MED12</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.4416-20_4416-16dup</td><td>het</td><td>0.153809620734</td>
					<td>not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The initiation of transcription is controlled in part by a large protein assembly known as the preinitiation complex. A component of this preinitiation complex is a 1.2 MDa protein aggregate called Mediator. This Mediator component binds with a CDK8 subcomplex which contains the protein encoded by this gene, mediator complex subunit 12 (MED12), along with MED13, CDK8 kinase, and cyclin C. The CDK8 subcomplex modulates Mediator-polymerase II interactions and thereby regulates transcription initiation and reinitation rates. The MED12 protein is essential for activating CDK8 kinase. Defects in this gene cause X-linked Opitz-Kaveggia syndrome, also known as FG syndrome, and Lujan-Fryns syndrome. [provided by RefSeq, Aug 2009]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>2</td><td>chr22</td>
					<td>41179534</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=EP300">EP300</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*592dup</td><td>het</td><td>0.244418925126</td>
					<td>Rubinstein-Taybi syndrome due to CREBBP mutations</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes the adenovirus E1A-associated cellular p300 transcriptional co-activator protein. It functions as histone acetyltransferase that regulates transcription via chromatin remodeling and is important in the processes of cell proliferation and differentiation. It mediates cAMP-gene regulation by binding specifically to phosphorylated CREB protein. This gene has also been identified as a co-activator of HIF1A (hypoxia-inducible factor 1 alpha), and thus plays a role in the stimulation of hypoxia-induced genes such as VEGF. Defects in this gene are a cause of Rubinstein-Taybi syndrome and may also play a role in epithelial cancer. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>3</td><td>chr22</td>
					<td>29628628</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NF2">NF2</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.115-8101del</td><td>het</td><td>None</td>
					<td>Neurofibromatosis, type 2</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a protein that is similar to some members of the ERM (ezrin, radixin, moesin) family of proteins that link cytoskeletal components with proteins in the cell membrane. The encoded protein is involved in regulation of contact-dependent inhibition of cell proliferation and functions in cell-cell adhesion and transmembrane signaling. The encoded protein has been shown to interact with cell-surface proteins, proteins involved in cytoskeletal dynamics, and proteins involved in regulating ion transport. Disruption of this protein&apos;s function has been implicated in tumorigenesis and metastasis. Mutations in this gene are associated with neurofibromatosis type II which is characterized by nervous system and skin tumors and ocular abnormalities. [provided by RefSeq, May 2022]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>4</td><td>chr19</td>
					<td>17825901</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=JAK3">JAK3</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*841_*842dup</td><td>het</td><td>0.288112927192</td>
					<td>Severe combined immunodeficiency disease</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The protein encoded by this gene is a member of the Janus kinase (JAK) family of tyrosine kinases involved in cytokine receptor-mediated intracellular signal transduction. It is predominantly expressed in immune cells and transduces a signal in response to its activation via tyrosine phosphorylation by interleukin receptors. Mutations in this gene are associated with autosomal SCID (severe combined immunodeficiency disease). [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>5</td><td>chr18</td>
					<td>51084014</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SMAD4">SMAD4</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11662245">rs11662245</a></td>
					<td>c.*5547A>G</td><td>het</td><td>0.0331441422146</td>
					<td>Myhre syndrome|Generalized juvenile polyposis/juvenile polyposis coli|Juvenile polyposis/hereditary hemorrhagic telangiectasia syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of the Smad family of signal transduction proteins. Smad proteins are phosphorylated and activated by transmembrane serine-threonine receptor kinases in response to transforming growth factor (TGF)-beta signaling. The product of this gene forms homomeric complexes and heteromeric complexes with other activated Smad proteins, which then accumulate in the nucleus and regulate the transcription of target genes. This protein binds to DNA and recognizes an 8-bp palindromic sequence (GTCTAGAC) called the Smad-binding element (SBE). The protein acts as a tumor suppressor and inhibits epithelial cell proliferation. It may also have an inhibitory effect on tumors by reducing angiogenesis and increasing blood vessel hyperpermeability. The encoded protein is a crucial component of the bone morphogenetic protein signaling pathway. The Smad proteins are subject to complex regulation by post-translational modifications. Mutations or deletions in this gene have been shown to result in pancreatic cancer, juvenile polyposis syndrome, and hereditary hemorrhagic telangiectasia syndrome. [provided by RefSeq, May 2022]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>6</td><td>chr17</td>
					<td>68532646</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=PRKAR1A">PRKAR1A</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs533508012">rs533508012</a></td>
					<td>c.*2197G>A</td><td>het</td><td>0.000318511912346</td>
					<td>Acrodysostosis 1 with or without hormone resistance|Carney complex, type 1</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">cAMP is a signaling molecule important for a variety of cellular functions. cAMP exerts its effects by activating the cAMP-dependent protein kinase, which transduces the signal through phosphorylation of different target proteins. The inactive kinase holoenzyme is a tetramer composed of two regulatory and two catalytic subunits. cAMP causes the dissociation of the inactive holoenzyme into a dimer of regulatory subunits bound to four cAMP and two free monomeric catalytic subunits. Four different regulatory subunits and three catalytic subunits have been identified in humans. This gene encodes one of the regulatory subunits. This protein was found to be a tissue-specific extinguisher that down-regulates the expression of seven liver genes in hepatoma x fibroblast hybrids. Mutations in this gene cause Carney complex (CNC). This gene can fuse to the RET protooncogene by gene rearrangement and form the thyroid tumor-specific chimeric oncogene known as PTC2. A nonconventional nuclear localization sequence (NLS) has been found for this protein which suggests a role in DNA replication via the protein serving as a nuclear transport protein for the second subunit of the Replication Factor C (RFC40). Several alternatively spliced transcript variants encoding two different isoforms have been observed. [provided by RefSeq, Jan 2013]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>7</td><td>chr17</td>
					<td>61680540</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=BRIP1">BRIP1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs182264505">rs182264505</a></td>
					<td>c.*2756G>A</td><td>het</td><td>0.0111360921801</td>
					<td>Fanconi anemia complementation group J</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The protein encoded by this gene is a member of the RecQ DEAH helicase family and interacts with the BRCT repeats of breast cancer, type 1 (BRCA1). The bound complex is important in the normal double-strand break repair function of breast cancer, type 1 (BRCA1). This gene may be a target of germline cancer-inducing mutations. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>8</td><td>chr17</td>
					<td>61680481</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=BRIP1">BRIP1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*2815_*2816insAAGAAA</td><td>het</td><td>0.213558652041</td>
					<td>Breast neoplasm|Fanconi anemia|not specified</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The protein encoded by this gene is a member of the RecQ DEAH helicase family and interacts with the BRCT repeats of breast cancer, type 1 (BRCA1). The bound complex is important in the normal double-strand break repair function of breast cancer, type 1 (BRCA1). This gene may be a target of germline cancer-inducing mutations. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>9</td><td>chr17</td>
					<td>43044805</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=BRCA1">BRCA1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*872_*873del</td><td>het</td><td>0.279154987153</td>
					<td>Hereditary breast ovarian cancer syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a 190 kD nuclear phosphoprotein that plays a role in maintaining genomic stability, and it also acts as a tumor suppressor. The BRCA1 gene contains 22 exons spanning about 110 kb of DNA. The encoded protein combines with other tumor suppressors, DNA damage sensors, and signal transducers to form a large multi-subunit protein complex known as the BRCA1-associated genome surveillance complex (BASC). This gene product associates with RNA polymerase II, and through the C-terminal domain, also interacts with histone deacetylase complexes. This protein thus plays a role in transcription, DNA repair of double-stranded breaks, and recombination. Mutations in this gene are responsible for approximately 40% of inherited breast cancers and more than 80% of inherited breast and ovarian cancers. Alternative splicing plays a role in modulating the subcellular localization and physiological function of this gene. Many alternatively spliced transcript variants, some of which are disease-associated mutations, have been described for this gene, but the full-length natures of only some of these variants has been described. A related pseudogene, which is also located on chromosome 17, has been identified. [provided by RefSeq, May 2020]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>10</td><td>chr17</td>
					<td>7668837</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=TP53">TP53</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*772del</td><td>het</td><td>0.631307090039</td>
					<td>Li-Fraumeni syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a tumor suppressor protein containing transcriptional activation, DNA binding, and oligomerization domains. The encoded protein responds to diverse cellular stresses to regulate expression of target genes, thereby inducing cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. Mutations in this gene are associated with a variety of human cancers, including hereditary cancers such as Li-Fraumeni syndrome. Alternative splicing of this gene and the use of alternate promoters result in multiple transcript variants and isoforms. Additional isoforms have also been shown to result from the use of alternate translation initiation codons from identical transcript variants (PMIDs: 12032546, 20937277). [provided by RefSeq, Dec 2016]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>11</td><td>chr16</td>
					<td>68833709</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDH1">CDH1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*221dup</td><td>het</td><td>0.194419475655</td>
					<td>Hereditary diffuse gastric adenocarcinoma|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a classical cadherin of the cadherin superfamily. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the mature glycoprotein. This calcium-dependent cell-cell adhesion protein is comprised of five extracellular cadherin repeats, a transmembrane region and a highly conserved cytoplasmic tail. Mutations in this gene are correlated with gastric, breast, colorectal, thyroid and ovarian cancer. Loss of function of this gene is thought to contribute to cancer progression by increasing proliferation, invasion, and/or metastasis. The ectodomain of this protein mediates bacterial adhesion to mammalian cells and the cytoplasmic domain is required for internalization. This gene is present in a gene cluster with other members of the cadherin family on chromosome 16. [provided by RefSeq, Nov 2015]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>12</td><td>chr16</td>
					<td>9760375</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=GRIN2A">GRIN2A</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*2771_*2774dup</td><td>het</td><td>None</td>
					<td>Landau-Kleffner syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of the glutamate-gated ion channel protein family. The encoded protein is an N-methyl-D-aspartate (NMDA) receptor subunit. NMDA receptors are both ligand-gated and voltage-dependent, and are involved in long-term potentiation, an activity-dependent increase in the efficiency of synaptic transmission thought to underlie certain kinds of memory and learning. These receptors are permeable to calcium ions, and activation results in a calcium influx into post-synaptic cells, which results in the activation of several signaling cascades. Disruption of this gene is associated with focal epilepsy and speech disorder with or without cognitive disability. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>13</td><td>chr16</td>
					<td>9760375</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=GRIN2A">GRIN2A</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*2773_*2774dup</td><td>het</td><td>None</td>
					<td>Landau-Kleffner syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of the glutamate-gated ion channel protein family. The encoded protein is an N-methyl-D-aspartate (NMDA) receptor subunit. NMDA receptors are both ligand-gated and voltage-dependent, and are involved in long-term potentiation, an activity-dependent increase in the efficiency of synaptic transmission thought to underlie certain kinds of memory and learning. These receptors are permeable to calcium ions, and activation results in a calcium influx into post-synaptic cells, which results in the activation of several signaling cascades. Disruption of this gene is associated with focal epilepsy and speech disorder with or without cognitive disability. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>14</td><td>chr15</td>
					<td>98963214</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=IGF1R">IGF1R</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*5789del</td><td>hom</td><td>0.636829615567</td>
					<td>Growth delay due to insulin-like growth factor I resistance</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This receptor binds insulin-like growth factor with a high affinity. It has tyrosine kinase activity. The insulin-like growth factor I receptor plays a critical role in transformation events. Cleavage of the precursor generates alpha and beta subunits. It is highly overexpressed in most malignant tissues where it functions as an anti-apoptotic agent by enhancing cell survival. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, May 2014]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>15</td><td>chr15</td>
					<td>98959855</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=IGF1R">IGF1R</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*2426del</td><td>hom</td><td>0.635246216486</td>
					<td>Growth delay due to insulin-like growth factor I resistance</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This receptor binds insulin-like growth factor with a high affinity. It has tyrosine kinase activity. The insulin-like growth factor I receptor plays a critical role in transformation events. Cleavage of the precursor generates alpha and beta subunits. It is highly overexpressed in most malignant tissues where it functions as an anti-apoptotic agent by enhancing cell survival. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, May 2014]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>16</td><td>chr15</td>
					<td>89268467</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FANCI">FANCI</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs142906652">rs142906652</a></td>
					<td>c.824T>C</td><td>het</td><td>0.000569178120934</td>
					<td>Fanconi anemia complementation group I|Fanconi anemia|not specified|not provided</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group I. Alternative splicing results in two transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>17</td><td>chr15</td>
					<td>38352242</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SPRED1">SPRED1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*592_*595dup</td><td>het</td><td>0.12014978602</td>
					<td>Legius syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The protein encoded by this gene is a member of the Sprouty family of proteins and is phosphorylated by tyrosine kinase in response to several growth factors. The encoded protein can act as a homodimer or as a heterodimer with SPRED2 to regulate activation of the MAP kinase cascade. Defects in this gene are a cause of neurofibromatosis type 1-like syndrome (NFLS). [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>18</td><td>chr14</td>
					<td>103699416</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=XRCC3">XRCC3</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs861539">rs861539</a></td>
					<td>c.722C>T</td><td>het</td><td>0.290368047865</td>
					<td>Melanoma, cutaneous malignant, susceptibility to, 6</td><td>risk factor</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of the RecA/Rad51-related protein family that participates in homologous recombination to maintain chromosome stability and repair DNA damage. This gene functionally complements Chinese hamster irs1SF, a repair-deficient mutant that exhibits hypersensitivity to a number of different DNA-damaging agents and is chromosomally unstable. A rare microsatellite polymorphism in this gene is associated with cancer in patients of varying radiosensitivity. Alternatively spliced transcript variants encoding the same protein have been identified. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>19</td><td>chr14</td>
					<td>95090172</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DICER1">DICER1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*326del</td><td>hom</td><td>0.676011739265</td>
					<td>Pleuropulmonary blastoma</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a protein possessing an RNA helicase motif containing a DEXH box in its amino terminus and an RNA motif in the carboxy terminus. The encoded protein functions as a ribonuclease and is required by the RNA interference and small temporal RNA (stRNA) pathways to produce the active small RNA component that represses gene expression. This protein also acts as a strong antiviral agent with activity against RNA viruses, including the Zika and SARS-CoV-2 viruses. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2021]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>20</td><td>chr14</td>
					<td>45159081</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FANCM">FANCM</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.1397-3_1397-2del</td><td>het</td><td>0.161452677558</td>
					<td>Fanconi anemia|Spermatogenic failure 28|Premature ovarian failure 15|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group M. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>21</td><td>chr14</td>
					<td>36516818</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NKX2-1">NKX2-1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*460dup</td><td>het</td><td>0.425434156218</td>
					<td>Brain-lung-thyroid syndrome|Benign hereditary chorea</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a protein initially identified as a thyroid-specific transcription factor. The encoded protein binds to the thyroglobulin promoter and regulates the expression of thyroid-specific genes but has also been shown to regulate the expression of genes involved in morphogenesis. Mutations and deletions in this gene are associated with benign hereditary chorea, choreoathetosis, congenital hypothyroidism, and neonatal respiratory distress, and may be associated with thyroid cancer. Multiple transcript variants encoding different isoforms have been found for this gene. This gene shares the symbol/alias &apos;TTF1&apos; with another gene, transcription termination factor 1, which plays a role in ribosomal gene transcription. [provided by RefSeq, Feb 2014]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>22</td><td>chr12</td>
					<td>120994314</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=HNF1A">HNF1A</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs56348580">rs56348580</a></td>
					<td>c.864G>C</td><td>het</td><td>0.263675836592</td>
					<td>Maturity onset diabetes mellitus in young|Type 2 diabetes mellitus|Maturity-onset diabetes of the young type 3|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The protein encoded by this gene is a transcription factor required for the expression of several liver-specific genes. The encoded protein functions as a homodimer and binds to the inverted palindrome 5&apos;-GTTAATNATTAAC-3&apos;. Defects in this gene are a cause of maturity onset diabetes of the young type 3 (MODY3) and also can result in the appearance of hepatic adenomas. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Apr 2015]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>23</td><td>chr12</td>
					<td>114684072</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=TBX3">TBX3</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.-873_-872del</td><td>het</td><td>0.00711197880351</td>
					<td>Ulnar-mammary syndrome|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. This protein is a transcriptional repressor and is thought to play a role in the anterior/posterior axis of the tetrapod forelimb. Mutations in this gene cause ulnar-mammary syndrome, affecting limb, apocrine gland, tooth, hair, and genital development. Alternative splicing of this gene results in three transcript variants encoding different isoforms; however, the full length nature of one variant has not been determined. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>24</td><td>chr12</td>
					<td>4369971</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FGF23">FGF23</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*367_*372del</td><td>hom</td><td>0.173208942088</td>
					<td>Familial hyperphosphatemic tumoral calcinosis/hyperphosphatemic hyperostosis syndrome|Hypophosphatemic Rickets, Dominant</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of the fibroblast growth factor family of proteins, which possess broad mitogenic and cell survival activities and are involved in a variety of biological processes. The product of this gene regulates phosphate homeostasis and transport in the kidney. The full-length, functional protein may be deactivated via cleavage into N-terminal and C-terminal chains. Mutation of this cleavage site causes autosomal dominant hypophosphatemic rickets (ADHR). Mutations in this gene are also associated with hyperphosphatemic familial tumoral calcinosis (HFTC). [provided by RefSeq, Feb 2013]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>25</td><td>chr11</td>
					<td>64808033</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=MEN1">MEN1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs607969">rs607969</a></td>
					<td>c.512G>A</td><td>het</td><td>0.0122104144527</td>
					<td>Hyperparathyroidism|Multiple endocrine neoplasia, type 1|Hereditary cancer-predisposing syndrome|Multiple endocrine neoplasia|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes menin, a tumor suppressor associated with a syndrome known as multiple endocrine neoplasia type 1. Menin is a scaffold protein that functions in histone modification and epigenetic gene regulation. It is thought to regulate several pathways and processes by altering chromatin structure through the modification of histones. [provided by RefSeq, May 2019]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>26</td><td>chr11</td>
					<td>32388004</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=WT1">WT1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>None</td><td>het</td><td>0.227680763027</td>
					<td>Nephroblastoma|11p partial monosomy syndrome|Nephrotic syndrome, type 4|Meacham syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a transcription factor that contains four zinc-finger motifs at the C-terminus and a proline/glutamine-rich DNA-binding domain at the N-terminus. It has an essential role in the normal development of the urogenital system, and it is mutated in a small subset of patients with Wilms tumor. This gene exhibits complex tissue-specific and polymorphic imprinting pattern, with biallelic, and monoallelic expression from the maternal and paternal alleles in different tissues. Multiple transcript variants have been described. In several variants, there is evidence for the use of a non-AUG (CUG) translation initiation codon upstream of, and in-frame with the first AUG. Authors of PMID:7926762 also provide evidence that WT1 mRNA undergoes RNA editing in human and rat, and that this process is tissue-restricted and developmentally regulated. [provided by RefSeq, Mar 2015]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>27</td><td>chr11</td>
					<td>22623348</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FANCF">FANCF</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs45554234">rs45554234</a></td>
					<td>c.*1338dup</td><td>hom</td><td>0.959894071852</td>
					<td>Fanconi anemia</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group F. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>28</td><td>chr10</td>
					<td>87966903</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=PTEN">PTEN</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*1458_*1459del</td><td>het</td><td>0.231438812084</td>
					<td>PTEN hamartoma tumor syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene was identified as a tumor suppressor that is mutated in a large number of cancers at high frequency. The protein encoded by this gene is a phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase. It contains a tensin like domain as well as a catalytic domain similar to that of the dual specificity protein tyrosine phosphatases. Unlike most of the protein tyrosine phosphatases, this protein preferentially dephosphorylates phosphoinositide substrates. It negatively regulates intracellular levels of phosphatidylinositol-3,4,5-trisphosphate in cells and functions as a tumor suppressor by negatively regulating AKT/PKB signaling pathway. The use of a non-canonical (CUG) upstream initiation site produces a longer isoform that initiates translation with a leucine, and is thought to be preferentially associated with the mitochondrial inner membrane. This longer isoform may help regulate energy metabolism in the mitochondria. A pseudogene of this gene is found on chromosome 9. Alternative splicing and the use of multiple translation start codons results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2015]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>29</td><td>chr10</td>
					<td>14935470</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DCLRE1C">DCLRE1C</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs41297018">rs41297018</a></td>
					<td>c.457G>A</td><td>het</td><td>0.0104059854748</td>
					<td>Severe combined immunodeficiency due to DCLRE1C deficiency|Histiocytic medullary reticulosis|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a nuclear protein that is involved in V(D)J recombination and DNA repair. The encoded protein has single-strand-specific 5&apos;-3&apos; exonuclease activity; it also exhibits endonuclease activity on 5&apos; and 3&apos; overhangs and hairpins. The protein also functions in the regulation of the cell cycle in response to DNA damage. Mutations in this gene can cause Athabascan-type severe combined immunodeficiency (SCIDA) and Omenn syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>30</td><td>chr10</td>
					<td>8074279</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=GATA3">GATA3</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*265dup</td><td>het</td><td>0.736817420436</td>
					<td>Hypoparathyroidism, deafness, renal disease syndrome|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a protein which belongs to the GATA family of transcription factors. The protein contains two GATA-type zinc fingers and is an important regulator of T-cell development and plays an important role in endothelial cell biology. Defects in this gene are the cause of hypoparathyroidism with sensorineural deafness and renal dysplasia. [provided by RefSeq, Nov 2009]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>31</td><td>chr10</td>
					<td>8054744</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=GATA3">GATA3</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.-503_-502dup</td><td>het</td><td>0.32433898974</td>
					<td>Hypoparathyroidism, deafness, renal disease syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a protein which belongs to the GATA family of transcription factors. The protein contains two GATA-type zinc fingers and is an important regulator of T-cell development and plays an important role in endothelial cell biology. Defects in this gene are the cause of hypoparathyroidism with sensorineural deafness and renal dysplasia. [provided by RefSeq, Nov 2009]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>32</td><td>chr9</td>
					<td>95174077</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FANCC">FANCC</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7850958">rs7850958</a></td>
					<td>c.346-1930T>C</td><td>het</td><td>0.461417977241</td>
					<td>not provided</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group C. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>33</td><td>chr9</td>
					<td>5128081</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=JAK2">JAK2</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*1329_*1330del</td><td>het</td><td>0.319065593619</td>
					<td>Budd-Chiari syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a non-receptor tyrosine kinase that plays a central role in cytokine and growth factor signalling. The primary isoform of this protein has an N-terminal FERM domain that is required for erythropoietin receptor association, an SH2 domain that binds STAT transcription factors, a pseudokinase domain and a C-terminal tyrosine kinase domain. Cytokine binding induces autophosphorylation and activation of this kinase. This kinase then recruits and phosphorylates signal transducer and activator of transcription (STAT) proteins. Growth factors like TGF-beta 1 also induce phosphorylation and activation of this kinase and translocation of downstream STAT proteins to the nucleus where they influence gene transcription. Mutations in this gene are associated with numerous inflammatory diseases and malignancies. This gene is a downstream target of the pleiotropic cytokine IL6 that is produced by B cells, T cells, dendritic cells and macrophages to produce an immune response or inflammation. Disregulation of the IL6/JAK2/STAT3 signalling pathways produces increased cellular proliferation and myeloproliferative neoplasms of hematopoietic stem cells. A nonsynonymous mutation in the pseudokinase domain of this gene disrupts the domains inhibitory effect and results in constitutive tyrosine phosphorylation activity and hypersensitivity to cytokine signalling. This gene and the IL6/JAK2/STAT3 signalling pathway is a therapeutic target for the treatment of excessive inflammatory responses to viral infections. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2020]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>34</td><td>chr7</td>
					<td>140734798</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=BRAF">BRAF</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.2128-28dup</td><td>hom</td><td>None</td>
					<td>Noonan syndrome with multiple lentigines|Cardio-facio-cutaneous syndrome|Noonan syndrome|not specified</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a protein belonging to the RAF family of serine/threonine protein kinases. This protein plays a role in regulating the MAP kinase/ERK signaling pathway, which affects cell division, differentiation, and secretion. Mutations in this gene, most commonly the V600E mutation, are the most frequently identified cancer-causing mutations in melanoma, and have been identified in various other cancers as well, including non-Hodgkin lymphoma, colorectal cancer, thyroid carcinoma, non-small cell lung carcinoma, hairy cell leukemia and adenocarcinoma of lung. Mutations in this gene are also associated with cardiofaciocutaneous, Noonan, and Costello syndromes, which exhibit overlapping phenotypes. A pseudogene of this gene has been identified on the X chromosome. [provided by RefSeq, Aug 2017]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>35</td><td>chr6</td>
					<td>152121769</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=ESR1">ESR1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.851-3487del</td><td>hom</td><td>0.453276658411</td>
					<td>Cerebellar ataxia|Emery-Dreifuss muscular dystrophy</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes an estrogen receptor and ligand-activated transcription factor. The canonical protein contains an N-terminal ligand-independent transactivation domain, a central DNA binding domain, a hinge domain, and a C-terminal ligand-dependent transactivation domain. The protein localizes to the nucleus where it may form either a homodimer or a heterodimer with estrogen receptor 2. The protein encoded by this gene regulates the transcription of many estrogen-inducible genes that play a role in growth, metabolism, sexual development, gestation, and other reproductive functions and is expressed in many non-reproductive tissues. The receptor encoded by this gene plays a key role in breast cancer, endometrial cancer, and osteoporosis. This gene is reported to have dozens of transcript variants due to the use of alternate promoters and alternative splicing, however, the full-length nature of many of these variants remain uncertain. [provided by RefSeq, Jul 2020]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>36</td><td>chr6</td>
					<td>151842200</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=ESR1">ESR1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2234693">rs2234693</a></td>
					<td>c.453-397T>C</td><td>hom</td><td>0.472820185244</td>
					<td>Myocardial infarction, susceptibility to</td><td>risk factor</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes an estrogen receptor and ligand-activated transcription factor. The canonical protein contains an N-terminal ligand-independent transactivation domain, a central DNA binding domain, a hinge domain, and a C-terminal ligand-dependent transactivation domain. The protein localizes to the nucleus where it may form either a homodimer or a heterodimer with estrogen receptor 2. The protein encoded by this gene regulates the transcription of many estrogen-inducible genes that play a role in growth, metabolism, sexual development, gestation, and other reproductive functions and is expressed in many non-reproductive tissues. The receptor encoded by this gene plays a key role in breast cancer, endometrial cancer, and osteoporosis. This gene is reported to have dozens of transcript variants due to the use of alternate promoters and alternative splicing, however, the full-length nature of many of these variants remain uncertain. [provided by RefSeq, Jul 2020]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>37</td><td>chr5</td>
					<td>177093242</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FGFR4">FGFR4</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs351855">rs351855</a></td>
					<td>c.1162G>A</td><td>hom</td><td>0.321121756917</td>
					<td>See cases|Cancer progression and tumor cell motility|not specified</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The protein encoded by this gene is a tyrosine kinase and cell surface receptor for fibroblast growth factors. The encoded protein is involved in the regulation of several pathways, including cell proliferation, cell differentiation, cell migration, lipid metabolism, bile acid biosynthesis, vitamin D metabolism, glucose uptake, and phosphate homeostasis. This protein consists of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment, and a cytoplasmic tyrosine kinase domain. The extracellular portion interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. [provided by RefSeq, Aug 2017]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>38</td><td>chr5</td>
					<td>112838413</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=APC">APC</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs544709767">rs544709767</a></td>
					<td>c.2819C>T</td><td>het</td><td>None</td>
					<td>Carcinoma of colon|Hereditary cancer-predisposing syndrome|Familial adenomatous polyposis 1|not specified|not provided</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a tumor suppressor protein that acts as an antagonist of the Wnt signaling pathway. It is also involved in other processes including cell migration and adhesion, transcriptional activation, and apoptosis. Defects in this gene cause familial adenomatous polyposis (FAP), an autosomal dominant pre-malignant disease that usually progresses to malignancy. Mutations in the APC gene have been found to occur in most colorectal cancers, where disease-associated mutations tend to be clustered in a small region designated the mutation cluster region (MCR) and result in a truncated protein product. [provided by RefSeq, Jun 2022]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>39</td><td>chr5</td>
					<td>6633319</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NSUN2">NSUN2</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs147269098">rs147269098</a></td>
					<td>c.-340C>T</td><td>het</td><td>0.0170237184392</td>
					<td>Intellectual Disability, Recessive|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a methyltransferase that catalyzes the methylation of cytosine to 5-methylcytosine (m5C) at position 34 of intron-containing tRNA(Leu)(CAA) precursors. This modification is necessary to stabilize the anticodon-codon pairing and correctly translate the mRNA. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Mar 2011]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>40</td><td>chr5</td>
					<td>6633172</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NSUN2">NSUN2</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs8192123">rs8192123</a></td>
					<td>c.-193G>C</td><td>het</td><td>0.0170643021179</td>
					<td>Intellectual Disability, Recessive|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a methyltransferase that catalyzes the methylation of cytosine to 5-methylcytosine (m5C) at position 34 of intron-containing tRNA(Leu)(CAA) precursors. This modification is necessary to stabilize the anticodon-codon pairing and correctly translate the mRNA. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Mar 2011]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>41</td><td>chr5</td>
					<td>6633052</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NSUN2">NSUN2</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs139145592">rs139145592</a></td>
					<td>c.-73C>T</td><td>het</td><td>0.0170204628036</td>
					<td>Intellectual Disability, Recessive|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a methyltransferase that catalyzes the methylation of cytosine to 5-methylcytosine (m5C) at position 34 of intron-containing tRNA(Leu)(CAA) precursors. This modification is necessary to stabilize the anticodon-codon pairing and correctly translate the mRNA. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Mar 2011]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>42</td><td>chr4</td>
					<td>56502497</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SRP72">SRP72</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*651_*652dup</td><td>het</td><td>0.186981285598</td>
					<td>Autosomal dominant aplasia and myelodysplasia</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes the 72 kDa subunit of the signal recognition particle (SRP), a ribonucleoprotein complex that mediates the targeting of secretory proteins to the endoplasmic reticulum (ER). The SRP complex consists of a 7S RNA and 6 protein subunits: SRP9, SRP14, SRP19, SRP54, SRP68, and SRP72, that are bound to the 7S RNA as monomers or heterodimers. SRP has at least 3 distinct functions that can be associated with the protein subunits: signal recognition, translational arrest, and ER membrane targeting by interaction with the docking protein. Mutations in this gene are associated with familial bone marrow failure. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jun 2012]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>43</td><td>chr3</td>
					<td>10152482</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=VHL">VHL</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*2540_*2545del</td><td>hom</td><td>0.283872598584</td>
					<td>Von Hippel-Lindau syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a component of a ubiquitination complex. The encoded protein is involved in the ubiquitination and degradation of hypoxia-inducible-factor (HIF), which is a transcription factor that plays a central role in the regulation of gene expression by oxygen. In addition to oxygen-related gene expression, this protein plays a role in many other cellular processes including cilia formation, cytokine signaling, regulation of senescence, and formation of the extracellular matrix. Variants of this gene are associated with von Hippel-Lindau syndrome, pheochromocytoma, erythrocytosis, renal cell carcinoma, and cerebellar hemangioblastoma. [provided by RefSeq, Jun 2022]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>44</td><td>chr3</td>
					<td>10152061</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=VHL">VHL</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.*2117del</td><td>hom</td><td>0.56986452426</td>
					<td>Von Hippel-Lindau syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a component of a ubiquitination complex. The encoded protein is involved in the ubiquitination and degradation of hypoxia-inducible-factor (HIF), which is a transcription factor that plays a central role in the regulation of gene expression by oxygen. In addition to oxygen-related gene expression, this protein plays a role in many other cellular processes including cilia formation, cytokine signaling, regulation of senescence, and formation of the extracellular matrix. Variants of this gene are associated with von Hippel-Lindau syndrome, pheochromocytoma, erythrocytosis, renal cell carcinoma, and cerebellar hemangioblastoma. [provided by RefSeq, Jun 2022]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>45</td><td>chr2</td>
					<td>47414421</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=MSH2">MSH2</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.942+29del</td><td>hom</td><td>0.146852814402</td>
					<td>Lynch syndrome 1|Turcot syndrome|Hereditary cancer-predisposing syndrome|Hereditary nonpolyposis colorectal neoplasms|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This locus is frequently mutated in hereditary nonpolyposis colon cancer (HNPCC). When cloned, it was discovered to be a human homolog of the E. coli mismatch repair gene mutS, consistent with the characteristic alterations in microsatellite sequences (RER+ phenotype) found in HNPCC. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Apr 2012]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>46</td><td>chr1</td>
					<td>241500603</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FH">FH</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.1237-16_1237-13dup</td><td>het</td><td>0.251112909753</td>
					<td>Hereditary leiomyomatosis and renal cell cancer|Fumarase deficiency|Hereditary cancer-predisposing syndrome|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">The protein encoded by this gene is an enzymatic component of the tricarboxylic acid (TCA) cycle, or Krebs cycle, and catalyzes the formation of L-malate from fumarate. It exists in both a cytosolic form and an N-terminal extended form, differing only in the translation start site used. The N-terminal extended form is targeted to the mitochondrion, where the removal of the extension generates the same form as in the cytoplasm. It is similar to some thermostable class II fumarases and functions as a homotetramer. Mutations in this gene can cause fumarase deficiency and lead to progressive encephalopathy. [provided by RefSeq, Jul 2008]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>47</td><td>chr1</td>
					<td>204549329</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=MDM4">MDM4</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs41299595">rs41299595</a></td>
					<td>c.1120A>C</td><td>het</td><td>0.00404914136985</td>
					<td>not provided</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a nuclear protein that contains a p53 binding domain at the N-terminus and a RING finger domain at the C-terminus, and shows structural similarity to p53-binding protein MDM2. Both proteins bind the p53 tumor suppressor protein and inhibit its activity, and have been shown to be overexpressed in a variety of human cancers. However, unlike MDM2 which degrades p53, this protein inhibits p53 by binding its transcriptional activation domain. This protein also interacts with MDM2 protein via the RING finger domain, and inhibits the latter&apos;s degradation. So this protein can reverse MDM2-targeted degradation of p53, while maintaining suppression of p53 transactivation and apoptotic functions. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene. [provided by RefSeq, Feb 2011]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>48</td><td>chr1</td>
					<td>162780323</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DDR2">DDR2</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs886045499">rs886045499</a></td>
					<td>c.*77C>G</td><td>het</td><td>None</td>
					<td>Spondyloepimetaphyseal dysplasia-short limb-abnormal calcification syndrome</td><td>Uncertain significance</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of the discoidin domain receptor subclass of the receptor tyrosine kinase (RTKs) protein family. RTKs play a key role in the communication of cells with their microenvironment. The encoded protein is a collagen-induced receptor that activates signal transduction pathways involved in cell adhesion, proliferation, and extracellular matrix remodeling. This protein is expressed in numerous cell types and may alos be involved in wound repair and regulate tumor growth and invasiveness. Mutations in this gene are the cause of short limb-hand type spondylometaepiphyseal dysplasia. [provided by RefSeq, Aug 2017]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>49</td><td>chr1</td>
					<td>158678534</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SPTA1">SPTA1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs199598260">rs199598260</a></td>
					<td>c.679G>A</td><td>het</td><td>0.00312464289795</td>
					<td>Pyropoikilocytosis, hereditary|Elliptocytosis 2|Hereditary spherocytosis type 3|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of a family of molecular scaffold proteins that link the plasma membrane to the actin cytoskeleton and functions in the determination of cell shape, arrangement of transmembrane proteins, and organization of organelles. The encoded protein is primarily composed of 22 spectrin repeats which are involved in dimer formation. It forms a component of the erythrocyte plasma membrane. Mutations in this gene result in a variety of hereditary red blood cell disorders, including elliptocytosis-2, pyropoikilocytosis, and spherocytosis, type 3. [provided by RefSeq, Aug 2017]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>50</td><td>chr1</td>
					<td>158678420</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SPTA1">SPTA1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs183647059">rs183647059</a></td>
					<td>c.793A>G</td><td>het</td><td>0.00309423146848</td>
					<td>Pyropoikilocytosis, hereditary|Elliptocytosis 2|Hereditary spherocytosis type 3|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of a family of molecular scaffold proteins that link the plasma membrane to the actin cytoskeleton and functions in the determination of cell shape, arrangement of transmembrane proteins, and organization of organelles. The encoded protein is primarily composed of 22 spectrin repeats which are involved in dimer formation. It forms a component of the erythrocyte plasma membrane. Mutations in this gene result in a variety of hereditary red blood cell disorders, including elliptocytosis-2, pyropoikilocytosis, and spherocytosis, type 3. [provided by RefSeq, Aug 2017]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>51</td><td>chr1</td>
					<td>158668076</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SPTA1">SPTA1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/None">None</a></td>
					<td>c.1834-16_1834-14del</td><td>het</td><td>0.285708830184</td>
					<td>Elliptocytosis|Pyropoikilocytosis, hereditary|Spherocytosis, Recessive|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of a family of molecular scaffold proteins that link the plasma membrane to the actin cytoskeleton and functions in the determination of cell shape, arrangement of transmembrane proteins, and organization of organelles. The encoded protein is primarily composed of 22 spectrin repeats which are involved in dimer formation. It forms a component of the erythrocyte plasma membrane. Mutations in this gene result in a variety of hereditary red blood cell disorders, including elliptocytosis-2, pyropoikilocytosis, and spherocytosis, type 3. [provided by RefSeq, Aug 2017]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>52</td><td>chr1</td>
					<td>158627717</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SPTA1">SPTA1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3737515">rs3737515</a></td>
					<td>c.5572C>G</td><td>het</td><td>0.255095324963</td>
					<td>Hemolytic anemia|Pyropoikilocytosis, hereditary|Elliptocytosis 2|Hereditary spherocytosis type 3|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of a family of molecular scaffold proteins that link the plasma membrane to the actin cytoskeleton and functions in the determination of cell shape, arrangement of transmembrane proteins, and organization of organelles. The encoded protein is primarily composed of 22 spectrin repeats which are involved in dimer formation. It forms a component of the erythrocyte plasma membrane. Mutations in this gene result in a variety of hereditary red blood cell disorders, including elliptocytosis-2, pyropoikilocytosis, and spherocytosis, type 3. [provided by RefSeq, Aug 2017]</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>53</td><td>chr1</td>
					<td>158618068</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SPTA1">SPTA1</a></td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs28525570">rs28525570</a></td>
					<td>c.6531-12C>T</td><td>het</td><td>0.255028054441</td>
					<td>Hemolytic anemia|Pyropoikilocytosis, hereditary|Elliptocytosis 2|Hereditary spherocytosis type 3|not specified|not provided</td><td>Conflicting interpretations of pathogenicity</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">This gene encodes a member of a family of molecular scaffold proteins that link the plasma membrane to the actin cytoskeleton and functions in the determination of cell shape, arrangement of transmembrane proteins, and organization of organelles. The encoded protein is primarily composed of 22 spectrin repeats which are involved in dimer formation. It forms a component of the erythrocyte plasma membrane. Mutations in this gene result in a variety of hereditary red blood cell disorders, including elliptocytosis-2, pyropoikilocytosis, and spherocytosis, type 3. [provided by RefSeq, Aug 2017]</td></tr>
				
		</tbody>
	</table>
	</div>
	<div class="last"></div>
</div>
	<button class="accordion accordion-heading major-risk"  >Coronary artery Disease</button>
	<div class="basic-panel major-risk">
		<button class="accordion accordion0 accordion1 major-risk"><div></div>Description</button>
	<div class="board variants panel prs-panel major-risk">
		<p>Coronary artery disease (CAD) is a condition in which the blood vessels that supply the heart muscle with oxygen and nutrients become narrowed or blocked, leading to reduced blood flow to the heart. Age is a major risk factor for CAD, as the risk of developing CAD increases with age. Genetic factors plays an important role in the development of coronary artery disease (CAD).</p>
		<p>This report contains data of PRS for coronary disease development (<a href="https://www.pgscatalog.org/score/PGS000818/">PGS000818</a>) and the list with genes, associated with an increased risk of CAD.</p>
		<p>However, it's important to note that genetics is just one of many factors that contribute to the development of CAD, and having a genetic predisposition to the disease does not necessarily mean that an individual will develop it.</p>
		</div>
		<button class="accordion accordion0 accordion1 prs-panel major-risk"><div></div>PRS</button>
	<div class="board variants panel prs-panel major-risk">
		<p class="how_to">How to read the result?</p>
<p></p>
<p>PRS is represented as a&nbsp;<u>percentile</u> within a given population. For example, you have the 70th percentile, which means your personal genetic risk of the disease development is higher than 70 out of every 100 people in a chosen population.
</p>
<p></p>
<p>So percentile &#65310;50 shows an increased risk, &#65308;50 - decreased one.</p>
<p></p>

<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Coronary heart disease (<a href="https://www.pgscatalog.org/score/PGS000818/">PGS000818</a>)</td><td rowspan="6"><canvas id="PGS000818_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>1.1501</td></tr>
    <tr><td>Count/total</td><td>73/138</td></tr>
    <tr><td>Average</td><td>0.007877397260273972</td></tr>
    <tr><td>Percentile</td><td>97%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS000818_canvas", 0.97, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>
		</div>

		<button class="accordion0 accordion accordion-result major-risk"><div></div>Results</button>
<div class="content content-variants major-risk">
	<table class="ui selectable celled teal table table cancer cancer1">
		<thead>
		<tr><th style="width:1%" class="clickable title" onclick="collapseExpandAll(this, 'coronaryTBody')"><div class="ui tiny circular compact positive button">+</div></th><th>#</th>
				<th>RSID</th><th>Gene</th><th>Risk Allele</th><th>Genotype</th><th>Pubmed ID</th>
				<th>Population</th><th>P-Value</th><th>Weight</th></tr>
		</thead>
		<tbody id="coronaryTBody">
			
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>1</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs8055236">rs8055236</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDH13">CDH13</a></td>
					<td>G</td>
					<td>G/G</td><td>PMID 17554300; PMID 19956433; PMID 20017983;</td><td>European</td>
					<td>[PMID 17554300]: 6 x 10-6</td><td style="background-color:#ff0000">-1.7</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>CDH13  encodes T-cadherin, is a regulator of vascular wall remodeling, angiogenesis and is essential for adiponectinвЂ™s vascular actions. GG genotype was shown to have increased risk for heart disease; the odds ratio is 2.23 (CI 1.56-3.17). [PMID 17554300]. There was no statistically significant association with CHD in the Chinese population [PMC4509238].<br/><b>Study design: </b>[PMID 17554300]: Discovery ancestry (country of recruitment): 4864 European (U.K.); Discovery sample description: 1,926 European ancestry cases, 2,938 European ancestry controls;</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>2</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs4977574">rs4977574</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDKN2B-AS1">CDKN2B-AS1</a></td>
					<td>G</td>
					<td>G/A</td><td>PMID 17478681; PMID 21378990; PMID 21378988; PMID 24916648; PMID: 30278588</td><td>European; Asian</td>
					<td>[PMID 29263402]: 1 x 10-7; [PMID 21239051]: 1 x 10-14;</td><td style="background-color:#ff3838">-1.3</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>Several GWASs have revealed that the single nucleotide polymorphism (SNP) rs4977574 of ANRIL (CDKN2B-AS1) was associated with increased CAD risk in different ethnicities. Gene product may affect development of CAD via regulating vascular cell proliferation with reduced expression of CDKN2A and CDKN2B. GA-carriers has 1.3 times increased susceptibility to Coronary artery disease (for Europeans) [PMID 17478681]. In meta-analysis it was shown, that rs4977574 in ANRIL increased CAD risk in Asian population [PMID: 30278588].<br/><b>Study design: </b>[PMID 29263402]: Discovery sample description: 1,778 Chinese ancestry cases, 391 Malay ancestry cases, 5,164 Chinese ancestry controls, 2,212 Malay ancestry controls Discovery ancestry (country of recruitment): 2603 South East Asian (Singapore), 6942 East Asian (Singapore); [PMID 29263402]: Discovery sample description: 1,808 European ancestry cases, 915 European ancestry controls Discovery ancestry (country of recruitment): 2723 European (U.S.); Replication sample description: 10,585 European ancestry cases, 6,468 European ancestry controls;  Replication ancestry (country of recruitment): 17053 European (U.S.)</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>3</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs599839">rs599839</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SORT1">SORT1</a></td>
					<td>A</td>
					<td>A/A</td><td>PMID: 32858814; PMID 33321069; PMID: 17634449; PMID 18262040; PMID 19380133;  PMID 19660754;  PMID 19750184;</td><td>European</td>
					<td>[PMID 33321069]: 2 x 10-7</td><td style="background-color:#ff4747">-1.2</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>NPs in PSRC1 that have regulatory potential may affect CAD risk by altering CELSR2, PSRC1, and SORT1 gene expression and circulating granulins and apolipoprotein B proteins [PMID: 35252377]. Gene product regulate apoB secretion and LDL catabolism. rs599839 is a SNP found to be associated with heart disease by the German MI (Myocardial infarction) Family Study group, over several populations. The odds ratio for the (common) risk allele, rs599839(A), is 1.29 (CI: 1.18-1.40, adjusted p=0.0006). Your genotype (AA) is associated with  1.4x increased risk for heart disease [PMID 17634449]<br/><b>Study design: </b>[PMID 33321069]: Discovery ancestry (country of recruitment): 16817 South Asian (Pakistan), 49826 European (Estonia, Finland, France, Germany, Greece, Italy, Netherlands, Sweden, U.K., NR, U.S.); Discovery sample description: 19,245 European ancestry cases, 8,463 South Asian ancestry cases, 30,581 European ancestry controls, 8,354 South Asian ancestry controls; Replication sample description16,704 European ancestry cases, 101,083 European ancestry controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>4</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs17465637">rs17465637</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=MIA3">MIA3</a></td>
					<td>C</td>
					<td>C/C</td><td>PMID: 35768776; PMID 19956433;  PMID 21804106;  PMID 21984477; PMID 21264445;</td><td>European; Iranian;</td>
					<td>[PMID 17634449]: 1 x 10-6; [PMID 21378990]: 1 x 10-8;</td><td style="background-color:#ff4747">-1.2</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>CC genotype increases increases susceptibility to Coronary artery disease and myocardial infarction (x1,3 elevated risk) [PMID 21264445].<br/><b>Study design: </b>[PMID 17634449]: Discovery ancestry (country of recruitment): 4864 European (U.K.);Discovery sample description: 1,926 European ancestry cases, 2,938 European ancestry controls ;  [PMID 21378990]: Discovery ancestry (country of recruitment): 86995 European (NR); Discovery sample description: 22,233 European ancestry cases, 64,762 European ancestry controls;</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>5</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1122608">rs1122608</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LDLR">LDLR</a></td>
					<td>G</td>
					<td>G/G</td><td>PMID: 27664493; PMID 23380588; PMID 23202125; PMID 20810930; PMID 19956433; PMID: 33321069; PMID 21378990;</td><td>European; Asian</td>
					<td>[PMID: 33321069]: 5 x 10-7; [PMID 21378990]:         1 x 10-9</td><td style="background-color:#ff6666">-1.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>It has been reported that rs1122608 adjacent to low-density lipoprotein cholesterol receptor (LDLR) locus is associated with the risk of coronary artery disease (CAD) and blood lipid profile. A statistically significant association was identified between rs1122608-G allele and CAD risk in European and Asian subgroups [PMID: 27664493]. Your genotype (GG) is associated with increased risk for coronary arthery disease.<br/><b>Study design: </b>[PMID: 33321069]: Discovery ancestry (country of recruitment)16817 South Asian (Pakistan), 49826 European (Estonia, Finland, France, Germany, Greece, Italy, Netherlands, Sweden, U.K., NR, U.S.); Discovery sample description19,245 European ancestry cases, 8,463 South Asian ancestry cases, 30,581 European ancestry controls, 8,354 South Asian ancestry controls; [PMID 21378990]: Discovery ancestry (country of recruitment)86995 European (NR); Discovery sample description22,233 European ancestry cases, 64,762 European ancestry controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>6</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7250581">rs7250581</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
					<td>G</td>
					<td>G/G</td><td>PMID 19955471;  PMID 19956433; PMID 17554300;</td><td>European</td>
					<td></td><td style="background-color:#ff8484">-0.8</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>rs7250581 has been reported in a large study to be associated with heart disease, in particular, coronary artery disease. The risk allele is (G); the odds ratio associated with GG genotype is 1.40 (CI 1.05-1.86). [PMID 17554300]<br/><b>Study design: </b>[PMID 17554300]: Discovery ancestry (country of recruitment): 4864 European (U.K.); Discovery sample description: 1,926 European ancestry cases, 2,938 European ancestry controls;</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>7</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12413409">rs12413409</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CNNM2">CNNM2</a></td>
					<td>G</td>
					<td>G/G</td><td>PMID 21378988; PMID 24262325; PMID: 21378990; PMID: 31228190; PMID 21378990;</td><td>European</td>
					<td>[PMID 21378990]: 1 x 10-9</td><td style="background-color:#ff8484">-0.8</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>CNNM2 gene affects the risk of coronary heart disease. G-allele is associated with increased risk of coronary disease. The rs7692387 risk (G) allele was associated with higher incidence of major CVD events. In two major primary CVD prevention trials, aspirin was found to confer a significant (21%) reduction in risk of major CVD amongst the 66% of participants who were homozygous for the rs7692387 risk (G) allele [PMID: 31228190].<br/><b>Study design: </b>[PMID 21378990]: Discovery ancestry (country of recruitment): 86995 European (NR); Discovery sample description: 22,233 European ancestry cases, 64,762 European ancestry controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>8</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7692387">rs7692387</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=GUCYA3">GUCYA3</a></td>
					<td>G</td>
					<td>G/G</td><td>PMID: 31228190; PMID 23202125; PMID 31883534;</td><td>European; Chinese</td>
					<td>[PMID: 29212778]: Discovery sample description: up to 122,733 cases, up to 424,528 controls</td><td style="background-color:#ffb2b2">-0.5</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>G-allele is associated with increased risk of heart disease for caucasian and Han-chinese populations GG-variant carriers may benefit from taking aspirin: In two randomized placebo-controlled trials in the setting of primary prevention, aspirin reduced the incidence of CVD events in individuals homozygous for the GUCY1A3 risk (G) allele [PMID: 31228190].<br/><b>Study design: </b>[PMID: 29212778]: Discovery sample description: up to 122,733 cases, up to 424,528 controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>9</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs55730499">rs55730499</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LPA">LPA</a></td>
					<td>T</td>
					<td>T/T</td><td>PMID: 26343387</td><td>Multiethnic;</td>
					<td>[PMID 26343387]: 5 x 10-39</td><td style="background-color:#ffc1c1">-0.4</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>The T allele of rs55730499 is reported to be associated with Coronary Artery Disease.<br/><b>Study design: </b>[PMID 26343387]: Discovery ancestry (country of recruitment)141217 European (Finland, Sweden, U.S., Iceland, Italy, Netherlands, Greece, Germany, U.K., Spain, Estonia, NR, Republic of Ireland), 25557 South Asian (India, U.K., Pakistan), 2268 Greater Middle Eastern (Middle Eastern, North African or Persian) (NR), 4095 Hispanic or Latin American (U.S.), 3139 African American or Afro-Caribbean (U.S.), 11323 East Asian (Republic of Korea, China); Discovery sample description: 42,096 European ancestry cases, 361 African American cases, 758 Hispanic American cases, 12,658 South Asian ancestry cases, 1,802 Lebanese ancestry cases, 3,614 East Asian ancestry cases, 99,121 European ancestry controls, 2,778 African American controls, 3,337 Hispanic American controls, 12,899 South Asian ancestry controls, 466 Lebanese ancestry controls, 7,709 East Asian ancestry controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>10</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs383830">rs383830</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APC">APC</a></td>
					<td>A</td>
					<td>T/T</td><td>PMID 26436499; PMID 17554300; PMID 21804106;  PMID 19956433;  PMID 21804106;</td><td>European; Chinese</td>
					<td></td><td style="background-color:#ffe0e0">-0.2</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>APC is a tumor suppressor gene that is involved in the processes of cell migration and adhesion, transcriptional activation, and apoptosis. T-allele was associated with male-specific increased risk of coronary heart disease (CHD) in Han Chinese. There was no association found for female [PMID 26436499]. For European population T-allele is not associated with increased risk of CHD.<br/><b>Study design: </b>[PMID 17554300]: Discovery sample description: 1,926 European ancestry cases, 2,938 European ancestry controls:  Discovery ancestry (country of recruitment): 4864 European (U.K.)</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>11</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs515135">rs515135</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APOB">APOB</a></td>
					<td>T</td>
					<td>C/C</td><td>PMID 23202125; PMID: 30507093; PMID 29212778;</td><td>European</td>
					<td>[PMID 29212778]: 6 x 10-17</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>APOB is a large glycoprotein that plays an important role in the metabolism of lipoproteins in the human body. Small changes in the structure and function of APOB can cause major problems in lipid metabolism. Large-scale association analysis identifies rs515135 as a risk loci for coronary artery disease. Your genotype (CC) does not contain the risk indicating typical risk for Coronary heart disease.<br/><b>Study design: </b>[PMID 29212778]: Discovery ancestry (country of recruitment): 547261 NR (U.K., NR); Discovery sample description: up to 122,733 cases, up to 424,528 controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>12</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2943634">rs2943634</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id="></a></td>
					<td>C</td>
					<td>C/A</td><td>PMID 17634449; PMID 23659870; PMID 22042884; PMID 19956433; PMID 19750184; PMID 22207032;</td><td>European</td>
					<td>[PMID 17634449]: 2 x 10-7</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>rs2943634 C/A single nucleotide polymorphism (SNP), located in a non coding region of chromosome 2q36.3, has been associated with coronary artery disease in two genome wide association studies.<br/><b>Study design: </b>[PMID 17634449]: Discovery ancestry (country of recruitment): 4864 European (U.K.); Discovery sample description: 1,926 European ancestry cases, 2,938 European ancestry controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>13</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs964184">rs964184</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ZPR1">ZPR1</a></td>
					<td>G</td>
					<td>C/C</td><td>PMID 19060906; PMID 20864672; PMID 21378990; PMID 22003152; PMID 21378990; PMID 28714975;</td><td>European</td>
					<td>[PMID 28714975]: 5 x 10-6; [PMID 21378990]: 1 x 10-17;</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>rs964184 variant in the ZPR1 gene has been associated with blood lipids levels both in fasting and postprandial state and with the risk of myocardial infarction and CAD. CC genotype is NOT associated with  increased risk of hypertriglyceridemia and coronary arthery disease.<br/><b>Study design: </b>[PMID 21378990]: Discovery ancestry (country of recruitment): 86995 European (NR);  Discovery sample description: 22,233 European ancestry cases, 64,762 European ancestry controls; [PMID 28714975]: Discovery ancestry (country of recruitment): 63731 European, NR (U.K.); Discovery sample description: 18,467 European and unknown ancestry cases, 45,264 European and unknown ancestry controls;</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>14</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs3184504">rs3184504</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SH2B3">SH2B3</a></td>
					<td>T</td>
					<td>C/C</td><td>PMID 24262325; PMID 21378990; PMID 26343387;</td><td>Multiethnic</td>
					<td></td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>SH2B adaptor protein 3 (SH2B3)  is a negative regulator of multiple cytokine signaling pathways and is associated with increased risk of myocardial infarction (MI) and fibrosis [PMID: 25628389]. CC variant according to GWAS studies is NOT associated with increased risk of CAD.<br/><b>Study design: </b>[PMID 26343387]: Discovery ancestry (country of recruitment): 141217 European (Finland, Sweden, U.S., Iceland, Italy, Netherlands, Greece, Germany, U.K., Spain, Estonia, NR, Republic of Ireland), 25557 South Asian (India, U.K., Pakistan), 2268 Greater Middle Eastern (Middle Eastern, North African or Persian) (NR), 4095 Hispanic or Latin American (U.S.), 3139 African American or Afro-Caribbean (U.S.), 11323 East Asian (Republic of Korea, China); Discovery sample description: 42,096 European ancestry cases, 361 African American cases, 758 Hispanic American cases, 12,658 South Asian ancestry cases, 1,802 Lebanese ancestry cases, 3,614 East Asian ancestry cases, 99,121 European ancestry controls, 2,778 African American controls, 3,337 Hispanic American controls, 12,899 South Asian ancestry controls, 466 Lebanese ancestry controls, 7,709 East Asian ancestry controls; [PMID 32469254]: Discovery ancestry (country of recruitment): 340799 East Asian, European (NR), 51442 East Asian (Japan); Discovery sample description: 15,302 Japanese ancestry cases, 76,014 European and East Asian ancestry cases, 36,140 Japanese ancestry controls, 264,785 European and East Asian ancestry controls.</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>15</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs17514846">rs17514846</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=FURIN">FURIN</a></td>
					<td>A</td>
					<td>A/A</td><td>PMID: 26343387; PMID 23202125; PMID: 32067586;</td><td>Multiethnic</td>
					<td>[PMID: 26343387]: 3 x 10-7;</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>The coronary artery disease risk variant rs17514846 at the 15q26.1 locus modulates FURIN expression in vascular ECs. FURIN levels in ECs affect monocyteвЂђendothelial adhesion and migration [PMID: 32067586]. GenomeвЂђwide association studies have shown an association between the singleвЂђnucleotide polymorphism (A is a risk allele) rs17514846 and coronary artery disease susceptibility. CC genotype is NOT associated with an increased risk of CAD.<br/><b>Study design: </b>[PMID: 26343387]: Discovery ancestry (country of recruitment): 141217 European (Finland, Sweden, U.S., Iceland, Italy, Netherlands, Greece, Germany, U.K., Spain, Estonia, NR, Republic of Ireland), 25557 South Asian (India, U.K., Pakistan), 2268 Greater Middle Eastern (Middle Eastern, North African or Persian) (NR), 4095 Hispanic or Latin American (U.S.), 3139 African American or Afro-Caribbean (U.S.), 11323 East Asian (Republic of Korea, China); Discovery sample description: 42,096 European ancestry cases, 361 African American cases, 758 Hispanic American cases, 12,658 South Asian ancestry cases, 1,802 Lebanese ancestry cases, 3,614 East Asian ancestry cases, 99,121 European ancestry controls, 2,778 African American controls, 3,337 Hispanic American controls, 12,899 South Asian ancestry controls, 466 Lebanese ancestry controls, 7,709 East Asian ancestry controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>16</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs11206510">rs11206510</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PCSK9">PCSK9</a></td>
					<td>T</td>
					<td>T/T</td><td>PMID 20864672; PMID 21378990; PMID 26343387;</td><td>Multiethnic; Asian</td>
					<td>[PMID 26343387]: 2 x 10-8; [PMID 32469254]: 1 x 10-8;</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>PCSK9 overexpression increases plasma LDL-C levels by downregulating LDL receptor (LDLR) expression after transcription. TT variant carriers has elevated risk of coronary arthery disease.<br/><b>Study design: </b>[PMID 26343387]: Discovery ancestry (country of recruitment): 141217 European (Finland, Sweden, U.S., Iceland, Italy, Netherlands, Greece, Germany, U.K., Spain, Estonia, NR, Republic of Ireland), 25557 South Asian (India, U.K., Pakistan), 2268 Greater Middle Eastern (Middle Eastern, North African or Persian) (NR), 4095 Hispanic or Latin American (U.S.), 3139 African American or Afro-Caribbean (U.S.), 11323 East Asian (Republic of Korea, China); Discovery sample description: 42,096 European ancestry cases, 361 African American cases, 758 Hispanic American cases, 12,658 South Asian ancestry cases, 1,802 Lebanese ancestry cases, 3,614 East Asian ancestry cases, 99,121 European ancestry controls, 2,778 African American controls, 3,337 Hispanic American controls, 12,899 South Asian ancestry controls, 466 Lebanese ancestry controls, 7,709 East Asian ancestry controls; [PMID 32469254]: Discovery ancestry (country of recruitment): 340799 East Asian, European (NR), 51442 East Asian (Japan); Discovery sample description: 15,302 Japanese ancestry cases, 76,014 European and East Asian ancestry cases, 36,140 Japanese ancestry controls, 264,785 European and East Asian ancestry controls.</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>17</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1746048">rs1746048</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CXCL12 (LINC02881)">CXCL12 (LINC02881)</a></td>
					<td>C</td>
					<td>T/C</td><td>PMID: 28614256; PMID 19956433; PMID 23531450; PMID 21378990; PMID 33632238; PMID 21378990;</td><td>European; Asian</td>
					<td>[PMID 33632238]: 1 x 10-6; [PMID 21378990]:3 x 10-10;</td><td style="background-color:#e0ffe0">0.2</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>The meta-analysis of 48,852 patients and 64,386 controls from 12 studies showed that patients with rs1746048 had 1.11 times of high risk in developing CHD (ORвЂЉ=вЂЉ1.11; 95% CIвЂЉ=вЂЉ1.09вЂ“1.14; PвЂЉ<вЂЉ.005; I2вЂЉ=вЂЉ35.8%). The increased risk of CHD was also found in both Asian (ORвЂЉ=вЂЉ1.07; 95%CIвЂЉ=вЂЉ1.02вЂ“1.12; PвЂЉ<вЂЉ.005; I2вЂЉ=вЂЉ40.6%) and Caucasian populations (ORвЂЉ=вЂЉ1.14; 95% CIвЂЉ=вЂЉ1.10вЂ“1.18; PвЂЉ<вЂЉ.005; I2вЂЉ=вЂЉ22.2%) [PMID: 28614256]. CT carriers has 0.94 decreased risk for coronary heart disease.<br/><b>Study design: </b>[PMID 21378990]: Discovery ancestry (country of recruitment): 86995 European (NR); Discovery sample description: 22,233 European ancestry cases, 64,762 European ancestry controls</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>18</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1333049">rs1333049</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CDKN2B-AS1">CDKN2B-AS1</a></td>
					<td>C</td>
					<td>C/G</td><td>PMID 17634449, PMID 18362232;  PMID 20031606;  PMID 19171343; PMID 18979498; PMID 24573017; PMID 23202125; PMID 22623978; PMID 17554300; PMID 33321069;</td><td>European; Multiethnic;</td>
					<td>[PMID 17554300]: 1 x 10-13; [PMID 33321069]: 5 x 10-18;</td><td style="background-color:#75ff75">0.9</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11"><b>Conclusion: </b>1.5x increased risk for CAD is reported for GC carriers<br/><b>Study design: </b>[PMID 17554300]: Discovery ancestry (country of recruitment): 4864 European (U.K.); Discovery sample description: 1,926 European ancestry cases, 2,938 European ancestry controls; [PMID 33321069]: Discovery ancestry (country of recruitment): 32307 European (Estonia, Finland, France, Germany, Greece, Italy, Netherlands, Sweden, U.K., NR, U.S.), 10077 South Asian (Pakistan); Discovery sample description: 13,381 European ancestry cases, 4,313 South Asian ancestry cases, 18,926 European ancestry controls, 5,764 South Asian ancestry controls;</td></tr>
				
		</tbody>
	</table>
</div>
<div class="last"></div>
</div>
<button class="accordion accordion-heading major-risk"  >Cardio Hereditary</button>
<div class="basic-panel major-risk">
<button class="accordion accordion0 accordion1 major-risk"><div></div>Description</button>
	<div class="board variants panel panel-description major-risk">
	<p>Timely detection and diagnosis of heart disorders can lead to enhanced treatment options, help to prevent sudden cardiac death, and improve prognosis. This report analyzes the most relevant genes for arrhythmias, congenital heart disease, and cardiomyopathies. 
Analyzed gene abnormalities may cause the following syndromes: Long and short QT, Brugada syndrome, catecholaminergic polymorphic ventricular tachycardia, cardiomyopathies dilated and hypertrophic, and congenital heart defects. In addition, this panel includes vascular abnormalities, such as dolichoectasia and hereditary hemorrhagic telangiectasia.
</p>
<p></p></div>
<button class="accordion0 accordion accordion-result major-risk"><div></div>Results</button>
<div class="content content-variants major-risk">
	<table class="ui selectable celled teal table table cancer cardio">
		<thead>
		<tr><th style="width:1%" class="clickable title" onclick="collapseExpandAll(this, 'cardioTBody')"><div class="ui tiny circular compact positive button">+</div></th><th>#</th>
				<th>Gene</th><th>rsID</th><th>cDNA change</th><th>Your genotype</th><th>Sequence ontology</th><th>SIFT prediction</th>
				<th>Allele Frequency</th><th>Significance</th><th>ClinVar ID</th><th>OMIM</th></tr>
		</thead>
		<tbody id="cardioTBody">
			
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>1</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DMD">DMD</a></td>
					<td>rs1801187</td></a></td>
					<td>c.5234G>A</td>
					<td>T/C</td><td>MIS</td><td>Damaging</td>
					<td>0.513357050928</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/94657/">94657</a></td>
					<td><a href="https://www.omim.org/entry/302045; 310200">302045; 310200</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene spans a genomic range of greater than 2 Mb and encodes a large protein containing an N-terminal actin-binding domain and multiple spectrin repeats. The encoded protein forms a component of the dystrophin-glycoprotein complex (DGC), which bridges the inner cytoskeleton and the extracellular matrix. Deletions, duplications, and point mutations at this gene locus may cause Duchenne muscular dystrophy (DMD), Becker muscular dystrophy (BMD), or cardiomyopathy. Alternative promoter usage and alternative splicing result in numerous distinct transcript variants and protein isoforms for this gene. [provided by RefSeq, Dec 2016]
						<p>Phenotype Name: Dystrophin deficiency|Cardiomyopathy|Becker muscular dystrophy|Dilated cardiomyopathy 3B|Duchenne muscular dystrophy|not specified|Cardiovascular phenotype|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>2</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DMD">DMD</a></td>
					<td>rs368974772</td></a></td>
					<td>c.8332C>T</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>None</td>
					<td>Uncertain significance</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1464030/">1464030</a></td>
					<td>1464030</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene spans a genomic range of greater than 2 Mb and encodes a large protein containing an N-terminal actin-binding domain and multiple spectrin repeats. The encoded protein forms a component of the dystrophin-glycoprotein complex (DGC), which bridges the inner cytoskeleton and the extracellular matrix. Deletions, duplications, and point mutations at this gene locus may cause Duchenne muscular dystrophy (DMD), Becker muscular dystrophy (BMD), or cardiomyopathy. Alternative promoter usage and alternative splicing result in numerous distinct transcript variants and protein isoforms for this gene. [provided by RefSeq, Dec 2016]
						<p>Phenotype Name: Duchenne muscular dystrophy</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>3</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=TBX1">TBX1</a></td>
					<td>rs4819522</td></a></td>
					<td>c.1049C>T</td>
					<td>T/C</td><td>MIS</td><td>Damaging</td>
					<td>0.207468145769</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/403525/">403525</a></td>
					<td>403525</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. This gene product shares 98% amino acid sequence identity with the mouse ortholog. DiGeorge syndrome (DGS)/velocardiofacial syndrome (VCFS), a common congenital disorder characterized by neural-crest-related developmental defects, has been associated with deletions of chromosome 22q11.2, where this gene has been mapped. Studies using mouse models of DiGeorge syndrome suggest a major role for this gene in the molecular etiology of DGS/VCFS. Several alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>4</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=RYR1">RYR1</a></td>
					<td>rs2071089</td></a></td>
					<td>c.9186A>G</td>
					<td>G/A</td><td>SYN</td><td>Damaging</td>
					<td>0.322394110213</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/93306/">93306</a></td>
					<td>93306</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a ryanodine receptor found in skeletal muscle. The encoded protein functions as a calcium release channel in the sarcoplasmic reticulum but also serves to connect the sarcoplasmic reticulum and transverse tubule. Mutations in this gene are associated with malignant hyperthermia susceptibility, central core disease, and minicore myopathy with external ophthalmoplegia. Alternatively spliced transcripts encoding different isoforms have been described. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Congenital multicore myopathy with external ophthalmoplegia|Central core myopathy|Malignant hyperthermia, susceptibility to, 1|King Denborough syndrome|Neuromuscular disease, congenital, with uniform type 1 fiber|not specified|RYR1-Related Disorders|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>5</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NOTCH3">NOTCH3</a></td>
					<td>rs115582213</td></a></td>
					<td>c.5854G>A</td>
					<td>T/C</td><td>MIS</td><td>Damaging</td>
					<td>0.00815624911576</td>
					<td>Benign/Likely benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/256146/">256146</a></td>
					<td><a href="https://www.omim.org/entry/125310">125310</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes the third discovered human homologue of the Drosophilia melanogaster type I membrane protein notch. In Drosophilia, notch interaction with its cell-bound ligands (delta, serrate) establishes an intercellular signalling pathway that plays a key role in neural development. Homologues of the notch-ligands have also been identified in human, but precise interactions between these ligands and the human notch homologues remains to be determined. Mutations in NOTCH3 have been identified as the underlying cause of cerebral autosomal dominant arteriopathy with subcortical infarcts and leukoencephalopathy (CADASIL). [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Cerebral arteriopathy, autosomal dominant, with subcortical infarcts and leukoencephalopathy, type 1|Lateral meningocele syndrome|Myofibromatosis, infantile, 2|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>6</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NOTCH3">NOTCH3</a></td>
					<td>rs4809029</td></a></td>
					<td>c.5913+28T>G</td>
					<td>C/C</td><td>INT</td><td>Damaging</td>
					<td>0.868945990553</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/811011/">811011</a></td>
					<td>811011</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes the third discovered human homologue of the Drosophilia melanogaster type I membrane protein notch. In Drosophilia, notch interaction with its cell-bound ligands (delta, serrate) establishes an intercellular signalling pathway that plays a key role in neural development. Homologues of the notch-ligands have also been identified in human, but precise interactions between these ligands and the human notch homologues remains to be determined. Mutations in NOTCH3 have been identified as the underlying cause of cerebral autosomal dominant arteriopathy with subcortical infarcts and leukoencephalopathy (CADASIL). [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Cerebral arteriopathy, autosomal dominant, with subcortical infarcts and leukoencephalopathy, type 1|Lateral meningocele syndrome|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>7</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=ADAMTS10">ADAMTS10</a></td>
					<td>rs7255721</td></a></td>
					<td>c.401G>C</td>
					<td>G/C</td><td>MIS</td><td>Damaging</td>
					<td>None</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/769496/">769496</a></td>
					<td>769496</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene belongs to the ADAMTS (a disintegrin and metalloproteinase domain with thrombospondin type-1 motifs) family of zinc-dependent proteases. ADAMTS proteases are complex secreted enzymes containing a prometalloprotease domain of the reprolysin type attached to an ancillary domain with a highly conserved structure that includes at least one thrombospondin type 1 repeat. They have been demonstrated to have important roles in connective tissue organization, coagulation, inflammation, arthritis, angiogenesis and cell migration. The product of this gene plays a major role in growth and in skin, lens, and heart development. It is also a candidate gene for autosomal recessive Weill-Marchesani syndrome. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>8</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=CCDC40">CCDC40</a></td>
					<td>rs7207166</td></a></td>
					<td>c.2832+381G>A</td>
					<td>A/A</td><td>INT</td><td>Damaging</td>
					<td>0.711252785343</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1283901/">1283901</a></td>
					<td>1283901</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a protein that is necessary for motile cilia function. It functions in correct left-right axis formation by regulating the assembly of the inner dynein arm and the dynein regulatory complexes, which control ciliary beat. Mutations in this gene cause ciliary dyskinesia type 15, a disorder due to defects in cilia motility. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2011]
						<p>Phenotype Name: not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>9</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=CCDC40">CCDC40</a></td>
					<td>rs753389715</td></a></td>
					<td>c.46G>A</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>4.01764549903e-06</td>
					<td>None</td>
					<td>None</td>
					<td>None</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a protein that is necessary for motile cilia function. It functions in correct left-right axis formation by regulating the assembly of the inner dynein arm and the dynein regulatory complexes, which control ciliary beat. Mutations in this gene cause ciliary dyskinesia type 15, a disorder due to defects in cilia motility. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2011]
						<p>Phenotype Name: None</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>10</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DNAH9">DNAH9</a></td>
					<td>rs1990236</td></a></td>
					<td>c.13122G>A</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>0.188142174856</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/402788/">402788</a></td>
					<td>402788</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes the heavy chain subunit of axonemal dynein, a large multi-subunit molecular motor. Axonemal dynein attaches to microtubules and hydrolyzes ATP to mediate the movement of cilia and flagella. The gene expresses at least two transcript variants; additional variants have been described, but their full length nature has not been determined. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>11</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DNAAF1">DNAAF1</a></td>
					<td>rs2288023</td></a></td>
					<td>c.2024G>C</td>
					<td>C/G</td><td>MIS</td><td>Damaging</td>
					<td>0.338545024335</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/163090/">163090</a></td>
					<td><a href="https://www.omim.org/entry/613193">613193</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is cilium-specific and is required for the stability of the ciliary architecture. It is involved in the regulation of microtubule-based cilia and actin-based brush border microvilli. Mutations in this gene are associated with primary ciliary dyskinesia-13. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]
						<p>Phenotype Name: Primary ciliary dyskinesia|Primary ciliary dyskinesia 13|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>12</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DNAAF4">DNAAF4</a></td>
					<td>rs600753</td></a></td>
					<td>c.572A>G</td>
					<td>C/T</td><td>MIS</td><td>Damaging</td>
					<td>0.554209658117</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/262315/">262315</a></td>
					<td>262315</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a tetratricopeptide repeat domain-containing protein. The encoded protein interacts with estrogen receptors and the heat shock proteins, Hsp70 and Hsp90. An homologous protein in rat has been shown to function in neuronal migration in the developing neocortex. A chromosomal translocation involving this gene is associated with a susceptibility to developmental dyslexia. Mutations in this gene are associated with deficits in reading and spelling. Alternative splicing results in multiple transcript variants. Read-through transcription also exists between this gene and the downstream cell cycle progression 1 (CCPG1) gene. [provided by RefSeq, Mar 2011]
						<p>Phenotype Name: Primary ciliary dyskinesia 25|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>13</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE2">SYNE2</a></td>
					<td>rs8010699</td></a></td>
					<td>c.9926A>G</td>
					<td>G/G</td><td>MIS</td><td>Damaging</td>
					<td>0.804419189222</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130521/">130521</a></td>
					<td><a href="https://www.omim.org/entry/612999">612999</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a nuclear outer membrane protein that binds cytoplasmic F-actin. This binding tethers the nucleus to the cytoskeleton and aids in the maintenance of the structural integrity of the nucleus. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009]
						<p>Phenotype Name: Emery-Dreifuss muscular dystrophy 5, autosomal dominant|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>14</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE2">SYNE2</a></td>
					<td>rs8010911</td></a></td>
					<td>c.9757G>C</td>
					<td>C/C</td><td>MIS</td><td>Damaging</td>
					<td>0.803822390229</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130520/">130520</a></td>
					<td><a href="https://www.omim.org/entry/612999">612999</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a nuclear outer membrane protein that binds cytoplasmic F-actin. This binding tethers the nucleus to the cytoskeleton and aids in the maintenance of the structural integrity of the nucleus. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009]
						<p>Phenotype Name: Emery-Dreifuss muscular dystrophy 5, autosomal dominant|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>15</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE2">SYNE2</a></td>
					<td>rs4027402</td></a></td>
					<td>c.6851C>T</td>
					<td>T/T</td><td>MIS</td><td>Damaging</td>
					<td>0.809269092335</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130504/">130504</a></td>
					<td><a href="https://www.omim.org/entry/612999">612999</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a nuclear outer membrane protein that binds cytoplasmic F-actin. This binding tethers the nucleus to the cytoskeleton and aids in the maintenance of the structural integrity of the nucleus. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009]
						<p>Phenotype Name: Emery-Dreifuss muscular dystrophy 5, autosomal dominant|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>16</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE2">SYNE2</a></td>
					<td>rs4902264</td></a></td>
					<td>c.5906T>C</td>
					<td>C/C</td><td>MIS</td><td>Damaging</td>
					<td>0.807647922039</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130501/">130501</a></td>
					<td><a href="https://www.omim.org/entry/612999">612999</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a nuclear outer membrane protein that binds cytoplasmic F-actin. This binding tethers the nucleus to the cytoskeleton and aids in the maintenance of the structural integrity of the nucleus. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009]
						<p>Phenotype Name: Emery-Dreifuss muscular dystrophy 5, autosomal dominant|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>17</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SOS2">SOS2</a></td>
					<td>rs3736760</td></a></td>
					<td>c.2057+38C>T</td>
					<td>A/A</td><td>INT</td><td>Damaging</td>
					<td>0.762008389107</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1266724/">1266724</a></td>
					<td>1266724</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a regulatory protein that is involved in the positive regulation of ras proteins. Mutations in this gene are associated with Noonan Syndrome-9. [provided by RefSeq, Jul 2016]
						<p>Phenotype Name: Noonan syndrome 9|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>18</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DNAAF2">DNAAF2</a></td>
					<td>rs2985684</td></a></td>
					<td>c.186G>C</td>
					<td>G/C</td><td>MIS</td><td>Damaging</td>
					<td>0.64203724391</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/95893/">95893</a></td>
					<td><a href="https://www.omim.org/entry/612518">612518</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a highly conserved protein involved in the preassembly of dynein arm complexes which power cilia. These complexes are found in some cilia and are assembled in the cytoplasm prior to transport for cilia formation. Mutations in this gene have been associated with primary ciliary dyskinesia. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Oct 2009]
						<p>Phenotype Name: Primary ciliary dyskinesia|Primary ciliary dyskinesia 10|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>19</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DNAAF2">DNAAF2</a></td>
					<td>rs9989177</td></a></td>
					<td>c.2303A>G</td>
					<td>C/T</td><td>MIS</td><td>Damaging</td>
					<td>0.441320711679</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/163091/">163091</a></td>
					<td><a href="https://www.omim.org/entry/612518">612518</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a highly conserved protein involved in the preassembly of dynein arm complexes which power cilia. These complexes are found in some cilia and are assembled in the cytoplasm prior to transport for cilia formation. Mutations in this gene have been associated with primary ciliary dyskinesia. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Oct 2009]
						<p>Phenotype Name: Primary ciliary dyskinesia|Primary ciliary dyskinesia 10|Congenital disorder of glycosylation|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>20</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=ACVRL1">ACVRL1</a></td>
					<td>rs706816</td></a></td>
					<td>c.1377+65A>G</td>
					<td>G/A</td><td>INT</td><td>Damaging</td>
					<td>0.269621828556</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/439373/">439373</a></td>
					<td>439373</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a type I cell-surface receptor for the TGF-beta superfamily of ligands. It shares with other type I receptors a high degree of similarity in serine-threonine kinase subdomains, a glycine- and serine-rich region (called the GS domain) preceding the kinase domain, and a short C-terminal tail. The encoded protein, sometimes termed ALK1, shares similar domain structures with other closely related ALK or activin receptor-like kinase proteins that form a subfamily of receptor serine/threonine kinases. Mutations in this gene are associated with hemorrhagic telangiectasia type 2, also known as Rendu-Osler-Weber syndrome 2. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Telangiectasia, hereditary hemorrhagic, type 2|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>21</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=CACNA1C">CACNA1C</a></td>
					<td>rs200231105</td></a></td>
					<td>c.5593G>A</td>
					<td>A/G</td><td>MIS</td><td>Tolerated</td>
					<td>0.000454879584276</td>
					<td>Conflicting interpretations of pathogenicity</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/191566/">191566</a></td>
					<td><a href="https://www.omim.org/entry/601005">601005</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes an alpha-1 subunit of a voltage-dependent calcium channel. Calcium channels mediate the influx of calcium ions into the cell upon membrane polarization. The alpha-1 subunit consists of 24 transmembrane segments and forms the pore through which ions pass into the cell. The calcium channel consists of a complex of alpha-1, alpha-2/delta, beta, and gamma subunits in a 1:1:1:1 ratio. There are multiple isoforms of each of these proteins, either encoded by different genes or the result of alternative splicing of transcripts. The protein encoded by this gene binds to and is inhibited by dihydropyridine. Alternative splicing results in many transcript variants encoding different proteins. Some of the predicted proteins may not produce functional ion channel subunits. [provided by RefSeq, Oct 2012]
						<p>Phenotype Name: Hypertrophic cardiomyopathy|Long QT syndrome|Brugada syndrome|History of neurodevelopmental disorder|not specified|Cardiovascular phenotype</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>22</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FXYD2">FXYD2</a></td>
					<td>rs138991613</td></a></td>
					<td>c.110C>A</td>
					<td>T/G</td><td>MIS</td><td>Damaging</td>
					<td>0.000633551720412</td>
					<td>Likely benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/736288/">736288</a></td>
					<td>736288</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a member of the FXYD family of transmembrane proteins. This particular protein encodes the sodium/potassium-transporting ATPase subunit gamma. Mutations in this gene have been associated with Renal Hypomagnesemia-2. Alternatively spliced transcript variants have been described. Read-through transcripts have been observed between this locus and the upstream FXYD domain-containing ion transport regulator 6 (FXYD6, GeneID 53826) locus.[provided by RefSeq, Feb 2011]
						<p>Phenotype Name: Renal hypomagnesemia 2|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>23</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=MMP3">MMP3</a></td>
					<td>rs679620</td></a></td>
					<td>c.133A>G</td>
					<td>C/C</td><td>MIS</td><td>Damaging</td>
					<td>0.578682343782</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/403098/">403098</a></td>
					<td>403098</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: Proteins of the matrix metalloproteinase (MMP) family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Most MMP&apos;s are secreted as inactive proproteins which are activated when cleaved by extracellular proteinases. This gene encodes an enzyme which degrades fibronectin, laminin, collagens III, IV, IX, and X, and cartilage proteoglycans. The enzyme is thought to be involved in wound repair, progression of atherosclerosis, and tumor initiation. The gene is part of a cluster of MMP genes which localize to chromosome 11q22.3. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>24</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=BAG3">BAG3</a></td>
					<td>rs3858340</td></a></td>
					<td>c.1220C>T</td>
					<td>T/C</td><td>MIS</td><td>Damaging</td>
					<td>0.118487067898</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/44776/">44776</a></td>
					<td><a href="https://www.omim.org/entry/612954; 613881">612954; 613881</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: BAG proteins compete with Hip for binding to the Hsc70/Hsp70 ATPase domain and promote substrate release. All the BAG proteins have an approximately 45-amino acid BAG domain near the C terminus but differ markedly in their N-terminal regions. The protein encoded by this gene contains a WW domain in the N-terminal region and a BAG domain in the C-terminal region. The BAG domains of BAG1, BAG2, and BAG3 interact specifically with the Hsc70 ATPase domain in vitro and in mammalian cells. All 3 proteins bind with high affinity to the ATPase domain of Hsc70 and inhibit its chaperone activity in a Hip-repressible manner. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Myofibrillar myopathy 6|Dilated cardiomyopathy 1HH|not specified|Cardiovascular phenotype|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>25</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=MYPN">MYPN</a></td>
					<td>rs7079481</td></a></td>
					<td>c.3403C>A</td>
					<td>A/A</td><td>MIS</td><td>Damaging</td>
					<td>0.430328419891</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/31800/">31800</a></td>
					<td><a href="https://www.omim.org/entry/615248">615248</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: Striated muscle in vertebrates comprises large proteins which must be organized properly to contract efficiently. Z-lines in striated muscle are a sign of this organization, representing the ends of actin thin filaments, titin, nebulin or nebulette and accessory proteins required for structure and function. This gene encodes a protein which interacts with nebulin in skeletal muscle or nebulette in cardiac muscle and alpha-actinin. In addition, this gene product can interact with a protein with the I-band indicating it has a regulatory as well as structural function. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2011]
						<p>Phenotype Name: Dilated cardiomyopathy 1KK|MYPN-related myopathy|not specified|Cardiovascular phenotype|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>26</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FKTN">FKTN</a></td>
					<td>rs34787999</td></a></td>
					<td>c.608G>A</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>0.249202580096</td>
					<td>Benign/Likely benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/93522/">93522</a></td>
					<td><a href="https://www.omim.org/entry/253800; 611588; 611615; 613152">253800; 611588; 611615; 613152</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a putative transmembrane protein that is localized to the cis-Golgi compartment, where it may be involved in the glycosylation of alpha-dystroglycan in skeletal muscle. The encoded protein is thought to be a glycosyltransferase and could play a role in brain development. Defects in this gene are a cause of Fukuyama-type congenital muscular dystrophy (FCMD), Walker-Warburg syndrome (WWS), limb-girdle muscular dystrophy type 2M (LGMD2M), and dilated cardiomyopathy type 1X (CMD1X). Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Nov 2010]
						<p>Phenotype Name: Walker-Warburg congenital muscular dystrophy|Muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A1|Muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A, 4|Autosomal recessive limb-girdle muscular dystrophy type 2M|Dilated cardiomyopathy 1X|Muscular dystrophy-dystroglycanopathy (congenital without intellectual disability), type B4|not specified|Cardiovascular phenotype|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>27</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FLNC">FLNC</a></td>
					<td>rs2291569</td></a></td>
					<td>c.4700G>A</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>0.0741965669712</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/129091/">129091</a></td>
					<td>129091</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes one of three related filamin genes, specifically gamma filamin. These filamin proteins crosslink actin filaments into orthogonal networks in cortical cytoplasm and participate in the anchoring of membrane proteins for the actin cytoskeleton. Three functional domains exist in filamin: an N-terminal filamentous actin-binding domain, a C-terminal self-association domain, and a membrane glycoprotein-binding domain. Mutations in this gene are a cause of cardiopathy. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2022]
						<p>Phenotype Name: Myofibrillar myopathy 5|Distal myopathy with posterior leg and anterior hand involvement|Hypertrophic cardiomyopathy 26|not specified|Cardiovascular phenotype|Dilated Cardiomyopathy, Dominant|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>28</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=ELN">ELN</a></td>
					<td>rs2071307</td></a></td>
					<td>c.1264G>A</td>
					<td>A/A</td><td>MIS</td><td>Damaging</td>
					<td>0.319828104552</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/43582/">43582</a></td>
					<td><a href="https://www.omim.org/entry/185500">185500</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a protein that is one of the two components of elastic fibers. Elastic fibers comprise part of the extracellular matrix and confer elasticity to organs and tissues including the heart, skin, lungs, ligaments, and blood vessels. The encoded protein is rich in hydrophobic amino acids such as glycine and proline, which form mobile hydrophobic regions bounded by crosslinks between lysine residues. Degradation products of the encoded protein, known as elastin-derived peptides or elastokines, bind the elastin receptor complex and other receptors and stimulate migration and proliferation of monocytes and skin fibroblasts. Elastokines can also contribute to cancer progression. Deletions and mutations in this gene are associated with supravalvular aortic stenosis (SVAS) and autosomal dominant cutis laxa. [provided by RefSeq, Aug 2017]
						<p>Phenotype Name: Supravalvar aortic stenosis|Cutis laxa, autosomal dominant|not specified</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>29</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=PKD1L1">PKD1L1</a></td>
					<td>rs2686817</td></a></td>
					<td>c.934G>T</td>
					<td>A/A</td><td>MIS</td><td>Damaging</td>
					<td>0.493475262581</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1237805/">1237805</a></td>
					<td>1237805</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a member of the polycystin protein family containing 11 transmembrane domains, a receptor for egg jelly (REJ) domain, and a polycystin-1, lipoxygenase, alpha-toxin (PLAT) domain. The encoded protein may play a role in the male reproductive system. Alternative splice variants have been described but their biological nature has not been determined. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>30</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NME8">NME8</a></td>
					<td>rs56128139</td></a></td>
					<td>c.1478T>C</td>
					<td>C/T</td><td>MIS</td><td>Damaging</td>
					<td>0.271294673582</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/178811/">178811</a></td>
					<td>178811</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a protein with an N-terminal thioredoxin domain and three C-terminal nucleoside diphosphate kinase (NDK) domains, but the NDK domains are thought to be catalytically inactive. The sea urchin ortholog of this gene encodes a component of sperm outer dynein arms, and the protein is implicated in ciliary function. Mutations in this gene are implicated in primary ciliary dyskinesia type 6.[provided by RefSeq, Nov 2009]
						<p>Phenotype Name: Primary ciliary dyskinesia|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>31</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=NME8">NME8</a></td>
					<td>rs10250905</td></a></td>
					<td>c.622T>C</td>
					<td>C/T</td><td>MIS</td><td>Damaging</td>
					<td>0.736013477567</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/164801/">164801</a></td>
					<td>164801</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a protein with an N-terminal thioredoxin domain and three C-terminal nucleoside diphosphate kinase (NDK) domains, but the NDK domains are thought to be catalytically inactive. The sea urchin ortholog of this gene encodes a component of sperm outer dynein arms, and the protein is implicated in ciliary function. Mutations in this gene are implicated in primary ciliary dyskinesia type 6.[provided by RefSeq, Nov 2009]
						<p>Phenotype Name: Primary ciliary dyskinesia|Primary ciliary dyskinesia 6|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>32</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DNAAF5">DNAAF5</a></td>
					<td>rs4720951</td></a></td>
					<td>c.1895T>C</td>
					<td>C/T</td><td>MIS</td><td>Damaging</td>
					<td>0.433830778724</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/260930/">260930</a></td>
					<td>260930</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is essential for the preassembly or stability of axonemal dynein arms, and is found only in organisms with motile cilia and flagella. Mutations in this gene are associated with primary ciliary dyskinesia-18, a disorder characterized by abnormalities of motile cilia. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Feb 2013]
						<p>Phenotype Name: Primary ciliary dyskinesia|Primary ciliary dyskinesia 18|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>33</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=DNAAF5">DNAAF5</a></td>
					<td>rs6946758</td></a></td>
					<td>c.1784-1298C>G</td>
					<td>G/C</td><td>INT</td><td>Damaging</td>
					<td>0.427423699075</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1185315/">1185315</a></td>
					<td>1185315</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is essential for the preassembly or stability of axonemal dynein arms, and is found only in organisms with motile cilia and flagella. Mutations in this gene are associated with primary ciliary dyskinesia-18, a disorder characterized by abnormalities of motile cilia. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Feb 2013]
						<p>Phenotype Name: Primary ciliary dyskinesia 18</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>34</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE1">SYNE1</a></td>
					<td>rs214976</td></a></td>
					<td>c.3104T>C</td>
					<td>G/A</td><td>MIS</td><td>Damaging</td>
					<td>0.475412503892</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130438/">130438</a></td>
					<td><a href="https://www.omim.org/entry/610743; 612998">610743; 612998</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a spectrin repeat containing protein expressed in skeletal and smooth muscle, and peripheral blood lymphocytes, that localizes to the nuclear membrane. Mutations in this gene have been associated with autosomal recessive spinocerebellar ataxia 8, also referred to as autosomal recessive cerebellar ataxia type 1 or recessive ataxia of Beauce. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Autosomal recessive ataxia, Beauce type|Emery-Dreifuss muscular dystrophy 4, autosomal dominant|Arthrogryposis multiplex congenita 3, myogenic type|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>35</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE1">SYNE1</a></td>
					<td>rs214950</td></a></td>
					<td>c.8384C>T</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>0.232072397549</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130452/">130452</a></td>
					<td><a href="https://www.omim.org/entry/610743; 612998">610743; 612998</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a spectrin repeat containing protein expressed in skeletal and smooth muscle, and peripheral blood lymphocytes, that localizes to the nuclear membrane. Mutations in this gene have been associated with autosomal recessive spinocerebellar ataxia 8, also referred to as autosomal recessive cerebellar ataxia type 1 or recessive ataxia of Beauce. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Autosomal recessive ataxia, Beauce type|Emery-Dreifuss muscular dystrophy 4, autosomal dominant|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>36</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE1">SYNE1</a></td>
					<td>rs4407724</td></a></td>
					<td>c.10191C>A</td>
					<td>T/T</td><td>SYN</td><td>Damaging</td>
					<td>0.658192709733</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130388/">130388</a></td>
					<td>130388</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a spectrin repeat containing protein expressed in skeletal and smooth muscle, and peripheral blood lymphocytes, that localizes to the nuclear membrane. Mutations in this gene have been associated with autosomal recessive spinocerebellar ataxia 8, also referred to as autosomal recessive cerebellar ataxia type 1 or recessive ataxia of Beauce. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Autosomal recessive ataxia, Beauce type|Emery-Dreifuss muscular dystrophy 4, autosomal dominant|Arthrogryposis multiplex congenita 3, myogenic type|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>37</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE1">SYNE1</a></td>
					<td>rs4645434</td></a></td>
					<td>c.12180G>T</td>
					<td>A/A</td><td>MIS</td><td>Damaging</td>
					<td>0.580499724183</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130397/">130397</a></td>
					<td><a href="https://www.omim.org/entry/610743; 612998">610743; 612998</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a spectrin repeat containing protein expressed in skeletal and smooth muscle, and peripheral blood lymphocytes, that localizes to the nuclear membrane. Mutations in this gene have been associated with autosomal recessive spinocerebellar ataxia 8, also referred to as autosomal recessive cerebellar ataxia type 1 or recessive ataxia of Beauce. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Autosomal recessive ataxia, Beauce type|Emery-Dreifuss muscular dystrophy 4, autosomal dominant|Arthrogryposis multiplex congenita 3, myogenic type|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>38</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SYNE1">SYNE1</a></td>
					<td>rs2295190</td></a></td>
					<td>c.26221C>A</td>
					<td>T/G</td><td>MIS</td><td>Damaging</td>
					<td>0.115176487232</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/130436/">130436</a></td>
					<td><a href="https://www.omim.org/entry/610743; 612998">610743; 612998</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a spectrin repeat containing protein expressed in skeletal and smooth muscle, and peripheral blood lymphocytes, that localizes to the nuclear membrane. Mutations in this gene have been associated with autosomal recessive spinocerebellar ataxia 8, also referred to as autosomal recessive cerebellar ataxia type 1 or recessive ataxia of Beauce. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Autosomal recessive ataxia, Beauce type|Emery-Dreifuss muscular dystrophy 4, autosomal dominant|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>39</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=LAMA4">LAMA4</a></td>
					<td>rs11757455</td></a></td>
					<td>c.460C>T</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>0.0511531051677</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/44392/">44392</a></td>
					<td>44392</td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the alpha chain isoform laminin, alpha 4. The domain structure of alpha 4 is similar to that of alpha 3, both of which resemble truncated versions of alpha 1 and alpha 2, in that approximately 1,200 residues at the N-terminus (domains IV, V and VI) have been lost. Laminin, alpha 4 contains the C-terminal G domain which distinguishes all alpha chains from the beta and gamma chains. The RNA analysis from adult and fetal tissues revealed developmental regulation of expression, however, the exact function of laminin, alpha 4 is not known. Tissue-specific utilization of alternative polyA-signal has been described in literature. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Aug 2011]
						<p>Phenotype Name: Dilated cardiomyopathy 1JJ|not specified</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>40</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=HFE">HFE</a></td>
					<td>rs1800562</td></a></td>
					<td>c.845G>A</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>0.0337711600521</td>
					<td>Conflicting interpretations of pathogenicity|other|risk factor</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/9/">9</a></td>
					<td><a href="https://www.omim.org/entry/612635; 235200; 614193">612635; 235200; 614193</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a membrane protein that is similar to MHC class I-type proteins and associates with beta2-microglobulin (beta2M). It is thought that this protein functions to regulate iron absorption by regulating the interaction of the transferrin receptor with transferrin. The iron storage disorder, hereditary haemochromatosis, is a recessive genetic disorder that results from defects in this gene. [provided by RefSeq, May 2022]
						<p>Phenotype Name: Alzheimer disease, susceptibility to|HFE-related disorder|Porphyria cutanea tarda, susceptibility to|Porphyria variegata, susceptibility to|Abnormality of the nervous system|Behavioral abnormality|Abnormal peripheral nervous system morphology|Cutaneous photosensitivity|Cardiomyopathy|Abdominal pain|Alzheimer disease|Peripheral neuropathy|Abnormality of the male genitalia|Porphyrinuria|Pain|Hereditary hemochromatosis|Microvascular complications of diabetes, susceptibility to, 7|Hereditary cancer-predisposing syndrome|Hemochromatosis type 2|Hemochromatosis type 1|Bronze diabetes|Hemochromatosis, juvenile, digenic|Transferrin serum level quantitative trait locus 2|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>41</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=FBN2">FBN2</a></td>
					<td>rs13180243</td></a></td>
					<td>c.952+4177C>T</td>
					<td>A/G</td><td>INT</td><td>Damaging</td>
					<td>0.163361307955</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/802154/">802154</a></td>
					<td><a href="https://www.omim.org/entry/121050">121050</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a component of connective tissue microfibrils and may be involved in elastic fiber assembly. Mutations in this gene cause congenital contractural arachnodactyly. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Congenital contractural arachnodactyly</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>42</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=ANK2">ANK2</a></td>
					<td>rs34270799</td></a></td>
					<td>c.9900C>A</td>
					<td>A/C</td><td>MIS</td><td>Damaging</td>
					<td>0.0193832599119</td>
					<td>Benign/Likely benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/191563/">191563</a></td>
					<td><a href="https://www.omim.org/entry/600919">600919</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a member of the ankyrin family of proteins that link the integral membrane proteins to the underlying spectrin-actin cytoskeleton. Ankyrins play key roles in activities such as cell motility, activation, proliferation, contact and the maintenance of specialized membrane domains. Most ankyrins are typically composed of three structural domains: an amino-terminal domain containing multiple ankyrin repeats; a central region with a highly conserved spectrin binding domain; and a carboxy-terminal regulatory domain which is the least conserved and subject to variation. The protein encoded by this gene is required for targeting and stability of Na/Ca exchanger 1 in cardiomyocytes. Mutations in this gene cause long QT syndrome 4 and cardiac arrhythmia syndrome. Multiple transcript variants encoding different isoforms have been described. [provided by RefSeq, Dec 2011]
						<p>Phenotype Name: Long QT syndrome|Cardiac arrhythmia, ankyrin-B-related|not specified|Cardiovascular phenotype|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>43</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=EVC2">EVC2</a></td>
					<td>rs4689278</td></a></td>
					<td>c.688A>G</td>
					<td>C/T</td><td>MIS</td><td>Damaging</td>
					<td>0.236896081025</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/262620/">262620</a></td>
					<td><a href="https://www.omim.org/entry/225500">225500</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a protein that functions in bone formation and skeletal development. Mutations in this gene, as well as in a neighboring gene that lies in a head-to-head configuration, cause Ellis-van Creveld syndrome, an autosomal recessive skeletal dysplasia that is also known as chondroectodermal dysplasia. Mutations in this gene also cause acrofacial dysostosis Weyers type, also referred to as Curry-Hall syndrome, a disease that combines limb and facial abnormalities. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2009]
						<p>Phenotype Name: Curry-Hall syndrome|Ellis-van Creveld syndrome|not specified</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>44</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=CLDN16">CLDN16</a></td>
					<td>rs3214506</td></a></td>
					<td>c.-45G>C</td>
					<td>C/C</td><td>UT5</td><td>Damaging</td>
					<td>0.192403888393</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/344445/">344445</a></td>
					<td><a href="https://www.omim.org/entry/248250">248250</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: Tight junctions represent one mode of cell-to-cell adhesion in epithelial or endothelial cell sheets, forming continuous seals around cells and serving as a physical barrier to prevent solutes and water from passing freely through the paracellular space. These junctions are comprised of sets of continuous networking strands in the outwardly facing cytoplasmic leaflet, with complementary grooves in the inwardly facing extracytoplasmic leaflet. The protein encoded by this gene, a member of the claudin family, is an integral membrane protein and a component of tight junction strands. It is found primarily in the kidneys, specifically in the thick ascending limb of Henle, where it acts as either an intercellular pore or ion concentration sensor to regulate the paracellular resorption of magnesium ions. Defects in this gene are a cause of primary hypomagnesemia, which is characterized by massive renal magnesium wasting with hypomagnesemia and hypercalciuria, resulting in nephrocalcinosis and renal failure. This gene and the CLDN1 gene are clustered on chromosome 3q28. [provided by RefSeq, Jun 2010]
						<p>Phenotype Name: Primary hypomagnesemia|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>45</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=MYLK">MYLK</a></td>
					<td>rs9840993</td></a></td>
					<td>c.439C>T</td>
					<td>A/A</td><td>MIS</td><td>Damaging</td>
					<td>0.94149056791</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/198606/">198606</a></td>
					<td><a href="https://www.omim.org/entry/613780">613780</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene, a muscle member of the immunoglobulin gene superfamily, encodes myosin light chain kinase which is a calcium/calmodulin dependent enzyme. This kinase phosphorylates myosin regulatory light chains to facilitate myosin interaction with actin filaments to produce contractile activity. This gene encodes both smooth muscle and nonmuscle isoforms. In addition, using a separate promoter in an intron in the 3&apos; region, it encodes telokin, a small protein identical in sequence to the C-terminus of myosin light chain kinase, that is independently expressed in smooth muscle and functions to stabilize unphosphorylated myosin filaments. A pseudogene is located on the p arm of chromosome 3. Four transcript variants that produce four isoforms of the calcium/calmodulin dependent enzyme have been identified as well as two transcripts that produce two isoforms of telokin. Additional variants have been identified but lack full length transcripts. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Aortic aneurysm, familial thoracic 7|Familial thoracic aortic aneurysm and aortic dissection|Megacystis, microcolon, hypoperistalsis syndrome|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>46</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=TTN">TTN</a></td>
					<td>rs10497520</td></a></td>
					<td>c.3601A>G</td>
					<td>C/C</td><td>MIS</td><td>Damaging</td>
					<td>0.688710066049</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/46973/">46973</a></td>
					<td><a href="https://www.omim.org/entry/600334; 604145; 608807; 611705; 603689">600334; 604145; 608807; 611705; 603689</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a large abundant protein of striated muscle. The product of this gene is divided into two regions, a N-terminal I-band and a C-terminal A-band. The I-band, which is the elastic part of the molecule, contains two regions of tandem immunoglobulin domains on either side of a PEVK region that is rich in proline, glutamate, valine and lysine. The A-band, which is thought to act as a protein-ruler, contains a mixture of immunoglobulin and fibronectin repeats, and possesses kinase activity. An N-terminal Z-disc region and a C-terminal M-line region bind to the Z-line and M-line of the sarcomere, respectively, so that a single titin molecule spans half the length of a sarcomere. Titin also contains binding sites for muscle associated proteins so it serves as an adhesion template for the assembly of contractile machinery in muscle cells. It has also been identified as a structural protein for chromosomes. Alternative splicing of this gene results in multiple transcript variants. Considerable variability exists in the I-band, the M-line and the Z-disc regions of titin. Variability in the I-band region contributes to the differences in elasticity of different titin isoforms and, therefore, to the differences in elasticity of different muscle types. Mutations in this gene are associated with familial hypertrophic cardiomyopathy 9, and autoantibodies to titin are produced in patients with the autoimmune disease scleroderma. [provided by RefSeq, Feb 2012]
						<p>Phenotype Name: Tibial muscular dystrophy|Myopathy, myofibrillar, 9, with early respiratory failure|Dilated cardiomyopathy 1G|Autosomal recessive limb-girdle muscular dystrophy type 2J|Early-onset myopathy with fatal cardiomyopathy|not specified|Cardiovascular phenotype|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>47</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=TTN">TTN</a></td>
					<td>rs2291311</td></a></td>
					<td>c.9781G>A</td>
					<td>T/T</td><td>MIS</td><td>Damaging</td>
					<td>0.844463342901</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/47692/">47692</a></td>
					<td><a href="https://www.omim.org/entry/600334; 604145; 608807; 611705; 603689">600334; 604145; 608807; 611705; 603689</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a large abundant protein of striated muscle. The product of this gene is divided into two regions, a N-terminal I-band and a C-terminal A-band. The I-band, which is the elastic part of the molecule, contains two regions of tandem immunoglobulin domains on either side of a PEVK region that is rich in proline, glutamate, valine and lysine. The A-band, which is thought to act as a protein-ruler, contains a mixture of immunoglobulin and fibronectin repeats, and possesses kinase activity. An N-terminal Z-disc region and a C-terminal M-line region bind to the Z-line and M-line of the sarcomere, respectively, so that a single titin molecule spans half the length of a sarcomere. Titin also contains binding sites for muscle associated proteins so it serves as an adhesion template for the assembly of contractile machinery in muscle cells. It has also been identified as a structural protein for chromosomes. Alternative splicing of this gene results in multiple transcript variants. Considerable variability exists in the I-band, the M-line and the Z-disc regions of titin. Variability in the I-band region contributes to the differences in elasticity of different titin isoforms and, therefore, to the differences in elasticity of different muscle types. Mutations in this gene are associated with familial hypertrophic cardiomyopathy 9, and autoantibodies to titin are produced in patients with the autoimmune disease scleroderma. [provided by RefSeq, Feb 2012]
						<p>Phenotype Name: Tibial muscular dystrophy|Myopathy, myofibrillar, 9, with early respiratory failure|Dilated cardiomyopathy 1G|Autosomal recessive limb-girdle muscular dystrophy type 2J|Early-onset myopathy with fatal cardiomyopathy|not specified|Cardiovascular phenotype|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>48</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=AGL">AGL</a></td>
					<td>rs2230307</td></a></td>
					<td>c.3343G>A</td>
					<td>A/G</td><td>MIS</td><td>Damaging</td>
					<td>0.071776646156</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/256738/">256738</a></td>
					<td><a href="https://www.omim.org/entry/232400">232400</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes the glycogen debrancher enzyme which is involved in glycogen degradation. This enzyme has two independent catalytic activities which occur at different sites on the protein: a 4-alpha-glucotransferase activity and a amylo-1,6-glucosidase activity. Mutations in this gene are associated with glycogen storage disease although a wide range of enzymatic and clinical variability occurs which may be due to tissue-specific alternative splicing. Alternatively spliced transcripts encoding different isoforms have been described. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Glycogen storage disease type III|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>49</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=CPT2">CPT2</a></td>
					<td>rs1799822</td></a></td>
					<td>c.1939A>G</td>
					<td>G/A</td><td>MIS</td><td>Damaging</td>
					<td>0.162231592375</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/92433/">92433</a></td>
					<td><a href="https://www.omim.org/entry/600649; 608836">600649; 608836</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: The protein encoded by this gene is a nuclear protein which is transported to the mitochondrial inner membrane. Together with carnitine palmitoyltransferase I, the encoded protein oxidizes long-chain fatty acids in the mitochondria. Defects in this gene are associated with mitochondrial long-chain fatty-acid (LCFA) oxidation disorders. [provided by RefSeq, Jul 2008]
						<p>Phenotype Name: Carnitine palmitoyl transferase II deficiency, severe infantile form|Carnitine palmitoyl transferase II deficiency, neonatal form|Carnitine palmitoyltransferase II deficiency|not specified|not provided</p>
						</td></tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>50</td><td><a href="https://genomics.senescence.info/longevity/gene.php?id=SELENON">SELENON</a></td>
					<td>rs2294228</td></a></td>
					<td>c.1506C>A</td>
					<td>A/A</td><td>MIS</td><td>Damaging</td>
					<td>0.771142168881</td>
					<td>Benign</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/95959/">95959</a></td>
					<td><a href="https://www.omim.org/entry/602771">602771</a></td>
					</tr>
					<tr class="hidden" style="display:none;"><td colspan="11">NCBI gene: This gene encodes a glycoprotein that is localized in the endoplasmic reticulum. It plays an important role in cell protection against oxidative stress, and in the regulation of redox-related calcium homeostasis. Mutations in this gene are associated with early onset muscle disorders, referred to as SEPN1-related myopathy. SEPN1-related myopathy consists of 4 autosomal recessive disorders, originally thought to be separate entities: rigid spine muscular dystrophy (RSMD1), the classical form of multiminicore disease, desmin related myopathy with Mallory-body like inclusions, and congenital fiber-type disproportion (CFTD). This protein is a selenoprotein, containing the rare amino acid selenocysteine (Sec). Sec is encoded by the UGA codon, which normally signals translation termination. The 3&apos; UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, designated the Sec insertion sequence (SECIS) element, that is necessary for the recognition of UGA as a Sec codon, rather than as a stop signal. A second stop-codon redefinition element (SRE) adjacent to the UGA codon has been identified in this gene (PMID:15791204). SRE is a phylogenetically conserved stem-loop structure that stimulates readthrough at the UGA codon, and augments the Sec insertion efficiency by SECIS. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2016]
						<p>Phenotype Name: Eichsfeld type congenital muscular dystrophy|not specified|SEPN1-Related Disorders|not provided</p>
						</td></tr>
				
		</tbody>
	</table>
	</div>
	<div class="last"></div>
</div>
<button class="accordion accordion-heading major-risk"  >Lipid Metabolism</button>
<div class="basic-panel major-risk">
	<button class="accordion accordion0 accordion1 major-risk"><div></div>Description</button>
	<div class="board variants panel panel-description major-risk">
		<p>Lipid metabolism plays an important role in the development of heart disease, and this risk increases with age. Cholesterol is a type of lipid that is carried in the blood by lipoproteins, including low-density lipoprotein (LDL) and high-density lipoprotein (HDL). LDL is often referred to as "bad" cholesterol, as high levels of LDL are associated with an increased risk of CAD. HDL, on the other hand, is often referred to as "good" cholesterol, as it helps to remove cholesterol from the bloodstream and may have a protective effect against CAD.
As we age, our lipid metabolism tends to change. For example, LDL cholesterol levels tend to increase, while HDL cholesterol levels tend to decrease. This can increase the risk of CAD and other heart diseases.
</p>
		<p>Genetics plays an important role in lipid metabolism and the development of heart disease. Genes control the production, transport, and metabolism of cholesterol and other lipids.</p>
		<p>This report contains data of the PRS of Coronary atherosclerosis development (<a href="https://www.pgscatalog.org/score/PGS001839/">PGS001839</a>), and the list with genes, associated with an increased risk of CAD.
It's important to note, however, that genetics is just one of many factors that contribute to lipid metabolism and the development of heart disease. Lifestyle factors, such as diet and physical activity, are also important contributors to lipid metabolism and the development of heart disease, and they can modify the effects of genetic factors.
</p>
		</div>
		<button class="accordion accordion0 accordion1 prs-panel major-risk"><div></div>PRS</button>
	<div class="board variants panel prs-panel major-risk">
		<p class="how_to">How to read the result?</p>
<p></p>
<p>PRS is represented as a&nbsp;<u>percentile</u> within a given population. For example, you have the 70th percentile, which means your personal genetic risk of the disease development is higher than 70 out of every 100 people in a chosen population.
</p>
<p></p>
<p>So percentile &#65310;50 shows an increased risk, &#65308;50 - decreased one.</p>
<p></p>

<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Coronary atherosclerosis (<a href="https://www.pgscatalog.org/score/PGS001839/">PGS001839</a>)</td><td rowspan="6"><canvas id="PGS001839_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>1.041014480505508</td></tr>
    <tr><td>Count/total</td><td>15920/25425</td></tr>
    <tr><td>Average</td><td>3.2695178407836306e-05</td></tr>
    <tr><td>Percentile</td><td>79%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS001839_canvas", 0.79, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>
		</div>
	<button class="accordion0 accordion accordion-result major-risk"><div></div>Results</button>
<div class="content content-variants major-risk">
		<table class="ui selectable celled teal table  table cancer bigger">
		<thead>
		<tr><th style="width:1%" class="clickable title" onclick="collapseExpandAll(this, 'lipidmetabolismTBody')"><div class="ui tiny circular compact positive button">+</div></th><th>#</th>
				<th>RSID</th><th>Gene</th><th>Risk Allele</th><th>Genotype</th><th>Pubmed ID</th>
				<th>Population</th><th>P-Value</th><th>Weight</th></tr>
		</thead>
		<tbody id="lipidmetabolismTBody">
			
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>1</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs429358">rs429358</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APOE">APOE</a></td>
					<td>C</td>
					<td>CC</td><td>[PMID 20429872]; [PMID 34887591]; [PMID: 34887591];</td><td>European; Asian; African;</td>
					<td>6,00E-256</td><td style="background-color:#ff0000">-3.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>Apolipoprotein E is encoded by the APOE gene, which plays a key role in lipid transport and metabolism and is essential in cholesterol homeostasis. Two SNPs lead to the formation of three APOE isoforms, E2, E3 and E4. Polymorphisms in APOE gene has been linked to cardiovascular disease (CVD), immunoregulation, cognitive decline and Alzheimer’s disease (AD), as well as infectious diseases. Carriers of the E4 allele tend to exhibit an increased risk for the above-mentioned chronic diseases. APOE E4 carriers may be more responsive to their external environment, where focusing on decreasing environmental stressors and pollution, ensuring quantity and quality of dietary fat is addressed, and increasing intake of phytonutrient rich foods, as well as including sufficient exercise, may prove beneficial in negating the effects of the SNP.<br/>
					<b>Genotype specific conclusion: </b> You are a carrier of unfavourable C-allele in rs429358, which ia a part of e4 APOE genotype estimation. If your genotype also harbors the rs7412(C) allele, the combination is known as an APOE-ε4 allele. The APOE-ε4 allele has a strong influence on the risk of Alzheimer's disease.However, it is important to note that this genetic variant is only one of many factors that can contribute to the risk of developing Alzheimer's disease, and having this variant does not necessarily mean that a person will definitely develop the disease. <br/>
					<b>Study design: </b>Discovery sample description: 40,963 South Asian ancestry individuals, 48,057 Hispanic individuals, 99,432 Admixed African or African ancestry individuals, 146,492 East Asian ancestry individuals, 1,320,016 European ancestry individuals Discovery ancestry (country of recruitment): 40963 South Asian (Pakistan, Sri Lanka, U.K., India), 48057 Hispanic or Latin American (U.S., Mexico), 99432 African American or Afro-Caribbean, African unspecified (Russian Federation, Costa Rica, Netherlands, U.S., El Salvador, Panama, Brazil, Poland, U.K., Bulgaria, Chile, Colombia, Argentina, Romania, Uruguay, Switzerland, Canada, Belgium, Norway, Denmark, Mexico, South Africa, Israel, Germany, Estonia), 146492 East Asian (Singapore, U.S., China, Japan, Taiwan, Republic of Korea), 1320016 European (Republic of Ireland, NR, Switzerland, Iceland, Greece, Canada, Netherlands, Sweden, Austria, U.S., Norway, Finland, Denmark, Italy, U.K., Australia, France, Germany, Croatia);</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>2</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs268">rs268</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LPL">LPL</a></td>
					<td>G</td>
					<td>GG</td><td>[PMID 22399527]; [PMID: 20429872]; [PMID 16741292]; [PMID 16651467];</td><td>European;</td>
					<td>2,00E-12</td><td style="background-color:#ff6666">-1.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>LPL is an enzyme that plays a key role in the breakdown and utilization of fats in the body. rs268, also known as LPL Asn291Ser as well as LPL N291S or N318S, a SNP in the lipoprotein lipase LPL gene, has been linked to increased susceptibility to hypertriglyceridemia, heart disease, Type-2 diabetes, idiopathic venous thromboembolism. This mutation has been shown to alter LPL activity and increase the risk of hypertriglyceridemia, a condition characterized by high levels of triglycerides in the blood. It's important to note that the impact of the N291S mutation on lipid metabolism and the risk of metabolic disorders can vary depending on other genetic and environmental factors. <br/>
					<b>Genotype specific conclusion: </b> GG genotype carriers may have a decreased activity of LPL and increased risk for venous thromboembolism (3x increased according to the [PMID 16651467]). This genotype is a risk factor for dyslipidemia, characterized by hypertriglyceridemia and low HDL-C levels. This LPL gene variant predisposes to more severe dyslipidemia with increasing age and weight gain [PMID: 16741292]. It's important to note that the impact of the rs268 mutation on lipid metabolism and the risk of metabolic disorders can vary depending on other genetic and environmental factors. <br/>
					<b>Study design: </b>Discovery sample description: 2,637 European ancestry cases, 7,927 European ancestry controls Discovery ancestry (country of recruitment): 10564 European (Finland)</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>3</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs676210">rs676210</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APOB">APOB</a></td>
					<td>G</td>
					<td>GG</td><td>[PMID: 26629186]; [PMID: 34156559]; [PMID: 23247145]</td><td>European, Chinese;</td>
					<td>6,00E-197</td><td style="background-color:#ff6666">-1.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>Encodes the apolipoprotein B protein, which is a major component of low-density lipoprotein (LDL) cholesterol. Mutations in this gene have been associated with high LDL cholesterol levels and an increased risk of heart disease. Apolipoprotein B (apoB), a key component of VLDL particles and microsomal triacylglycerol transfer protein (MTP), plays an important role in lipid metabolism. Reduced secretion of apoB protein can result in reduced production of chylomicron and VLDL and then leading to malabsorption of fats and fat-soluble vitamins. This SNP was shown to affect atorvastatin response among Iraqi population: in response to 40 mg/day of atorvastatin therapy, the minor allele (A) observed a greater LDL-C lowering than the wild type allele (G) [PMID: 34156559]. This SNP was shown to have association with Acute Coronary Syndrome in Mexican population [PMID: 35440891].<br/>
					<b>Genotype specific conclusion: </b> G allele is unfavourable. Carriers of G allele have elevated risk of hyperlipidemia. AG and GG genotypes were shown to be associated with hyperlipidemia in in the Chinese Yugur population [PMID: 28902930]. A GWAS study by AtheroRemo Consortium showed AG and GG genotypes are more likely to have elevated LDL level, but not cardiovascular events [PMID: 23247145]<br/>
					<b>Study design: </b>Discovery sample description: 441,016 European ancestry individuals Discovery ancestry (country of recruitment): 441016 European (NR)</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>4</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs328">rs328</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=LPL">LPL</a></td>
					<td>G</td>
					<td>GG</td><td>[PMID: 18193044]; [PMID: 30944368]; [PMID: 19148283];</td><td>European; African</td>
					<td>2,00E-28</td><td style="background-color:#ff8484">-0.8</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>The LPL gene provides instructions for making the enzyme lipoprotein lipase, which breaks down fats in the body. To be precise it is responsible for the hydrolysis of triglycerides and very low-density lipoproteins (VLDL).  A particular variant of the LPL gene, called rs328, has been studied for its potential associations with various health conditions. This SNP is also called LPL 1595C>G or LPL Ser447X. The S447X mutation is a variant in the LPL (lipoprotein lipase) gene that results in a deficiency of functional LPL enzyme. This mutation has been associated with impaired lipid metabolism and an increased risk of certain metabolic disorders. Individuals with the S447X mutation may have high levels of triglycerides in their blood, which can increase the risk of pancreatitis and other health problems. This mutation may also increase the risk of developing coronary artery disease, a condition in which the arteries that supply blood to the heart become narrowed or blocked. The S447X mutation is inherited in an autosomal recessive manner, which means that a person must inherit two copies of the mutated gene (one from each parent) to have the condition. Individuals with only one copy of the mutated gene are called carriers and do not typically have any symptoms or health problems related to the mutation. It's important to note that the impact of the S447X mutation on lipid metabolism and the risk of metabolic disorders can vary depending on other genetic and environmental factors.<br/>
					<b>Genotype specific conclusion: </b> You have an unfavourable variant of S447X (rs328) LPL polymorphism. That incrase risk of  having high levels of triglycerides in blood. This mutation may also increase the risk of developing coronary artery disease, a condition in which the arteries that supply blood to the heart become narrowed or blocked. It have to mentioned, that risk of metabolic disorders can vary depending on other genetic and environmental factors. <br/>
					<b>Study design: </b>Discovery sample description: 2,758 European ancestry individuals Discovery ancestry (country of recruitment): 2758 European (NR)</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>5</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7679">rs7679</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=PCIF1">PCIF1</a></td>
					<td>C</td>
					<td>CC</td><td>[PMID: 19060906]; [PMID: 19965587];</td><td>European;</td>
					<td>7,00E-11</td><td style="background-color:#ff8484">-0.8</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>The PCIF1 gene provides instructions for making a protein called PCI domain-containing family 1 protein. This protein is involved in the regulation of lipid (fat) metabolism and energy homeostasis in the body. Studies have suggested that the PCIF1 protein may be involved in the breakdown and storage of fats, as well as in the regulation of cholesterol and triglyceride levels in the blood. Variations in the PCIF1 gene may be associated with changes in lipid metabolism and an increased risk of certain health conditions, such as obesity and metabolic syndrome. <br/>
					<b>Genotype specific conclusion: </b> The minor allele C was associated with decreased HDL cholesterol and increased triglycerides. Your genotype CC is associated with lipid profile disturbance risk.<br/>
					<b>Study design: </b>Discovery sample description: 19,840 European ancestry individuals Discovery ancestry (country of recruitment): 19840 European (Finland, Sweden, U.S., Italy, U.K., France); </td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>6</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1800961">rs1800961</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=HNF4A">HNF4A</a></td>
					<td>T</td>
					<td>TT</td><td>[PMID: 24097068]; [PMID: 18660489]; [PMID: 30698716];</td><td>European;</td>
					<td>2,00E-34</td><td style="background-color:#ff8484">-0.8</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>HNF4A (Hepatocyte nuclear factor 4 alpha) is a transcription factor that plays a key role in the regulation of hepatic (liver) gene expression. HNF4A has been shown to be involved in the regulation of lipid metabolism, specifically the synthesis and metabolism of cholesterol and lipoproteins in the liver. HNF4A has also been linked to the development of metabolic disorders such as diabetes and non-alcoholic fatty liver disease (NAFLD). Mutations in the HNF4A gene have been associated with an increased risk of developing these disorders.<br/>
					<b>Genotype specific conclusion: </b> The minor allele of this SNP (T) was associated with decreased HDL ("good") cholesterol, so is a risk allele.  Alcohol consumption has a negative impact on cholesterole level for T-allele carriers [PMID: 30698716]<br/>
					<b>Study design: </b>Discovery sample description: 94,595 European ancestry individuals Discovery ancestry (country of recruitment): 94595 European (NR)</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>7</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs12740374">rs12740374</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=CELSR2">CELSR2</a></td>
					<td>T</td>
					<td>TT</td><td>[PMID: 18262040]; [PMID: 25350695]; [PMID: 28577571];</td><td>European; African;</td>
					<td>2,00E-26</td><td style="background-color:#ff8484">-0.8</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>CELSR2 and PSRC1 are mapped to the same locus as the SORT1 gene. The SORT1 gene encodes sortilin which is involved in the cholesterol homeostasis in humans. Sortilin increases the hepatic output of VLDL which acts as a precursor for LDL in plasma. GWAS study showed minor allele is associated with increased LDL [PMID: 18262040]; Pharmacogenetic meta-analysis of genome-wide association studies of LDL cholesterol response to statins found each minor allele to increase statin effectiveness [PMID: 25350695];<br/>
					<b>Genotype specific conclusion: </b> TT genotype is associated with increased risk of LDL elevation, but the better response to statins. <br/>
					<b>Study design: </b>Reported trait: LDL cholesterol levels Genotyping technology: Genome-wide genotyping array, Genome-wide sequencing Discovery sample description: 14,126 African ancestry individuals Discovery ancestry (country of recruitment): 14126 Sub-Saharan African (Kenya, Uganda, South Africa, Ghana, Nigeria)</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>8</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6756629">rs6756629</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ABCG8, ABCG5">ABCG8, ABCG5</a></td>
					<td>G</td>
					<td>GG</td><td>[PMID: 20832063]; [PMID:19060911];</td><td>European;</td>
					<td>3,00E-06</td><td style="background-color:#ffb2b2">-0.5</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>ABCG5/ABCG8 are key in regulating whole-body sterol trafficking, by eliminating sterols via the biliary tree as well as the intestinal tract. ABCG5/ABCG8 are genetically associated with lipid levels and downstream atherosclerotic disease, as well as formation of gallstones and the risk of gallbladder cancer.<br/>
					<b>Genotype specific conclusion: </b> G allele is a risk allele. You genotype is associated with increased risk of elevated LDL and TG.<br/>
					<b>Study design: </b>Genotyping technology: Genome-wide genotyping array Discovery sample description: 17,083 European ancestry individuals, 714 Orcadian individuals Discovery ancestry (country of recruitment): 17797 European (Finland, Sweden, Australia, Italy, Netherlands, Germany, U.K., Croatia, Norway, Denmark)</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>9</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2967605">rs2967605</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=RAB11B">RAB11B</a></td>
					<td>T</td>
					<td>TT</td><td>[PMID: 29507422];</td><td>European;</td>
					<td>8,00E-09</td><td style="background-color:#ffd1d1">-0.3</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>RAB11B is a member of the RAB family of small GTPases, which play a role in intracellular membrane transport. RAB11B has been shown to be involved in the regulation of lipid metabolism, specifically the recycling of endocytosed lipoproteins in the lysosome. Dysregulation of RAB11B has been linked to the development of various diseases, including obesity and diabetes. <br/>
					<b>Genotype specific conclusion: </b> The minor allele of this SNP (T) was associated with decreased HDL cholesterol, so is unfavourable. You are carrying two unfavourable alleles of this SNP. <br/>
					<b>Study design: </b>Discovery sample description: 76,627 European ancestry individuals, 7,795 Hispanic individuals, 6,855 East Asian ancestry individuals, 2,958 African American individuals, 439 South Asian ancestry individuals Discovery ancestry (country of recruitment): 76627 European (NR), 7795 Hispanic or Latin American (NR), 2958 African American or Afro-Caribbean (NR), 439 South Asian (NR)</td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>10</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs7412">rs7412</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=APOE">APOE</a></td>
					<td>C</td>
					<td>CC</td><td>[PMID 20406466]; [PMID 20429872]; [PMID 22174202]; [PMID: 34887591]; [PMID: 34887591]; [PMID: 28270201];</td><td>European; Asian; African;</td>
					<td>6,00E-14</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>The APOE gene provides instructions for making a protein called apolipoprotein E. This protein plays a role in the metabolism of lipids (fats) in the body, specifically in the transport of cholesterol and other fats in the blood. There are three different versions (alleles) of the APOE gene: E2, E3, and E4. Each person inherits one copy of the APOE gene from each parent, so an individual can have two copies of the same allele (such as E2/E2 or E4/E4) or one copy of each (such as E3/E4). The specific combination of alleles that an individual has can influence their risk of developing certain health conditions. Research has suggested that E4 allele is  linked to cardiovascular disease (CVD), immunoregulation, cognitive decline and Alzheimer’s disease (AD), as well as infectious diseases. Carriers of the E4 allele tend to exhibit an increased risk for the above-mentioned chronic diseases. Individuals with the E2 allele may have a lower risk of these conditions. APOE E4 carriers may be more responsive to their external environment, where focusing on decreasing environmental stressors and pollution, ensuring quantity and quality of dietary fat is addressed, and increasing intake of phytonutrient rich foods, as well as including sufficient exercise, may prove beneficial in negating the effects of the SNP. <br/>
					<b>Genotype specific conclusion: </b> C allele is ancestral, it is more common and is not protective. C-allele is a part of APOE4 genoset evaluation<br/>
					<b>Study design: </b>Discovery sample description: 19,259 British ancestry individuals from 6863 families. Discovery ancestry (country of recruitment): 19259 European (U.K.)</td>
					</tr>
				
		</tbody>
	</table>
	</div>
	<div class="last"></div>
	</div>
</div>
<button class="accordion accordion-heading major-risk"  >Thrombophilia</button>
<div class="basic-panel major-risk">
	<button class="accordion accordion0 accordion1 major-risk"><div></div>Description</button>
	<div class="board variants panel panel-description major-risk">
		<p>Thrombophilia is a condition in which an individual has an increased tendency to develop blood clots. This can be caused by genetic or acquired factors, and the risk of developing thrombophilia increases with age. Thrombophilia risk has a strong genetic component, as certain genetic mutations can increase the risk of developing blood clots. The most common genetic mutations associated with thrombophilia are mutations in the genes that control the production and function of clotting factors.</p>
		<p>This report contains data of blood clot or deep vein thrombosis (<a href="https://www.pgscatalog.org/score/PGS000931/">PGS000931</a>), and the list with genes, associated with an increased risk of thrombophilia.</p>
		<p>However, it's important to note that having a genetic mutation does not necessarily mean that an individual will develop thrombophilia. Other factors, such as lifestyle factors and medical conditions, can also contribute to the development of thrombophilia.</p>
		</div>
		<button class="accordion accordion0 accordion1 prs-panel major-risk"><div></div>PRS</button>
	<div class="board variants panel panel-description major-risk">
		<p class="how_to">How to read the result?</p>
<p></p>
<p>PRS is represented as a&nbsp;<u>percentile</u> within a given population. For example, you have the 70th percentile, which means your personal genetic risk of the disease development is higher than 70 out of every 100 people in a chosen population.
</p>
<p></p>
<p>So percentile &#65310;50 shows an increased risk, &#65308;50 - decreased one.</p>
<p></p>

<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Blood clot or deep vein thrombosis (<a href="https://www.pgscatalog.org/score/PGS000931/">PGS000931</a>)</td><td rowspan="6"><canvas id="PGS000931_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>0.47095440857000004</td></tr>
    <tr><td>Count/total</td><td>360/534</td></tr>
    <tr><td>Average</td><td>0.0006541033452361111</td></tr>
    <tr><td>Percentile</td><td>47%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS000931_canvas", 0.47, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>
		</div>
	<button class="accordion0 accordion accordion-result major-risk"><div></div>Results</button>
<div class="content content-variants major-risk">
		<table class="ui selectable celled teal table table cancer bigger">
		<thead>
		<tr><th style="width:1%" class="clickable title" onclick="collapseExpandAll(this, 'thrombophiliaTBody')"><div class="ui tiny circular compact positive button">+</div></th><th>#</th>
				<th>RSID</th><th>Gene</th><th>Risk Allele</th><th>Genotype</th><th>Pubmed ID</th>
				<th>Population</th><th>P-Value</th><th>Weight</th></tr>
		</thead>
		<tbody id="thrombophiliaTBody">
			
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>1</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1799889">rs1799889</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=SERPINE">SERPINE</a></td>
					<td>G</td>
					<td>GG</td><td>[PMID 9700201]; [PMID 16424345]; [PMID 25020710]; [PMID: 17896948];</td><td>None</td>
					<td>None</td><td style="background-color:#ff3838">-1.3</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>Plasminogen activator inhibitor 1 (PAI-1) functions as the primary blocker of plasminogen activator in plasma. Increased levels of plasminogen activator inhibitor 1 could lead to excessive blocking of tissue-plasminogen activator, leading to decreased clot breakdown and eventually an unwanted blood clot. rs1799889 is an insertion/deletion polymorphism in the promoter region of the SERPINE1 gene, which is also known as plasminogen activator inhibitor type 1 (or PAI1 or PAI-1). The most common allele has a run of 5 G's in the area around this SNP, and in the literature the allele is commonly called the 5G allele. A deletion of one nucleotide leads to the "4G" allele, i.e. rs1799889(-). The 4G allele has been reported to increase risk for atherosclerosis, thrombosis and coronary artery disease [PMID 9484978]; <br/>
					<b>Genotype specific conclusion: </b> AA variant is associated with increased risk of blood clots, atherosclerosis and coronary artery disease<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>2</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2036914">rs2036914</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=F11">F11</a></td>
					<td>C</td>
					<td>CT</td><td>[PMID: 19583818]; [PMID: 31420334]; [PMID 21232005]; [PMID 23150947]; [PMID: 25091233]</td><td>European; African Americans</td>
					<td>[PMID 31420334]: 2 x 10-54</td><td style="background-color:#ff4747">-1.2</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>The F11 gene provides instructions for making a protein called factor XI. This protein plays a role in the coagulation cascade, which is a series of chemical reactions that forms blood clots in response to injury. After an injury, clots seal off blood vessels to stop bleeding and trigger blood vessel repair. Polymorphisms in this gene are associated with increased risk of deep vein thrombosis.  <br/>
					<b>Genotype specific conclusion: </b> TC variant is a risk variant.<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>3</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1800790">rs1800790</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=FGB">FGB</a></td>
					<td>A</td>
					<td>AG</td><td> [PMID 29235504]; [PMID: 31354890]</td><td>None</td>
					<td>None</td><td style="background-color:#ffd1d1">-0.3</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>FGB gene encodes the beta subunit of fibrinogen (genetic marker FGB (-455GA). Gene polymorphism leads to an increase in fibrinogen synthesis, resulting in an increased risk of peripheral and coronary thrombosis, the risk of thromboembolic complications during pregnancy, childbirth and the postpartum period.<br/>
					<b>Genotype specific conclusion: </b> GA variant is associated with increased cardiovascular diseases risk.<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>4</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs8176719">rs8176719</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ABO">ABO</a></td>
					<td>C</td>
					<td>C-</td><td>[PMID: 22672568]; [PMID 21463476];</td><td>European</td>
					<td>[PMID: 22672568]: 6 x 10-12</td><td style="background-color:#ffd1d1">-0.3</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>A genome-wide association study of venous thromboembolism identifies ABO polymorphism as risk variants. The replication study confirmed a significant association of ABO with VTE (venous thromboembolism). The ABO blood groups are determined by the ABO locus located on chromosome 9 (9q34.1-q34.2). ABO blood group was one of the first identified risk factor for venous thrombosis (VT), and its thrombotic effect is considered to be mediated by the ABO(H) structures, which are expressed on the glycans of factor VIII (FVIII) and von Willebrand factor (VWF), as well as by modulation of soluble ICAM-1, soluble P-selectin, and soluble E-selectin levels.<br/>
					<b>Genotype specific conclusion: </b> C genotype is associated with an increased venous thromboembolism risk.<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>5</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs2519093">rs2519093</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ABO">ABO</a></td>
					<td>T</td>
					<td>CC</td><td>[PMID: 21463476]; [PMID: 33512453]; [PMID: 31420334]</td><td>European; African American;</td>
					<td>[PMID: 31420334]: 4 x 10-169</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>A genome-wide association study of venous thromboembolism identifies ABO polymorphism as risk variants. The replication study confirmed a significant association of ABO with VTE (venous thromboembolism). The ABO blood groups are determined by the ABO locus located on chromosome 9 (9q34.1-q34.2). ABO blood group was one of the first identified risk factor for venous thrombosis (VT), and its thrombotic effect is considered to be mediated by the ABO(H) structures, which are expressed on the glycans of factor VIII (FVIII) and von Willebrand factor (VWF), as well as by modulation of soluble ICAM-1, soluble P-selectin, and soluble E-selectin levels. <br/>
					<b>Genotype specific conclusion: </b> CC genotype is NOT associated with an increased venous thromboembolism risk.<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>6</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs6025">rs6025</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=F5">F5</a></td>
					<td>T</td>
					<td>TT</td><td>[PMID 28373160];  [PMID 23900608]; [PMID 14996674]; [PMID 10666427]; [PMID 10477778]; </td><td>European; African American; </td>
					<td>[PMID: 31676865]: 1 x 10-300; </td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>The F5 gene encodes coagulation factor V. Factor V Leiden thrombophilia is the most common inherited form of thrombophilia and prothrombin-related thrombophilia is the second most common genetic form of thrombophilia, occurring in about 1.7-3% of the European and US general populations. rs6025 represents a SNP in the Factor V F5 gene, encoding a change in the protein from an arginine at position 506 to a glutamine. The resulting rs6025(A) allele encodes a mutation known as the Leiden mutation, R506Q. Polymorphism leads to Factor V resistance to inactivation by  activated protein C, which increases thrombophilia risk. <br/>
					<b>Genotype specific conclusion: </b> CC genotype is NOT associated with increased risk.<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>7</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1799963">rs1799963</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=F2">F2</a></td>
					<td>A</td>
					<td>AA</td><td>[PMID 23900608]; [PMID 19404532];[PMID 22784820];</td><td>European</td>
					<td>[PMID: 26908601]:1 x 10-24</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>G20210A polymorphism (rs1799963) within the prothrombin gene is associated with a higher circulation level of prothrombin. That in turn increasing the likelihood of developing thrombosis and myocardial infarction (MI). A change in the prothrombin gene is present in 2-4% (or 1 in 50 to 1 in 25) of Caucasians, and is more common in individuals of European ancestry. In the United States, approximately 0.4% (about 1 in 250) of African Americans also have the mutation. Prothrombin G20210A mutation is rare in other groups. <br/>
					<b>Genotype specific conclusion: </b> GG carriers have normal/common risk of thrombosis.<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>8</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs1801133">rs1801133</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=MTHFR">MTHFR</a></td>
					<td>A</td>
					<td>AA</td><td>[PMID: 30466296]; [PMID: 20031578]; [PMID: 34707639];</td><td>European; East Asians; West Asians;</td>
					<td>[PMID: 20031578]: 8 x 10-35; [PMID: 34707639]: 4 x 10-104;</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>Gene product is envolved in the folate metabolism. Polymorphisms are associated with elevated homocysteine risk, folic acid and B12 deficiency. For the rs1801133 polymorphism, significant association with the risk of venous thromboembolism was found. Further subgroup analyses according to ethnicity of participants revealed that the rs1801133 polymorphism was significantly associated with the risk of venous thromboembolism in Caucasians, East Asians, and West Asians [PMID: 30466296]. <br/>
					<b>Genotype specific conclusion: </b> GG variant is NOT associated with hyperhomocysteinemia and thrombophilia.<br/>
					<b>Study design: </b></td>
					</tr>
				
				<tr class="clickable" onclick="doOnClick(this)"><td><div class="ui tiny circular compact positive button">+</div></td><td>9</td>
					<td><a href="https://www.ncbi.nlm.nih.gov/snp/rs5918">rs5918</a></td>
					<td><a href="https://genomics.senescence.info/longevity/gene.php?id=ITGB3">ITGB3</a></td>
					<td>C</td>
					<td>CC</td><td>[PMID 8598867]; [PMID 9700201]; [PMID 11723016]; [PMID 19786296]</td><td>None</td>
					<td>None</td><td style="background-color:#ffffff">0.0</td></tr>
				<tr class="hidden" style="display:none;"><td colspan="11">
					<b>Conclusion: </b>The ITGB3 gene provides instructions for making the beta3 subunit of a receptor protein called integrin alphaIIb/beta3 (αIIbβ3), which is found on the surface of small cells called platelets. Platelets circulate in blood and are an essential component of blood clots. The 'A2' allele of the ITGB3 is encoded by rs5918(C), and it has been implicated as increasing the risk of myocardial infarction, heart disease, and resistance to blood-thinning benefits of aspirin. On its own, the A2 allele is implicated especially in early onset heart disease [PMID 8598867]; in combination with the 4G allele of the PAI1 gene, rs1799889, the increased risk of myocardial infarction in a Finnish study population was 4 fold higher (odds ratio = 4.5, p=0.001), particularly in males (odds ratio = 6.4, p=0.0005) [PMID 9700201]. A2 allele carriers also appear to be relatively resistant to the anti-thrombotic (i.e. anti-clotting) actions normally associated with aspirin use.[PMID 11723016]. <br/>
					<b>Genotype specific conclusion: </b> TT variant is not associated with increased heart diseases risk. Aspirin is efficient as anti-thrombotic drug.<br/>
					<b>Study design: </b></td>
					</tr>
				
		</tbody>
	</table>
	</div>
	<div class="last"></div>
</div>
<h2 id="prs" style="color:rgb(57, 145, 123);">All polygentic risk scores</h2>
<button class="accordion accordion0 accordion1 prs-panel major-risk special" ><div></div>PRS</button>
<div class="board variants panel prs-panel major-risk">
	<div class="board variants prs-panel panel">
		</div>
		
<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Longevity PRS (PRS5)</td><td rowspan="6"><canvas id="PRS51_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-0.24635944</td></tr>
    <tr><td>Count/total</td><td>156/330</td></tr>
    <tr><td>Average</td><td>-0.0007896135897435898</td></tr>
    <tr><td>Percentile</td><td>1%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PRS51_canvas", 0.01, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 1);
})
</script>

<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Coronary heart disease (<a href="https://www.pgscatalog.org/score/PGS000818/">PGS000818</a>)</td><td rowspan="6"><canvas id="PGS0008181_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>1.1501</td></tr>
    <tr><td>Count/total</td><td>73/138</td></tr>
    <tr><td>Average</td><td>0.007877397260273972</td></tr>
    <tr><td>Percentile</td><td>97%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS0008181_canvas", 0.97, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>


<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Coronary atherosclerosis (<a href="https://www.pgscatalog.org/score/PGS001839/">PGS001839</a>)</td><td rowspan="6"><canvas id="PGS0018391_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>1.041014480505508</td></tr>
    <tr><td>Count/total</td><td>15920/25425</td></tr>
    <tr><td>Average</td><td>3.2695178407836306e-05</td></tr>
    <tr><td>Percentile</td><td>79%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS0018391_canvas", 0.79, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>


<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Blood clot or deep vein thrombosis (<a href="https://www.pgscatalog.org/score/PGS000931/">PGS000931</a>)</td><td rowspan="6"><canvas id="PGS0009311_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>0.47095440857000004</td></tr>
    <tr><td>Count/total</td><td>360/534</td></tr>
    <tr><td>Average</td><td>0.0006541033452361111</td></tr>
    <tr><td>Percentile</td><td>47%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS0009311_canvas", 0.47, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>


<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">C-reactive protein measurement (<a href="https://www.pgscatalog.org/score/PGS000314/">PGS000314</a>)</td><td rowspan="6"><canvas id="PGS0003141_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>2.0094686</td></tr>
    <tr><td>Count/total</td><td>38/77</td></tr>
    <tr><td>Average</td><td>0.026440376315789473</td></tr>
    <tr><td>Percentile</td><td>1%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS0003141_canvas", 0.01, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>



<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Osteoporosis (<a href="https://www.pgscatalog.org/score/PGS001273/">PGS001273</a>)</td><td rowspan="6"><canvas id="PGS001273_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>0.13876892539999997</td></tr>
    <tr><td>Count/total</td><td>200/314</td></tr>
    <tr><td>Average</td><td>0.00034692231349999994</td></tr>
    <tr><td>Percentile</td><td>99%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS001273_canvas", 0.99, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>




<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Facial ageing (<a href="https://www.pgscatalog.org/score/PGS001141/">PGS001141</a>)</td><td rowspan="6"><canvas id="PGS001141_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-0.69890911514094</td></tr>
    <tr><td>Count/total</td><td>4889/7668</td></tr>
    <tr><td>Average</td><td>-7.147771682766823e-05</td></tr>
    <tr><td>Percentile</td><td>9%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS001141_canvas", 0.09, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>



<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Obesity (<a href="https://www.pgscatalog.org/score/PGS001298/">PGS001298</a>)</td><td rowspan="6"><canvas id="PGS001298_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-0.00971829751099984</td></tr>
    <tr><td>Count/total</td><td>5759/9223</td></tr>
    <tr><td>Average</td><td>-8.437486986455844e-07</td></tr>
    <tr><td>Percentile</td><td>17%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS001298_canvas", 0.17, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>




<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Chronic kidney disease (<a href="https://www.pgscatalog.org/score/PGS001272/">PGS001272</a>)</td><td rowspan="6"><canvas id="PGS001272.txt_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>0.13935973627</td></tr>
    <tr><td>Count/total</td><td>105/158</td></tr>
    <tr><td>Average</td><td>0.0006636177917619048</td></tr>
    <tr><td>Percentile</td><td>89%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS001272.txt_canvas", 0.89, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>




<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Essential (primary hypertension) (<a href="https://www.pgscatalog.org/score/PGS000957/">PGS000957</a>)</td><td rowspan="6"><canvas id="PGS000957.txt_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>0.6894196517595985</td></tr>
    <tr><td>Count/total</td><td>7075/11265</td></tr>
    <tr><td>Average</td><td>4.872223687346986e-05</td></tr>
    <tr><td>Percentile</td><td>45%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS000957.txt_canvas", 0.45, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>



<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Longevity PRS with local cohort (PRS5_local)</td><td rowspan="6"><canvas id="PRS5_local_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>3.5692209080805424</td></tr>
    <tr><td>Count/total</td><td>156/331</td></tr>
    <tr><td>Average</td><td>0.01143981060282225</td></tr>
    <tr><td>Percentile</td><td>1%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PRS5_local_canvas", 0.01, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 1);
})
</script>


<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Longevity PRS extended local cohort (PRS4)</td><td rowspan="6"><canvas id="PRS4_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-0.07321543999999996</td></tr>
    <tr><td>Count/total</td><td>472/1216</td></tr>
    <tr><td>Average</td><td>-7.755872881355929e-05</td></tr>
    <tr><td>Percentile</td><td>8%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PRS4_canvas", 0.08, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 1);
})
</script>


<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Preeclampsia (pregnancy complication) - mother (<a href="https://www.pgscatalog.org/score/PGS003586/">PGS003586</a>)</td><td rowspan="6"><canvas id="PGS003586_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-0.09672966086996579</td></tr>
    <tr><td>Count/total</td><td>585432/1086946</td></tr>
    <tr><td>Average</td><td>-8.261391662051766e-08</td></tr>
    <tr><td>Percentile</td><td>99%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS003586_canvas", 0.99, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>


<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Vitamin B12 deficiency, anemia - mother (<a href="https://www.pgscatalog.org/score/PGS001305/">PGS001305</a>)</td><td rowspan="6"><canvas id="PGS001305_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-0.67183784348</td></tr>
    <tr><td>Count/total</td><td>77/121</td></tr>
    <tr><td>Average</td><td>-0.0043625833992207794</td></tr>
    <tr><td>Percentile</td><td>49%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS001305_canvas", 0.49, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>



<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Rhesus isoimmunization in pregnancy - mother (<a href="https://www.pgscatalog.org/score/PGS002080/">PGS002080</a>)</td><td rowspan="6"><canvas id="PGS002080_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>0.003624179714136298</td></tr>
    <tr><td>Count/total</td><td>17701/32414</td></tr>
    <tr><td>Average</td><td>1.0237217428778877e-07</td></tr>
    <tr><td>Percentile</td><td>67%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS002080_canvas", 0.67, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>



<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Type 1 diabetes - child (<a href="https://www.pgscatalog.org/score/PGS002025/">PGS002025</a>)</td><td rowspan="6"><canvas id="PGS002025_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>0.00477906467097806</td></tr>
    <tr><td>Count/total</td><td>58140/106789</td></tr>
    <tr><td>Average</td><td>4.109962737339233e-08</td></tr>
    <tr><td>Percentile</td><td>48%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS002025_canvas", 0.48, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>


<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Type 1 diabetes - child (<a href="https://www.pgscatalog.org/score/PGS003749/">PGS003749</a>)</td><td rowspan="6"><canvas id="PGS003749_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-39.279</td></tr>
    <tr><td>Count/total</td><td>1824/6612</td></tr>
    <tr><td>Average</td><td>-0.010767269736842106</td></tr>
    <tr><td>Percentile</td><td>1%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS003749_canvas", 0.01, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>




<table class="ui celled teal table prs">
	<tr><td>Title</td><td  style="width:100%">Neurodevelopmental disorder / autism spectrum disorder - child (<a href="https://www.pgscatalog.org/score/PGS000327/">PGS000327</a>)</td><td rowspan="6"><canvas id="PGS000327_canvas"></canvas></td></tr>
    <tr><td>Sum</td><td>-4.108067599999986</td></tr>
    <tr><td>Count/total</td><td>16089/35078</td></tr>
    <tr><td>Average</td><td>-0.0001276669650071473</td></tr>
    <tr><td>Percentile</td><td>89%</td></tr>
</table>
<br/>
<script>
	window.addEventListener('load', function() {
	drawGausian("PGS000327_canvas", 0.89, "#ffffff", "#c2f3ec", "#000000", "#b4ea54", 0);
})
</script>


	</div>
<div class="last outside"></div>
</div>
<script>
const BtnBackToTop = document.querySelector("#toTop")

window.addEventListener("scroll", scrollFunction);

function scrollFunction() {
  if (window.pageYOffset > 300) { // Show backToTopButton
    if(!BtnBackToTop.classList.contains("btnEntrance")) {
		BtnBackToTop.classList.remove("btnExit");
		BtnBackToTop.classList.add("btnEntrance");
		BtnBackToTop.style.display = "block";
    }
  }
  else { // Hide backToTopButton
    if(BtnBackToTop.classList.contains("btnEntrance")) {
		BtnBackToTop.classList.remove("btnEntrance");
		BtnBackToTop.classList.add("btnExit");
      	setTimeout(function() {
        BtnBackToTop.style.display = "none";
      }, 250);
    }
  }
}



BtnBackToTop.addEventListener("click", backToTop);

function backToTop() {
	window.scrollTo(0, 0);
}
let currentWidth = document.body.clientWidth;

let styles = calculatingStyles(currentWidth);

window.addEventListener("resize", function(event) {
    currentWidth = document.body.clientWidth;
	styles = calculatingStyles(currentWidth);
	var acc = document.getElementsByClassName("accordion");
	var i;
	for (i = 0; i < acc.length; i++) {
		if (acc[i].classList.contains("panel-description")){
		acc[i].classList.toggle("active");
		acc[i].style.borderRadius = "10px 10px 0px 0px"
		var panel = acc[i].nextElementSibling;
		panel.style.display="block"
		panel.style.maxHeight = panel.scrollHeight + "px";
	}
	}
})

function calculatingStyles(width){

let listOfStyles = {widthForActive:"", widthForClosed:"", marginForActive:"", marginForClosed:""};

if (width >= 1400){
	listOfStyles.widthForActive = "85";
	listOfStyles.widthForClosed = "50";
	console.log(listOfStyles)
}else if((width < 1400) && (currentWidth >= 600)){
		listOfStyles.widthForActive = "98";
		listOfStyles.widthForClosed = "80";
} else {
	listOfStyles.widthForActive = "100";
	listOfStyles.widthForClosed = "95";
}
listOfStyles.marginForActive = (100-listOfStyles.widthForActive)/2 + '%';
listOfStyles.marginForClosed = (100-listOfStyles.widthForClosed)/2 + '%';
listOfStyles.widthForActive = listOfStyles.widthForActive + "%";
listOfStyles.widthForClosed = listOfStyles.widthForClosed + "%";

return listOfStyles

};





var acc = document.getElementsByClassName("accordion");
var i;

for (i = 0; i < acc.length; i++) {
	if (acc[i].nextElementSibling.classList.contains("panel-description")){
		acc[i].classList.toggle("active");
		acc[i].style.borderRadius = "10px 10px 0px 0px"
		var panel = acc[i].nextElementSibling;
		panel.style.display="block"
		panel.style.maxHeight = panel.scrollHeight + "px";
	}
  acc[i].addEventListener("click", function() {
    this.classList.toggle("active");
	let button = this;
    var panel = this.nextElementSibling;
    if (panel.style.maxHeight) {
		let height = panel.style.maxHeight;
		let time = parseInt((height.slice(0,-2)))/7;
		if (time>300){
			time =200;
		}
        panel.style.maxHeight = null;
	    setTimeout(function() {
			panel.style.display = "none"
			button.style.borderRadius = "10px";
			if (button.classList.contains("accordion-result") || button.classList.contains("accordion-heading")){
				button.style.width = styles.widthForClosed;
				button.style.marginRight = styles.marginForClosed;
				button.style.marginLeft = styles.marginForClosed;
	}
      }, time);
    } else {
		if (button.classList.contains("accordion-result") || button.classList.contains("accordion-heading")){
			button.style.width =  styles.widthForActive;
			button.style.marginRight = styles.marginForActive;
			button.style.marginLeft = styles.marginForActive;
			setTimeout(function() {
			panel.style.display = "block"
			panel.style.maxHeight = panel.scrollHeight + "px";
			panel.parentElement.style.maxHeight = Number(panel.parentElement.style.maxHeight.slice(0, -2)) + panel.scrollHeight + "px";
			button.style.borderRadius = "10px 10px 0px 0px";
		}, 150);}
		else{
			panel.style.display = "block"
			panel.style.maxHeight = panel.scrollHeight + "px";
			panel.parentElement.style.maxHeight = Number(panel.parentElement.style.maxHeight.slice(0, -2)) + panel.scrollHeight + "px";
			button.style.borderRadius = "10px 10px 0px 0px"
		}
	}
  });
}
var canvas1 = document.getElementById('canvas1');

</script>
<style>

.btnExit{
			animation-duration: 0.5s;
			animation-fill-mode: both;
			animation-name: btnExit;
	}

@keyframes btnExit {
	from {
				opacity: 1;
			}
			to {
				opacity: 0;
				transform: scale3d(0.8, 0.8, 0.8);
			}
			}


.btnEntrance{
			animation-duration: 0.5s;
			animation-fill-mode: both;
			animation-name: btnEntrance;
		}

		@keyframes btnEntrance {
			from {
				opacity: 0;
				transform: scale3d(0.3, 0.3, 0.3);
			}
			to {
				opacity: 1;
			}
			}

</style>
</body>
</html>